% !TEX encoding = UTF-8 Unicode
% MOST latex template
% https://www.math.sinica.edu.tw/www/tex/default.jsp
% Ipe drawing: the extensible drawing editor Ipe; or Inkscape https://ipe.otfried.org/manual/manual.html
\documentclass[12pt, a4paper]{article}

\usepackage{pdfpages} % for more pages of PDF \includegraphics
\usepackage{pbox} 

%\usepackage[table,xcdraw]{xcolor}
\usepackage{xcolor}
\definecolor{asparagus}{rgb}{0.53, 0.66, 0.42}

\newenvironment{MyColorPar}[1]{% for RED marked of revision
    \leavevmode\color{#1}\ignorespaces%
}{%
}%

\usepackage{titlecaps}
\Addlcwords{are or etc is of the}

%\usepackage{url}

\usepackage{array}
\usepackage{ragged2e}
\usepackage{rotating}
\usepackage{tabularx} % wide table
\usepackage{makecell}
% \usepackage{array}
\usepackage{colortbl} % \arrayrulecolor
\usepackage{multirow} % \multirow
\usepackage{hhline}
\usepackage{siunitx} % for  1e-10 scientific notation
%\usepackage{caption}
%\usepackage{subcaption}
\usepackage{booktabs, multirow} % for borders and merged ranges
\usepackage{soul}% for underlines
%\usepackage[table]{xcolor} % for cell colors
\usepackage{changepage,threeparttable} 
%%%



\usepackage[bf]{caption}
\newcommand{\bcaption}[2]{\caption{\textbf{#1} #2}}

\usepackage{outlines}

\usepackage{pgfgantt} % for Gantt chart/flow chart
\definecolor{barblue}{RGB}{153,204,254}
\definecolor{groupblue}{RGB}{51,102,254}
\definecolor{linkred}{RGB}{165,0,33}

\usepackage{newfloat} % for caption of smartdiagram
\DeclareFloatingEnvironment[fileext=diag,placement={!ht},name=Figure ]{diag} % diag

\usepackage{smartdiagram} %flowchart
\usesmartdiagramlibrary{additions} 
\usepackage{tikz}   % for DAG plot by  tikzpicture
\usetikzlibrary{arrows}
\input{arrowsnew}

\usepackage{times}
\usepackage{geometry}                % See geometry.pdf to learn the layout options. There are lots.
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{epstopdf}
\usepackage{wrapfig}
\usepackage{natbib}
\bibpunct{(}{)}{;}{a}{}{,} % to follow the A&A style
\usepackage[pdftex, plainpages=false, colorlinks=true, linkcolor=blue, citecolor=blue, bookmarks=false]{hyperref}
\usepackage{setspace}
\usepackage{multicol}
\usepackage{sectsty}
\usepackage{url}
\usepackage{lipsum}
\usepackage[tiny,compact]{titlesec}
\usepackage{fancyhdr}
\usepackage[font=footnotesize,labelfont=bf]{caption}
\usepackage{verbatim}
\usepackage[super]{nth}
\usepackage{lastpage}

% ---- Chinese characters --------------------------------------------------------
\usepackage{CJKutf8}
\newcommand{\cntext}[1]{\begin{CJK*}{UTF8}{bkai}#1\end{CJK*}}
%----------------------------------------------------------------------------------------
% --- Page Style ----

%\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrulewidth}{0pt}
\setlength{\paperheight}{29.7cm}
\setlength{\paperwidth}{21cm}
\addtolength{\voffset}{-1in}
\addtolength{\hoffset}{-1in}
\setlength{\topmargin}{1in}
\setlength{\oddsidemargin}{1.5cm}
\setlength{\evensidemargin}{1.5cm}
\setlength{\textwidth}{18cm}
\setlength{\textheight}{23.5cm}
\setlength{\footskip}{32pt}
\setlength{\marginparsep}{0.5cm}
\setlength{\marginparwidth}{1.5cm}
\setlength{\headheight}{0pt}
\setlength{\headsep}{1cm}
\setlength{\parindent}{0cm}
\setlength{\parskip}{.1cm}

%--- Fancy look -------------------------------------------------------------------------------------------- C003
\pagestyle{fancy} % for header and footer
\fancyhf{} % clears the header and footer,
\lhead{\fancyplain{}{\doctitle}}
\rhead{\fancyplain{}{Somaliland}}
%co-PI: Li-Hsing Chi}} % *** Fill your name here ***

% ***
%\cfoot{\fancyplain{}{\cntext{共} \pageref*{LastPage} \cntext{頁}~~~~\cntext{第} \thepage \cntext{頁}}}
%\lfoot{\underline{\cntext{表} C003}}
%  ******

%
% C004 redefine?
%\fancypagestyle{fancy004}[fancy]{ % based on fancy
%\fancyhf{} % clears the header and footer,
%\lhead{\fancyplain{}{\doctitle}}
%\rhead{\fancyplain{}{PI: Li-Hsing Chi}} % *** Fill your name here ***
%\cfoot{\fancyplain{}{\cntext{共} \pageref*{LastPage} \cntext{頁}~~~~\cntext{第} \thepage \cntext{頁}}}
%\fancyfoot[l]{\underline{\cntext{表} C004}}
%}


% --- define the title page 

\fancypagestyle{titlepage}{
\renewcommand{\headrulewidth}{0pt}
\rhead{}
\lhead{}
%\rfoot{}
%\lfoot{}
\cfoot{\fancyplain{}{\cntext{共} \pageref*{LastPage} \cntext{頁}~~~~\cntext{第} \thepage \cntext{頁}}}
\lfoot{\underline{\cntext{表} C002}}
}

%----------------------------------------------------------------------------------------------------------------
\bibliographystyle{apj} % it needs \citep{}

% --- additional stuff --------------------------------------------------------------------------------------
\newcommand{\todo}[1]{{\color{red}$\blacksquare$~\textsf{[TODO: #1]}}}
\newcommand{\doctitle}{Hargeisa Group Hospital}
%Deep Learning in Verrucous Hyperplasia} % *** Project Short Title ***

% *** specific aim 1 2 3 4:
\newcommand{\aimone}{To construct \acrshort{ovh} and leukoplakia datasets from Chi-Mei Hospital and \acrfull{tmuh}}
\newcommand{\aimtwo}{To identify genetic variants associated with malignant transformation of \acrshort{ovh} by implementing genome-wide association testing (GWAS)}
\newcommand{\aimthree}{To construct a deep learning platform of genomics (mutation and expression) from pathology images}
\newcommand{\aimfour}{To develop risk prediction model for malignant transformation of \acrshort{ovh} by integrating risk factors identified from all sub-projects: SP1, SP2, and SP3}


% --------------------------------------------------------------------------------
%%% for abbreviations, or acronyms
\usepackage[automake, acronym, nopostdot]{glossaries} 
\usepackage{glossary-inline}
%\setacronymstyle{long-short}
%\renewcommand*{\glossarysection}[2][]{} 
%\renewcommand*{\glossarysection}[2][]{\textbf{#1}: }
% for abbreviations environment
%\newcommand{\abbrlabel}[1]{\makebox[3cm][l]{\textbf{#1}\ \dotfill}}
\newenvironment{abbreviation}%{\begin{list}{}{\renewcommand{\makelabel}{\abbrlabel}}}{\end{list}}
% \newenvironment{<name>}[<number>][<default>]


\makeglossaries %https://tex.stackexchange.com/questions/110095/list-of-acronyms-is-not-displayed

\newacronym{fat1}{FAT1}{FAT atypical cadherin 1}
\newacronym{acta2}{ACTA2}{actin alpha 2, smooth muscle} % or alias as alpha smooth muscle actin,  (α-SMA, SMactin, alpha-SM-actin, ASMA)

\newacronym{cnn}{CNN}{Convolutional Neural Network}
\newacronym{gnn}{GNN}{Graph Neural Network}
\newacronym{gcnn}{GCNN}{Graph Convolutional Neural Network}
\newacronym{rnn}{RNN}{Recurrent Neural Network}
\newacronym{ehr}{EHR}{Electric Healthcare Records}

\newacronym{ihc}{IHC}{immunohistochemistry}
\newacronym{fdr}{FDR}{false discovery rate}

\newacronym{hpa}{HPA}{the Human Protein Atlas}
% The Health Promotion Administration (HPA) in Taiwan
\newacronym{Thpa}{HPA}{Health Promotion Administration}

\newacronym{oscc}{OSCC}{squamous cell carcinoma}
\newacronym{ovh}{OVH}{oral verrucous hyperplasia}

\newacronym{hnscc}{HNSCC}{head and neck squamous cell carcinoma}
\newacronym{tcga}{TCGA}{the Cancer Genome Atlas}
\newacronym{tcpa}{TCPA}{the Cancer Proteome Atlas}
\newacronym{tcia}{TCIA}{the Cancer Imaging Archive}

\newacronym{rna}{RNA}{ribonucleic acid}
\newacronym{rnaseq}{RNA-Seq}{RNA sequencing}
\newacronym{lncrna}{lncRNA}{long non-coding RNA}
%\newacronym{km}{KM}{Kaplan-Meier}
\newacronym{rppa}{RPPAs}{reverse-phase protein arrays}
\newacronym{rpma}{RPMA}{reverse-phase protein lysate microarray}



\newacronym{go}{GO}{Gene Ontology}
\newacronym{ipi}{IPI}{international protein index database}

\newacronym{mmp}{MMP}{matrix metalloproteinase}
 %DKK1, CAMK2N1, STC2, PGK1, SURF4, USP10, NDFIP1, FOXA2, STIP1, and DKC1
 %ZNF557, ZNF266, IL19, MYO1H, FCGBP, LOC148709, EVPLL, PNMA5, KIAA1683, and NPB

\newacronym{DKK1}{DKK1}{dickkopf WNT signaling pathway inhibitor 1} 
\newacronym{CAMK2N1}{CAMK2N1}{calcium/calmodulin dependent protein kinase II inhibitor 1} 
\newacronym{STC2}{STC2}{stanniocalcin 2} 
\newacronym{PGK1}{PGK1}{phosphoglycerate kinase 1} 
\newacronym{SURF4}{SURF4}{surfeit 4} 
\newacronym{USP10}{USP10}{ubiquitin specific peptidase 10} 
\newacronym{NEDD4}{NEDD4}{neural precursor cell expressed, developmentally down-regulated 4}
\newacronym{NDFIP1}{NDFIP1}{NEDD4 family interacting protein 1} 
\newacronym{FOXA2}{FOXA2}{forkhead box A2} 
\newacronym{STIP1}{STIP1}{stress-induced-phosphoprotein 1} 
\newacronym{DKC1}{DKC1}{dyskeratosis congenita 1, dyskerin} 

\newacronym{ZNF557}{ZNF557}{zinc finger protein 557} 
\newacronym{ZNF266}{ZNF266}{zinc finger protein 266} 
\newacronym{IL19}{IL19}{interleukin 19} 
\newacronym{MYO1H}{MYO1H}{myosin 1H} 
\newacronym{FCGBP}{FCGBP}{Fc fragment of IgG binding protein} 
\newacronym{LOC148709}{LOC148709}{LncRNA LOC148709} 
\newacronym{EVPLL}{EVPLL}{envoplakin-like protein} 
\newacronym{PNMA5}{PNMA5}{paraneoplastic antigen like 5} 
%\newacronym{KIAA1683}{KIAA1683}{IQCN, IQ Motif Containing N} 
\newacronym{IQCN}{IQCN}{IQ motif containing N} % previous name KIAA1683
% "IQ" refers to the first two amino acids of the motif: isoleucine (commonly) and glutamine (invariably)
\newacronym{NPB}{NPB}{neuropeptide B} 

 \newacronym{rt}{RT}{radiation therapy}
 \newacronym{nccn}{NCCN}{National Comprehensive Cancer Network}
 \newacronym{hif}{HIF}{hypoxia-inducible factor}
 \newacronym{egfr}{EGFR}{epidermal growth factor receptor}
 \newacronym{ras}{RAS}{rat sarcoma}
 \newacronym{hras}{HRAS}{Harvey rat sarcoma viral oncoprotein}
 \newacronym{erk}{ERK}{extracellular signal-regulated kinases}
 \newacronym{us}{US}{United States}
 \newacronym{fda}{FDA}{Food and Drug Administration}
 \newacronym{tpf}{Tax-PF}{docetaxel, cisplatin, and 5-fluorouracil}
 \newacronym{tki}{TKI}{tyrosine kinase inhibitor}
 \newacronym{her}{HER}{human epidermal growth factor receptor}
 \newacronym{ici}{ICI}{immune-checkpoint inhibitor}
 %\newacronym{ctla4}{CTLA-4}{cytotoxic T lymphocyte antigen 4}
 \newacronym{pd1}{PD-1}{programmed death 1}
 %\newacronym{pdl1}{PD-L1}{programmed death ligand 1}
 \newacronym{tim3}{TIM-3}{T-cell immunoglobulin mucin protein 3}
 \newacronym{lag3}{LAG-3}{lymphocyte activation gene 3}
 \newacronym{ifng}{IFN-$\gamma$}{interferon gamma}
 \newacronym{tigit}{TIGIT}{T cell immunoglobin and immunoreceptor tyrosine-based inhibitory motif}
 \newacronym{gitr}{GITR}{glucocorticoid-induced tumor necrosis factor receptor}
 \newacronym{vista}{VISTA}{V-domain Ig suppressor of T-cell activation}
 \newacronym{tmsb4x}{TMSB4X}{thymosin beta-4 X-linked}
 \newacronym{emt}{EMT}{epithelial-mesenchymal-transition}
 \newacronym{gdc}{GDC}{Genomic Data Commons}
 \newacronym{nci}{NCI}{the National Cancer Institute}
 \newacronym{gdac}{GDAC}{Genome Data Analysis Center}
 \newacronym{rest}{REST}{Representational State Transfer} 
 \newacronym{api}{API}{Application Programmable Interface}
\newacronym{grch38}{GRCh38}{Genome Reference Consortium Homo sapiens genome assembly 38}
\newacronym{fpkm}{FPKM}{Fragments per kilobase per million reads mapped}
\newacronym{rsem}{RSEM}{RNA-Seq by Expectation-Maximization}
\newacronym{slca}{SLC35E2A}{solute carrier family 35 member E2A}
\newacronym{slcb}{SLC35E2B}{solute carrier family 35 member E2B}
\newacronym{cde}{CDE}{Common Data Element}
\newacronym{id}{ID}{identification}
\newacronym{ajcc}{AJCC}{the American Joint Committee on Cancer}
\newacronym{uicc}{UICC}{he Union for International Cancer Control}
\newacronym{tnm}{TNM}{the tumor size (T), cervical lymph node metastases (N), and distal metastasis status (M)}
\newacronym{ci95}{95\% CI}{95\% confidence interval}
\newacronym{os}{OS}{overall survival}
\newacronym{rfs}{RFS}{recurrence-free survival}
\newacronym{hr}{HR}{hazard ratio}
\newacronym{hpv}{HPV}{human papillomavirus}
\newacronym{ene}{ENE}{extra-nodal extension}
\newacronym{lvsi}{LVSI}{lymph-vascular space invasion}
\newacronym{pni}{PNI}{perineural invasion}
\newacronym{doi}{DOI}{depth of invasion}
\newacronym{lnd}{LND}{lymph node density}
\newacronym{wpoi5}{WPOI-5}{worst pattern of invasion score 5}
\newacronym{glut4}{GLUT4}{glucose transporters 4}
\newacronym{slc2a4}{SLC2A4}{solute carrier family 2 member A4}
\newacronym{trim24}{TRIM24}{tripartite motif-containing 24}
\newacronym{til}{TIL}{tumor-infiltrating lymphocytes}
\newacronym{tmb}{TMB}{tumor mutational burden}

%\newacronym{hpa}{HPA}{the Human Protein Atlas}
\newacronym{cart}{CAR-T}{chimeric antigen receptor T cells}

\newacronym{ptta}{PTTA}{\textit{P} value of t-test or ANOVA}
\newacronym{anova}{ANOVA}{analysis of variance}
\newacronym{lcms}{LC-MS/MS}{liquid chromatography with tandem mass spectrometry}
\newacronym{maldi}{MALDI MS}{matrix-assisted laser desorption/ionization mass spectrometry}
\newacronym{maldii}{MALDI IMS}{matrix-assisted laser desorption/ionization imaging mass spectrometry}

\newacronym{pcr}{PCR}{polymerase chain reaction}
\newacronym{rtpcr}{RT-PCR}{reverse-transcription PCR}
\newacronym{qpcr}{RT-qPCR}{quantitative real-time reverse-transcription PCR}

\newacronym{tmuh}{TMUH}{Taipei Medical University Hospital}

\newacronym{dag}{DAG}{direct acyclic graph}

%ohm$V3V1V2 disserttion
\newacronym{cd}{CD}{cluster of differentiation}
\newacronym{ctla4}{CTLA-4}{cytotoxic T-lymphocyte associated protein 4 (CD152)}
\newacronym{clsm}{CLSM}{confocal laser scanning microscopy}
\newacronym{dapi}{DAPI}{4’,6-diamidino‐2-phenylindole}
\newacronym{dc}{DC}{dendritic cell}
\newacronym{eb}{EB}{Epstein Barr virus}
\newacronym{ent}{ENT}{ear- nose- and throat,  otorhinolaryngology}
\newacronym{fas}{Fas}{Fas cell surface death receptor (CD95)} % ligand; Fas -> tumor necrosis factor (TNF) family; 
\newacronym{fasl}{FasL}{Fas ligand (CD95L or CD178)} % FasL = CD95L or CD178; 
\newacronym{gphase}{G-phase}{gap phases in mitosis}
\newacronym{gvhd}{GVHD}{graft versus host disease}
\newacronym{hiv}{HIV}{human immunodeficiency virus}
\newacronym{hla}{HLA}{human leukocyte antigen}
%\newacronym{hpv}{HPV}{human papilloma virus}
\newacronym{icd}{ICD}{international classification of diseases}
\newacronym{il}{IL}{interleukin}
\newacronym{lc}{LC}{langerhans cell}
\newacronym{lp}{LP}{lichen planus}
\newacronym{lpl}{LPL}{leukoplakia}
\newacronym{lpldys}{LPL-dys}{leukoplakia with dysplasia but without malignant transformation}
\newacronym{lplca}{LPL‐ca}{leukoplakia with dysplasia with malignant transformation}
\newacronym{mab}{mAb}{monoclonal antibody}
\newacronym{mdsc}{MDSC}{myelo-derived suppressor cells}
\newacronym{mhc}{MHC}{major histocompatibility complex}
%\newacronym{mmp}{MMP}{matrix metalloproteinases}
\newacronym{nkg2d}{NKG2D}{natural killer group 2 member D}
%  MICA MICB is the ligand of NKG2D
\newacronym{mica}{MICA}{MHC-class-I-polypeptide-related sequence A}
\newacronym{micb}{MICB}{MHC-class-I-polypeptide-related sequence B}

\newacronym{nsaid}{NSAID}{non-steroidal anti-inflammatory drugs}
\newacronym{olp}{OLP}{oral lichen planus}
\newacronym{oscc}{OSCC}{oral squamous cell carcinoma}
\newacronym{pdl1}{PD‐L1}{programmed death-ligand 1 (CD274)}
\newacronym{pmod}{PMOD}{potentially malignant oral disorder}
\newacronym{ptld}{PTLD}{post‐transplant lymphoproliferative disorder}
\newacronym{pvl}{PVL}{proliferative verrucous leukoplakia}
\newacronym{sir}{SIR}{standard incidence ratio}
\newacronym{sot}{SOT}{solid organ transplantation}
\newacronym{taa}{TAA}{tumor associated antigen}
\newacronym{tam}{TAM}{tumor associated macrophage}
\newacronym{tcr}{TCR}{T cell receptor}
%\newacronym{til}{TIL}{tumor infiltrating lymphocyte}
\newacronym{tgf}{TGF}{transforming growth factor}
\newacronym{th}{Th}{T helper}
\newacronym{tls}{TLS}{tertiary lymphoid structure}
%\newacronym{tnm}{TNM}{tumor, node, metastasis classification system}
\newacronym{trail}{TRAIL}{tumor necrosis factor-related apoptosis‐inducing ligand}
\newacronym{treg}{Treg}{regulatory T cell}

\newacronym{aldh2}{ALDH2}{aldehyde dehydrogenase 2}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%---------------------------------
%
%. Document
%
% -----------------------------------------------------------------------------------------------------------------

\usepackage[utf8]{inputenc}

\usepackage{pdfpages}


\newcommand{\comm}[1]{} % for comment it out by \comm{} https://tex.stackexchange.com/questions/17816/commenting-out-large-sections

% MOST template from https://github.com/kuochuanpan/most_proposal/blob/master/most_proposal.tex
\begin{document}
\thispagestyle{titlepage}
% MOST page; cover page C001
\includegraphics[trim=1.5cm 0 0 3cm]{111年度深耕轉譯創新研究計畫計畫書_2022版_template (sp3_title).pdf}
%\includepdf[pages=-,pagecommand={},width=\textwidth]{111年度深耕轉譯創新研究計畫計畫書_2022版_template (sp3_title).pdf}
%MOST_proposal_DeepLearning_HNSCC-C001.pdf}


%\includegraphics[width=18cm]{MOST_proposal_DeepLearning_HNSCC-C001.pdf}

\thispagestyle{titlepage}

\clearpage
% \cntext{TMU 口腔醫學院 2021 project}
% The real title page
%%%%%%%%%%%%%%%%%%%%%%%%%% title
\title{\Large \vspace{-2.5cm} \titlecap{Deep learning platform and risk prediction models for malignant transformation of oral verrucous hyperplasia}}
%Deep learning for whole-slide images and holistic} Electric Healthcare Records in Head and Neck Squamous Cell Carcinoma}
% or Deep learning for whole slide image and holistic EHR
% Deep Learning for Whole-slide Images and Electric Healthcare Records in Head and Neck Squamous Cell Carcinoma
%\author{Li-Hsing Chi}
%\date{May 2021}

\author{\small Co-PI: Li-Hsing Chi (\cntext{祁力行})$^{1,2}$\\
{\footnotesize $^{1}$ \quad Division of Oral and Maxillofacial Surgery, Department of Dentistry\unskip, 
%    Taipei Municipal Wanfang Hospital\unskip, Taipei\unskip, Taiwan\\
Wan Fang Hospital\unskip, Taipei Medical University}\\
    
{\footnotesize $^{2}$ \quad Division of Oral and Maxillofacial Surgery, Department of Dentistry\unskip,
    Taipei Medical University Hospital\unskip, Taipei Medical University}\\
    
%$^{4}$ \quad Genomics Research Center\unskip, Academia Sinica\unskip, Taipei\unskip, Taiwan\\
    
%$^{5}$ \quad Graduate Institute of Biomedical Informatics, College of Medical Science and Technology\unskip, Taipei Medical University\unskip, No.172-1, Sec. 2, Keelung Rd.\unskip, Taipei 106\unskip, Taiwan\\

%Taipei\unskip, Taiwan\\
%{\footnotesize $^1$ Department of Physics and Astronomy, Michigan State University, East Lansing, MI 48824, USA}\\
%{\footnotesize $^2$Institute of Astronomy, Nation Tsing Hua University, Hsinchu 30013, Taiwan}\\ 
%{\footnotesize $^3$Department of Physics, Nation Tsing Hua University, Hsinchu 30013, Taiwan}
}
\date{\small \today}

%Sub-project 3:\\
\maketitle

\thispagestyle{titlepage}

% ------------------------------------------------------
%
% Abstract
%
% ------------------------------------------------------
%\section{Abstract} % for navigation

\begin{abstract} %not anymore (as sub project)
\cntext{二、研究計畫中英文摘要:}

\textbf{Background:}
Head and neck squamous cell carcinoma (\acrshort{hnscc}) represents a significant health concern worldwide. Surgery and systemic therapy are still the standard of care for \acrshort{hnscc} patients. Oral squamous cell carcinoma (\acrshort{oscc}) is the most common part of \acrshort{hnscc}. The break-through improvement of those interventions should depend on 1) the discovery of high impact prognostic biomarkers, and 2) early-detection of potentially malignant oral lesion.
%proper treatment planning by accurate sub-population of those patients.


%According to the National Comprehensive Cancer Network (NCCN) guidelines (Version 3.2021~\citep{Pfister2021}), the current treatment of \acrfull{hnscc} should include of surgery alone, radiation therapy alone, or a combination of the two with adjuvant chemotherapy.
%Despite advancements in these therapies, the five-year survival rate for this disease has only marginally improved over the previous decade, and recurrent disease is seen in up to 50\% of all patients~\citep{Forastiere2001,Warnakulasuriya2009}.
%During 2008 to 2018 in Taiwan, the one-year survival rate for this disease among males is from 79.56\% to 81.62\%~(statistics from Cancer Registry of \acrlong{Thpa}, available at \url{https://cris.hpa.gov.tw/pagepub/Home.aspx}, accessed on May 2022). The five-year survival rate for this disease among males ranged from 55.13 percent to 56.03 percent between 2009 and 2014.

The survival analysis of the \acrfull{tcga} dataset is a well-known method to discover biomarkers of \acrshort{hnscc}. 
Biomarkers of cancer have been utilized for diagnostic, predictive or prognostic purposes. 
A predictive marker helps to select which specific treatment benefits a better survival.
For example, adjuvant radiation therapy should be done after tumor-ablative surgery in positive surgical margin or metastatic lymph node from the neck dissection. %; it  to select a particular treatment over another.
%It guides to determine which further treatment (e.g., ).

Clinically visible oral potentially malignant disorders (OPMDs) precede the most of \acrshort{oscc} cases. Oral submucosal fibrosis (OSF), oral verrucous hyperplasia (OVH), and oral lichen planus (OLP) are among the OPMDs, with OVH holds the greatest risk of malignant transformation. 
The key clinical challenge is how to properly care patients with OVH for oral cancer prevention and early identification. One important strategy for dealing with this clinical situation should be the establishment of risk prediction models for malignant transformation.
Oral verrucous hyperplasia (\acrshort{ovh}) is a pre-malignant  mucosal lesion with a predominantly verrucous or papillary surface; this lesion can subsequently transform into verrucous carcinoma (VC) or squamous cell carcinoma (SCC).
A predictive biomarker for \acrshort{ovh} could provide early warning before malignant transformation without the need for frequent biopsies.
%OPMDs include lesions or conditions that involve morphological, color, and consistency changes in the oral mucosa and indicate the risk of malignant progression. 
%Although visual screening, tactile palpation, tissue biopsy or optical diagnostic adjuncts have been used for definite diagnosis, it is still  a challenge for clinicians to discriminate against the malignant potentials of OPMDs. 


%A prognostic marker infers the natural history of disease (survival), which is independent on treatment of choice. Gene-expression based biomarker can indicate a need for further treatment, but does not help to determine which treatment. 
	

%It is necessary to determine a cutoff point of the patients' dichotomization in survival analysis with the continuous gene expression measurements. There is some optimization software for cutoff mining.
%However, the software's determination of cutoffs is usually set at the median, 1/4 quantile, or 3/4 quantile of \acrfull{rnaseq} value to find a significant \textit{P} value by the Kaplan-Meier analysis. There are few clinicopathological or spiritual features available on their pre-processed data sets.
%The survival analysis could conclude the manifest of clinical, pathological, and transcriptomics data to find the prognostic biomarkers.

%Methods: how the study was performed and statistical tests used
\textbf{Methods:}
The identification of risk factors is used to construct risk prediction models. The likelihood of malignant transformation in OVH patients may be influenced by genetic variations and/or gene-environment interactions. There are not too many similar studies so far, and they are candidate gene studies. Using a microarray-based chip to acquire genotypic data for a large number of genetic variants across the human genome has been cost-effective. Genome-wide association testing is a simple and effective way to find disease-associated genetic variants that is not even constrained by current knowledge of the disorder.

R script has also been used to develop a comprehensive workflow, named "pvalueTex", running on the Rstudio platform. It includes data retrieving and pre-processing, feature selection, cutoff mining engine, Kaplan-Meier survival analysis, Cox proportional hazard modeling to discover prognostic biomarkers.
We further plan to develop a deep-learning based data mining as well as validation platform. It might incorporate whole-slide images of pathology from cohort of Taipei Medical University (TMU) and Chi-Mei Hospital.
%the Cancer Imaging Archive (TCIA) and Taipei Medical University (TMU) biobank.
%RNA-Seq data of \acrfull{tcga} for survival analysis. 
Convolutional neural network (CNN) will be used for feature extraction from whole-slide images.
Graph convolutional neural network (GCNN) and explainable autoencoder will deal with "survival analysis". 
%Recurrent neural network (RNN) will deal with "holistic features" from electric healthcare records (EHR) of Taipei Medical University Hospital (TMUH).
These patient's features should include physical, pathological, psychological data, and even more spiritual information investigated by physicians.
During their follow-up period, a "survival analysis" should also be used to build a predictive model of malignant transformation. 
% construction of gene expression network.
%GNN graph neural network for CNN convolutional neural network
%The Cancer Imaging Archive (TCIA) whole-slide by Deep learning (ResNet) 
% \acrfull{tcia} and TCGA
% *** AI: deep learning, machine learning \citep{Huang2020}
%plot http://iltabiai.github.io/tips/latex/2015/09/15/latex-tikzdevice-r.html
%***Deep learning-based cancer survival prognosis from RNA-Seq data: approaches and evaluations
%*transfer learning from TCGA model
%torchvision, NanoMets Models
%save checkpoint, load then deepcopy()




%\thispagestyle{titlepage}
% Conclusions: brief summary and potential implications
\textbf{Conclusions:}
Based on the risk indicators gathered from all subprojects, this subproject will construct risk prediction models for malignant transformation in OVH patients. The established risk prediction models should make clinical care of patients with OVH for \acrshort{oscc} detection and control screening efficient.
%Those 11 validated biomarkers (DKK1, CAMK2N1, STC2, PGK1, SURF4, NDFIP1, STIP1, DKC1, ZNF557, ZNF266, and FCGBP), 
%The candidate biomarkers, clinical tumor size, and surgical margin status are suggested prognosis factors in \acrshort{hnscc} through TCGA analysis.
%Using those biomarkers, we will analyze the transcriptomic/mutational representatives of \acrshort{ovh} in our cohorts. 
%could be applied to validate those candidates by independent TMU cohort (n=46, obtained at TMU Joint Biobank).
%We will conclude that combining genomics data with pathology images will aid in the development of a predictive model for early detection of \acrshort{oscc}.
% pathology images (by NCI)
% pvalueTex, equipped with an optimal cutoff finder, 

%Survival analysis using the Cancer Genome Atlas (TCGA) dataset is a well-known method to discover gene expression-based prognostic biomarkers of head and neck squamous cell carcinoma (HNSCC). It is necessary to determine a cutoff point by patients' dichotomization for the continuous gene expression. Usually, an optimization algorithm for cutoff determination has been set at the median or quantiles of RNA sequencing value.
%There are few clinicopathological features available on those pre-processed datasets. (feature selection or feature engineering issue)
% validated by the other HNSCC cohort:
%1) poor prognosis: CAMK2N1 (0.048214), PGK1 (0.009978), SURF4 (0.023127), USP10 (0.017768), NDFIP1 (0.022758), FOXA2 (0.001587);\\ % FOXA2 (0.038125)
%2) better prognosis: IL19 (0.049731), FCGBP (0.005658), KIAA1683 (IQCN, 0.005886), NPB (0.014177);\\

% row 20: The result showed ten overexpressed genes (symbol as DKK1, CAMK2N1, STC2, PGK1, SURF4, USP10, NDFIP1, FOXA2, STIP1, and DKC1) are significantly associated with a poor prognosis of overall survival. Furthermore, the other ten overexpressed genes (symbol as ZNF557, ZNF266, IL19, MYO1H, FCGBP, LOC148709, EVPLL, PNMA5, IQCN - former symbol as KIAA1683, and NPB) are correlated with better survival.


% Keywords
\textbf{Keyword:}
%\keyword{
\titlecap{genome-wide association study}; 
\titlecap{malignant transformation};
\titlecap{oral potentially malignant disorders}; 
\titlecap{risk prediction model};
Head and Neck Squamous Cell Carcinoma (HNSCC);
Oral Squamous Cell Carcinoma (OSCC);
Oral Verrucous Hyperplasia (\acrshort{ovh});
%Genome Database\sep
the Cancer Genome Atlas (TCGA);
the Cancer Imaging Archive (TCIA);
RNA-sequencing;
Survival Analysis;
%Optimal Cutoff;
%Biomarker Discovery; %\sep
%Tumor Type-agnostic Therapy\sep
%Immuno-Oncology\sep
%Targeted Therapy\sep
%Systemic Therapy\sep
Surgical Margin;
Deep Learning; Explainable;
Graph Neural Network (GNN);
Graph Convolutional Neural Network (GCNN);
%Recurrent Neural Network (RNN);
%Electric Healthcare Records (EHR);
Holistic Cancer Care.
%Therapeutic Relationship;
%Mindfulness Meditation\\[1cm]
%}

\clearpage

\cntext{中文摘要:}

\begin{CJK*}{UTF8}{bkai}
%頭頸部鱗狀細胞癌數位病理影像與全人照護電子病歷的深度學習
以深度學習及風險預測模式探討口腔疣狀增生惡轉風險\\[0.3cm]

頭頸部鱗狀細胞癌 (\acrfull{hnscc}) 在全世界都是重要的問題。手術、放射治療和全身性治療(化療、免疫)仍然是這些患者的標準治療方式。如何減少死亡率，取決於有效的預後生物標記(創新療法)，並早期預警口腔黏膜病變的癌化(加強預防)。

%自2008年至2018年，臺灣男性罹患頭頸部鱗狀細胞癌的一年存活率為79.56\% 至81.62\%（健康促進管理局癌症登記處的統計數據，網址https://cris.hpa.gov.tw/pagepub/Home.aspx，2022年5月查詢）。2009年到2014年，男性患者的的五年存活率一直沒有進步(從55.13\%到56.03\%)。

利用The Cancer Genome Atlas (\acrshort{tcga})資料庫做存活分析，是找尋頭頸部鱗狀細胞癌生物標記最常用的方法。癌症的生物標記已被用於診斷、預測病情與疾病的預後。預測性標記有助於選擇有利的治療方式。例如，腫瘤切除手術後，若是手術切緣不足、頸部淋巴結轉移，則應進行術後的放射治療，以增加治療的成功率。

大多數口腔癌病例在罹癌之前都有臨床上明顯的口腔潛在惡性疾病 (oral potentially malignant disorders, OPMDs)。OPMDs包括白斑、紅斑、口腔黏膜下纖維化、口腔疣狀增生（oral verrucous hyperplasia, OVH）和口腔扁平苔蘚，其中OVH惡轉風險最高。
口腔疣狀增生(\acrfull{ovh})是一種外形為疣狀(或乳頭狀)的癌前病變；這種病變日後有機會轉化為疣狀癌 (verrucous carcinoma, VC)或鱗狀細胞癌(squamous cell carcinoma, SCC)。
我們的目標是找出預測性生物標記，期望無需頻繁切片，就能早期預警正在癌化中的口腔疣狀增生。

風險因子的確認是開發風險預測模式的基礎。OVH患者惡轉風險可能會受到遺傳變異或基因-環境交互作用的影響。然而，到目前為止，相關的研究並不多，且均為候選基因研究。目前，使用以微陣列技術為基礎的晶片，獲取人類基因組中大量遺傳變異的基因型資料，是具高度成本效益的方法。以所得全基因組基因型資料進行全基因組關聯研究是尋找與疾病相關的遺傳變異的有效方法，且此研究方法不受限於當前對該疾病的理解。

我們亦使用R語言開發"pvalueTex"程式，可執行資料檢索、資料處理、特徵選擇、資料探勘、Kaplan--Meier存活分析、Cox比例風險模型，以用來大規模搜尋生物標記。我們接下來計劃開發深度學習(Deep Learning)資料探勘和驗證平台，利用數位病理影像(\acrfull{tcia}, 與TCGA串連)，及臺北醫學大學(TMU)生物資料庫中，口腔疣狀增生的病理切片數位影像，以AI人工智慧演算法，來做存活分析，預測(癌)惡轉變化。
%它包含來自癌症影像檔案 (TCIA) 和臺北醫學大學 (TMU) 生物資料庫的病理學全幻燈片圖像。
%\acrfull{tcga} 的 %RNA-Seq 數據用於生存分析。
深度學習的卷積神經網路(\acrfull{cnn})可由數位病理影像中提取特徵。
圓論卷積神經網路(\acrfull{gcnn})則處理電子病歷的"全人照護特徵"。
這些患者的特徵應該包括身體、病理、心理狀態，社會關係，甚至全人照顧醫師所紀錄的靈性紀錄。\\
結論：
%那些 11 個經過驗證的生物標誌物（DKK1、CAMK2N1、STC2、PGK1、SURF4、NDFIP1、STIP1、DKC1、ZNF557、ZNF266 和 FCGBP），
%通過TCGA分析，我們得到生物標記，並確認臨床腫瘤大小和手術切緣狀態，都會是頭頸部鱗狀細胞癌患者的重要預後因素。使用這些生物標誌物，我們將分析口腔疣狀增生檢體的基因變化，結合數位病理影像，以開發鱗狀細胞癌的早期預警模型。
以各子計畫所找到的風險因子為基礎，本子計畫將開發OVH患者惡轉風險預測模型。我們預期所開發的風險預測模型將有助於OVH患者的臨床管理，以預防和早期發現口腔癌。
%而進一步分析\acrshort{tcia}和TMU的癌細胞基因表現量、數位病理影像和全人照護特徵，將有助於驗證候選基因對預後的影響，以發展未來頭頸部鱗狀細胞癌的精準治療。

關鍵詞：口腔潛在惡性疾病;全基因組關聯研究;風險預測模式;惡轉; 頭頸部鱗狀細胞癌; 口腔鱗狀細胞癌; 口腔疣狀增生; 存活分析; 手術切緣; 手術切緣; 卷積神經網路; 圓論卷積神經網路; 全人照顧。

\end{CJK*}

\end{abstract}

\clearpage




%\thispagestyle{titlepage}
% ------------------------------------------------------
%
% Introduction
% https://www.most.gov.tw/most/attachments/8eb87bfc-502c-4da8-b026-f9a499ae1622
% http://lgdntu.blogspot.com/2015/01/blog-post.html 專訪張典顯與楊慕華老師
% writing a fundable grant:
% critically significant, how do I approach this (解釋得很清楚，不可以草率，或僅止於做流水帳),  is there any 替代策略 (fallback approaches)
% just like 一篇 manuscript (手稿) 適合投到哪個 journal (期刊) 需要仔細審度
% being concise and precise (NIH: maximal 12 pages)
% ------------------------------------------------------
%「研究計畫之背景及目的: 詳述研究計畫之背景、目的、重要性及國內外有關本計畫之研究情況、重要參 考文獻之評述等」、
%「研究方法、進行步驟及執行進度」、
%「預期完成之工作項目 及成果」、
%「經費」
%「對社會的貢獻」
%\section{Introduction}」

\cntext{三、研究計畫內容:}

\section*{\cntext{(一)研究計畫之背景及目的}} % 大約 3 頁 Background and Significance
%請詳述本研究計畫之背景、目的 具體目標 (Specific Aims)、重要性及國內外有關本計畫之研究情況、重要參考文獻之評述等。

% *** 講一個故事，先告訴大家領域的重要性和重大發現，然後再告訴大家什麼重要的問題尚待解決，而我所提的研究方案能提供什麼樣的解答，這就是 "rationale"
%\begin{outline}
%\1 Big picture.
%\1 Specific and important problems.
%\2 How much has been done and how much hasn’t?
%\2 How am I able to contribute solving the important and unsolved questions?
%\end{outline}
% * 研究背景的介紹需具備邏輯連貫性，且要與研究主軸相關聯。最佳的寫作方式是利用背景介紹點出尚未解決之問題，同時引出研究之重要議題。
% *長期目標願景 (long-term goal) 說清楚，包含哪些是領域當中重要但未解決的議題，以及個人的研究主題和此有何關聯
% then 將個人的研究主題和其它重要的主題連結 (to attract your reviewers)

% 文獻回顧及評述: review articles then discussion => 有助於說明計畫的重要性、創新性 
%*** 應就其中重要、關鍵的論文提出與之論辯、對話的過程，歸納出過去研究中值得參考的成果及論點，有哪些值得檢討、翻盤的盲點，以及迄今未獲解決或被忽略的面向，乃至前人研究的錯誤。



%\chapter{Introduction}
\label{chap:intro}
%\chaptermark{Epidemiology, Etiology, Pathogenesis, and Treatment of HNSCC}

%Introduction
%Background of the Problem
%Statement of the Problem
%Purpose of the Study
%Research Questions
%Significance of the Study
%Definition of Terms
%Assumptions, Limitations, and Delimitations
%Conclusion

\section{Background}


%GLUT4 \citep{Chang2017b}
%lactate



%===================
%http://www.biomedicine.org.tw/Upload/07_DNA甲基化程度在大腸直腸癌扮演的角色.pdf
%脊椎動物之基因體中約20\%之序列為5'-GC-3' (簡稱CpG)之雙核苷酸(dinucleotide)。然而有 些長約1至2kb之序列段，其CpG含量高達60% (CpG islands)
%Epigenetic trinity in cancer: CpG islands
%60\% protein-coding genes with CpG islands on their promoter region
%(
%McGough JM, Yang D, Huang S, et al. DNA methylation represses IFN-gamma-induced and signal transducer and activator of transcription 1-mediated IFN regulatory factor 8 activation in colon carcinoma cells. Mol Cancer Res 2008;6:1841-1851.
%)
%methyl CpG-binding domain protein; MBD prevents TF binding on the promoter of gene
%=> causing impairment of mismatch repair: MMR => microsatellite instable: MSI

%環境因子的致癌機轉
%=>CpG hypermethylation to turn off tumor suppressor genes
%=>CpG Hypomethylation turn on oncogene by such as DKK1

%loss of gene imprinting <- hypomethylation
%alleles 

%histone methylation

%http://webcache.googleusercontent.com/search?q=cache:z8zvnjS-C3AJ:sites.mc.ntu.edu.tw/board.php?courseID%3D83%26f%3Ddoc%26folderID%3D581%26cid%3D3060&client=safari&hl=en&gl=tw&strip=1&vwsrc=0
%雖然以上我們討論的是癌症的基因體，但癌症的厲害就是它本來就利用人體原有的生物機制來生存（co-opting），因此，以上在癌症觀察到的基因體複雜性，其實也存在於正常人體.

%從最早，很單純的基因之exon序列，轉錄成mRNA，再轉譯成polypeptide，形成protein，到現在還要考慮epigenetics、mi RNA、TUF (ENCODE將之名為transcripts of unknown function (TUF)=> non-coding RNA 後來很紅)、chromosome environment，基因體的複雜性真是增加了不知多少倍，這也提醒我們在了解基因功能方面，必需更有全面性的關照。這也是「基因體醫學」必需以「基因體科學」為基礎的道理。





\subsection{Cancer Initiation}
The main risk factors for \acrshort{oscc} include carcinogens in tobacco and betel nut, alcohol, oncogenic viruses.
The betel-nut chewing, alcohol \acrfull{aldh2} mutation should be the major concern as a variety of \acrshort{oscc} in Taiwan.
In Asian population, \acrfull{aldh2} mutation plays a very important rule to enhance \acrshort{oscc}~\citep{Chien2019} by alcohol consumption.
Wu and colleagues~\citep{Wu2018a} found that non-buccal/non-tongue OVH are especially at risk of malignant transformation in a observatory cohort study. 
The oncogenic viruses such as \acrfull{hpv} and Epstein–Barr virus play a major carcinogenic role in tumors of the oropharynx and nasopharynx, respectively~\citep{Ferrarotto2017}.
Moreover, physical irritations, ionized radiation and host susceptibility may also contribute to the etiology of \acrshort{oscc}~\citep{Ko1995,Znaori2003} (Figure~\ref{fig:DAG_etiology}).
Primary prevention by minimizing exposure to known carcinogens is promising, but limited success is achieved due to addiction, economic, social or spiritual issues.
The possible molecular mechanisms could be also related with the mind-brain-body axis~\citep{Berens2017}.

Clinically visible oral potentially malignant disorders (OPMDs) precede the most of \acrshort{oscc} cases. Oral submucosal fibrosis (OSF), oral verrucous hyperplasia (OVH), and oral lichen planus (OLP) are among the OPMDs, with OVH holds the greatest risk of malignant transformation. 
The malignant transformation rate in the OVH cohort was 24.55 per 1000 person-years, according to a recent study based on a nationwide oral cancer screening program in Taiwan~\citep{Chiu2021}.
% This code uses the tikz package
%\begin{frame}[fragile]
%\frametitle{Directed acyclic graphs for %\acrshort{hnscc}}
\begin{figure}
\centering
\begin{tikzpicture}[scale=3.5, every node/.style={scale=1.3}] %, transform shape]
\node (v0) at (-1.49,1.64) {Age/Gender};
\node (v1) at (0.705,1.42) {Alcohol/Betel/Cigarette};
\node (v2) at (-1.18,-1.38) {Healthy Food/Exercise};
\node (v3) at (-0.167,0.515) {Other Risky Behavior};
\node (v4) at (1.19,-1.52) {\large \acrshort{oscc}};

\draw [-latexnew, arrowhead=0.5cm, line width=4pt, ultra thick, dashed] (v0) .. controls (-0.408,1.72) and (0.325,1.64) .. (v1);
\draw [-latexnew, arrowhead=1cm, line width=4pt, ultra thick, green, dashed] (v0) edge (v2);
\draw [-latexnew, arrowhead=0.5cm, line width=4pt, ultra thick, dashed] (v0) .. controls (-0.761,1.13) and (-0.318,0.754) .. (v3);
\draw [-latexnew, arrowhead=0.5cm, line width=4pt, ultra thick, red] (v0) .. controls (-0.852,0.234) and (0.0430,-0.821) .. (v4);
\draw [-latexnew, arrowhead=2cm, line width=4pt, ultra thick, red] (v1) .. controls (1.30,0.923) and (1.46,-0.0573) .. (v4);
%  \draw [options/.expand once=\x] (0,-1.2*\i) -- (3,-1.2*\i) node [right] {\x};
\draw [-latexnew, arrowhead=1cm, line width=0.4cm, ultra thick, green] (v2) edge (v4);
\draw [-latexnew, arrowhead=1cm, line width=4pt, ultra thick, red] (v3) edge (v4);
\end{tikzpicture}

\bcaption{OSCC will be caused by carcinogen or other factors.}
{
From the intrinsic factors (such as age and gender), the accumulated genetic alteration will be part of etiology of cancer. Carcinogen from alcohol, betel nut and/or cigarette will definitely induce \acrshort{oscc} by time passes. People usually learn "risky" or "healthy" behaviors by many reasons, which modify their possibility to get cancer.\\
(\textcolor{red}{red line}: increasing the chance of getting \acrshort{oscc}; \textcolor{green}{green line} will reduce it; black dot: indicate unknown reasons for taking those habits)
}
\label{fig:DAG_etiology}
\end{figure}



%\paragraph{Ancient History of \acrshort{hnscc}}
% \citep{Fornaciari2012}
% \citep{Rehemtulla2010}
%Inchingolo, F., Santacroce, L., Ballini, A., Topi, S., Dipalma, G., Haxhirexha, K., Bottalico, L., \& Charitos, I. A. (2020). Oral cancer: A historical review. In International Journal of Environmental Research and Public Health (Vol. 17, Issue 9). MDPI AG. https://doi.org/10.3390/ijerph17093168 \citep{Inchingolo2020}




%Overview of HNSCC (Indian)
%https://www.intechopen.com/books/oral-cancer/oral-cancer-an-overview

% ALDH2 mutaion 戒酒有理 05/09
% 在美國北加州參加演講。
% 很棒！心得是：該戒酒了，至少要去檢測 ALDH2 基因是否突變影響解酒酵素→ 因為乙醛堆積致癌！
% 菸與酒在全身癌症的貢獻度是可以相比的。只有壞處沒有好處
% http://taies.org/
% 喝酒臉紅（酒精不耐症）表示危險，因為代謝酒精的酵素不正常，身體容易大量累積乙醛，這是一級致癌物。
% 尤其是，如果酒精不耐者加上吸菸，口腔癌罹患機會，是一般人的400 倍。
% 酒駕更會製造社會問題。
% 
% #鼓勵檢測 ALDH2 基因
% #建議酒精不耐者戒酒
% #戒菸戒酒同樣重要
% #適量紅酒對亞洲民族的心血管完全沒有好處
% #亞洲人一天最多10克酒精相當於350mL鋁罐啤酒
% 
% 專題演講 : Alcohol Intolerance: An Under-Recognized Health Risk and Precision Oral Cancer
% 演 講 者: Che-Hong Chen, PhD.
% 陳哲宏
% Alcohol lntolerance Education Society
% 台灣酒精不耐淀衛教協會
% LINE ID: jchehong
% Email: chehong@stanford.edu
% STANFORD
% SCHOOL OF MEDICINE
% Stanforl Cuirersity Medical Center
% Chemical and Systems Biology
% 歡迎捐款!協會慈善捐款收據備索
% 郵政劃撥劃撥帳號:50434421
% http://taies.org/
% 戶名:社團法人台灣酒精不耐症衛教協會
% 謝謝陳老師的演講，我是提問的 北醫 祁力行。
% 戒菸戒酒都能使用藥物幫忙，加上專業衛教，建立動機、鞏固決心，我還知道一些中醫穴道，雷射針灸可以幫忙戒除之，多管齊下。
% Acetaldehyde
% 老師提到戒酒已有藥物，Disulfiram 正是抑制 ALDH1，讓乙醯醛堆積，酒後症狀嚴重。:-)


%% http://web.tccf.org.tw/lifetype/index.php?op=ViewArticle&articleId=4480&blogId=1 振興醫院血液腫瘤科 陳國維醫師


%慈悲心 利他心
%博士班學生，越研究越發現，科學的不足，心理與靈性的力量，更是背後推動的冥冥力量，身心靈是合一的
%#正念 除了可以減壓，還可以治癒疼痛
% modify the dark environment 改造患者的心，就能改造他的身體環境，有利消除癌症


%we educate them
%1. Mong, L.-C., Liao, F., Chiou, J. & Chiang, P. Tips for Integrating Spiritual Care Delivery in Dental Education. in Asia Pacific Medical Education Conference. Short Communication 5 (National University of Singapore, 2021).\citep{Ling-ChengMong2021}

% 目的與功效  出入息 呼吸念頭 回到覺知，不再被潛意識自動導航干擾情緒
%正念，放下二元評價 正念覺知 XAI saliency mapping and trustworthiness in deep learning
%覺知 漸入空性 Y=wX + bias
%解決情緒苦(七情) 四聖諦
%慈悲心 安心（情緒穩定）
%了卻見人苦的「慈悲心」易感者同理心 empathy
%道歉不是一段文字，不是一個儀式（療癒冤親債主來自慈悲心）
%冤親債主 決定不再影響環境 基因 expression，減少意外與疾病、癌症

% 自動導航（潛意識）易受情緒影響，造成焦慮、發炎、免疫 Cancer
%Mindfulness based stress reduction provides psychological benefit and restores immune function of women newly diagnosed with breast cancer: A randomized trial with active control.
%BACKGROUND: Women newly diagnosed with breast cancer experience psychological distress, accompanied by reduced Natural Killer Cell Activity (NKCA) and altered levels of cytokines, which may compromise cancer control. Few studies have evaluated psycho-immune outcomes of mindfulness-based stress reduction (MBSR) for women newly diagnosed with breast cancer in comparison to an active control condition. OBJECTIVE: The purpose of this study was to determine whether MBSR benefits psychological, behavioral, and immunological function in women recently diagnosed with breast cancer. DESIGN: After confirmation of breast cancer staging, women diagnosed with early-stage breast cancer (n = 192) were randomized to an 8-week MBSR program or an 8-week active control condition (ACC). The ACC consisted of a series of cancer recovery and health education classes. Both MBSR and the ACC were administered in group format. METHODS: Women completed psychometric instruments and provided blood for NKCA and cytokine levels at pre-, mid-, and completion of program, as well as at 1- and 6-months post-program. One hundred and twenty four women completed all five-assessments (MBSR, n = 63; ACC, n = 61). Hierarchical linear modeling was used to analyze trajectories of outcomes over time and between groups. 
%RESULTS: Compared to the ACC group, women randomized to MBSR exhibited decreasing trajectories of perceived stress, fatigue, sleep disturbance, and depressive symptoms. 
%Further, compared to women randomized to ACC, MBSR women exhibited trajectories demonstrating significantly more rapid restoration of NKCA, accompanied by 
%lower circulating TNF-alpha levels, lower IL-6 production =>  pro-inflammatory cytokines
%and greater IFN-gamma production => IFN gamma supports an immune-mediated response against cancer and this may be important during the transition following cancer treatment
%A randomized clinical trial with active control proved that early provision of MBSR for women newly diagnosed with breast cancer provides psychological benefits (reduction of stress, fatigue, sleep disturbance, and depressive symptoms), and also optimizes immune functions (higher natural killer cell activity, higher \acrshort{ifng}, lower TNF-alpha, lower IL-6) supportive of cancer survival\citep{WitekJanusek2019}.

% IL-6 in Covid-19 infection 


%%%% PCA and ECA: the way to delivery of MBSR
%The therapeutic relationship (TR) is essential towards spiritual care.
%According to clinical experience in healthcare, sometimes patients could tell the physicians: "How amazing it is. I want to talk about myself more when I visit this doctor whom I trust."
%In 1979, Carl Rogers wrote\citep{Rogers1979}: "I find that when I am closest to my inner, intuitive self, when I am somehow in touch with the unknown in me, when perhaps I am in a slightly altered state of consciousness, then whatever I do seems to be full of healing. Then simply my presence is releasing and helpful."

%empowering
%(一)雙方有心理的接觸。
%(二)當事人正感到焦慮或不一致(incongruence)。 
%(三)治療者在治療關係中是一致的 (congruence, agreement or harmony; compatibility)、真實的或真誠的。
%(四)治療者能夠對當事人表現出無條件的關懷。無條件關懷(不批判/頓悟)
%(五)治療者能夠同理的瞭解當事人的內在，並努力傳達「我懂你的」。
%(六) 第五點只要有個適當的開始，療癒就會開始
%Dealing with negative emotions. 情緒影響，造成焦慮、發炎、免疫 Cancer
%As the cancer patient feels intense negative emotions of extreme pleasure, anger, overthinking, sadness, fear, shock, anxiety, 
%七情: 喜、怒、思、悲、恐、驚、憂
%depression, fatigue, stress, it is not the patient-centred therapist’s task to remove these feelings: the task is to stay with the patient’s experiences. 
% 只是陪伴
%The research implication is to investigate the necessity and sufficiency of the six conditions.
%Rogers, C. R. A Theory of Therapy, Personality, and Interpersonal Relationships: As Developed in the Client-centered Framework. in Psychology: A Study of a Science. Study 1, Volume 3: Formulations of the Person and the Social Context (ed. Koch, S.) 184–256 (McGraw Hill, 1959).\citep{Rogers1959}
%PCA理論 by *** Tex Chi
%治療關係的六核心條件: 說好話 眼對眼 engaging 用真心
%以心印心 engaging
%身心相連 body-mind connection
%捨末求本 find the source: emotion

%身心靈全人醫療，展現魅力、慈悲心，靈性成熟的醫師，用藥物或手術來治療病患時，「他自己本身」就具有療癒效果
%「把自己當藥方開出去」
%Spirituality
%慈悲心 compassion 
%同情心 sympathy
%同理心 empathy
%non-verbal contact


%Personhood: Carl Rogers
% ***尋找生命的意義 meaning of life
%真誠、能深刻地善解人意
%非認知性核心能力
%正念止觀(在當下) mindfulness meditation
%人人有自癒能力 resilience
%真誠 (Genuineness/Validity/Bona Fides) 善意?
%內在聲音與外在態度一致/明心見性/超越的意識狀態 (Congruence/Consistency)
%共情理解(設身處地/同理心/除我執 自他不二) (Empathic Understanding of Reference/Track)
%無條件關懷(不批判/頓悟) (unconditional positive regard) 只是陪伴
%***引導出案主的無條件自我關懷 => 療癒/復原力（resilience）by 諮商員的靈性功力
%治療關係: 
%Tex Chi's Therapeutic relationship: patient said "I would like to talk about myself more when I visit this doctor who I trusted" % 不自覺就講出，不自主就想要聽從他的建議，不再有抗拒
%I find that when I am closest to my inner, intuitive self, when I am somehow in touch with the unknown in me, when perhaps I am in a slightly altered state of consciousness, then whatever I do seems to be full of healing. Then simply my presence is releasing and helpful\citep{Rogers1979}.
%當最接近我的內在、直覺的自我，當我與我內在那不知名的角落（the unknown in me）接觸時(靈性)，當我有一些不一樣的意識狀態時（明心見性），我發現，單純只是我的存在（presence）就能鬆綁案主而且具有療效的。
%Rogers, C. R. The Foundations of the Person-Centered Approach. Education 100, 98–107 (1979).
%***「診後說三句，更勝良帖十方」


%% why holistic? 太極 氣功 情緒 myofascia and sympathetic system
%中醫人體觀－五臟與經絡情緒 工作坊《課程簡介》情緒是人體自身的氣，我們因情緒而調動它，也因它內傷，誘發情緒的原因有外在和內在，外在多跟經絡接收訊息的能力有關，當外在的影響如果沒有適當阻斷的時候，也會影響到五臟功能；內生的情緒則多以活出自我價值感有關，當我們活出此生想要體驗的人生與成就，我們就能夠不受情緒控制而保持內在平靜，這一切的源頭皆與我們是否活出真實的自己有關。
%養生和治療的原則:
%一、某些病症可以透過食療而改善的會建議自行調養
%二、某些病症已需要醫療輔助的時候會建議醫食同時進行勿偏癈其一
%三、某些病症需要醫療系統監控的時候會建議多找幾家醫院醫師綜合性評估
%四、沒有一勞永逸的食療，不宜執著特定的食療神話
%五、體質會變，所以變的是療法，而不是讓體質去適應固定療程
%六、認識自己身體的狀態比吸收營養資訊更重要
%七、沒有好或不好的飲食或療法，只有適不適合自身體質的療法
%情志養生 喜、怒、思、悲、恐 or 七情
%喜、怒、思、悲、恐為代表，就稱為五志 Joy, anger, thought, sadness, fear
%喜、怒、思、悲、恐、"驚憂"，稱為七氣，即七情。加之以寒熱，稱為九氣
%人俯仰於天地之間
%順從四季氣候變化
%保養正氣陶冶性情
%自我療癒身心靈疾病
%自他不二 心存正念 向內看，解決苦之源
%1.衷心懺悔
%2.真心感謝
%3.誠意祝福
%4.永存善念 慈悲
%5.心無恐懼: 情志養生
%每位醫師都可以成為「創傷知情者」幫助我們身邊的病患，懂他的心靈創傷，讓他有安全感， 才有機會改變疾病的走向
%"每一位患者都有自癒能力， 我們知情之後， 也要逐步讓他本人知情， 看見之後， 在良好的「治療關係」中， 協助他們漸漸找回自己的療癒。（以他們自己的腳步）"
%解說：患者的自癒力就是復原力（resilience),  強調「治療關係」（安全、 信任、 分享權力、 自決） 以及”知情”(暸解過往創傷經驗對自身的影響，進而開始療癒的過程）的重要性。

%% why holistic? % Environment 環境 => 分享權力、 自決 empowering
%The WHO's second principle is empowering – supporting people to manage and take responsibility for their health.
%Bruce Lipton introduced mind-brain-body system in 1992\citep{Mokhtari2011}\citep{Kobisi2012}\citep{Gustafson2017}.
%Lipton 1992, epinephrine and histamine signals in endothelium, %心智作用
%In a state of fear, the brain releases chemicals of stress hormones and inflammatory cytokines.
%Conversely, dopamine, oxytocin, vasopressin, and growth hormone will be released by the central nervous system in response to the perception of love.
% Arginine vasopressin (AVP) and oxytocin (OXT) are two centrally released neuropeptides that are involved in shaping essential social behaviors, like aggression, social recognition, and social motivation.
%Bruce Lipton: the biology of belief %信念的力量 \citep{Lipton2015}
%placebo effect and the old belief of genetic determinism\citep{Mokhtari2011}\citep{Kobisi2012}\citep{Gustafson2017}
%biowell 勿膨脹自我: 我非常擔心，所謂的科學實驗，研究的疾病，發明的藥物，都會潛藏一種危機，因為它的療效與科學證據，都是建立在「意識統一場域」中，科學家在研究時，一定希望有效，能夠發現別人沒有發現且有用的事情，所以「成功」還是會帶有「人為的意念」。
%那麼，這個結果不「公正」。甚至，當原來藥物發明者的心念改變了，（開始只想賺錢），那麼藥物的作用就更無法掌握。也許，藥物副作用會很嚴重。而目前科學的解釋是，只是因為個體的基因差異。
%% 1. Kobisi, W. Mind inhibitors. in The 30th International Congress Of The World Federation Of Hemophilia, July 8‐12 vol. 18 170–171 (John Wiley & Sons, Ltd, 2012).
%As a person with hemophilia (PWH), my passion led me to conduct personal research to uncover the main cause of my drawbacks in my education, my career, and my romantic life. I have experienced unjustified conflicts in my social relations, feeling stale in mind and body
%Findings: Through my research, I uncovered that I used to suffer from what I call 'mind inhibitors' these were the hidden cause of my life disorders and spontaneous bleeds. These mind inhibitors are responsible for locking the body factor receptors, blocking body energy centres (like the solar plexus), discouraging the super-intelligent body cells from running their auto-healing system, and paralyzing the instinctive high-speed cellular response to external molecular treatment. 
%Mind inhibitors originate from different sources like thinking of oneself in a victim role, anxiety about medical treatments, lack of financial stability, negative attitudes, self-rejection, and contradictory thoughts and feelings. 
%Conclusion: Mind inhibitors should be handled as seriously as biological inhibitors. As Dr. Bruce Lipton explains in his book, 'The Biology of Belief'\citep{Lipton2015}: DNA does not control our biology; instead DNA is controlled by signals from outside the cell. Our bodies can be changed as we retrain our thinking, since thoughts are made of energy that affects the cellular energy level, and thus everything is created twice: once in our minds and then the physical creation follows. % 萬法唯心造
%Holistic healthcare should be considered for treating patients as a complete synchronized system of soul, mind, and body; treatment should include more than medication or surgery, but a holistic approach would also address the patient's need for a subconscious paradigm shift in the person's thought and belief system.\citep{Kobisi2012}\citep{Lipton2015}


%騙過你的病-安慰劑效應 
%The Placebo Effect (film by Emmanuelle Sapin, Pascal Goblot)\citep{Sapin2014}
%1. Sapin, E. & Goblot, P. L’effet placebo (English: The Placebo Effect) [enregistrement vidéo] : le formidable pouvoir de l’auto-guérison. 53 PP-Quebec (Cinéfête, 2014).\citep{Sapin2014}
% https://biblio.bdeb.qc.ca/in/sim/faces/details.xhtml;jsessionid=E3DD39109012D0BEA598E6E5B66A4FD2?id=p::usmarcdef_0000058243&posInPage=8
%1. Finniss, D. G. Placebo Effects: Historical and Modern Evaluation. in International review of neurobiology vol. 139 1–27 (2018).\citep{Finniss2018}
%蓋倫劑量 placebo effect 身心靈醫學的核心
%In 1772, Willian Cullen, a British physician, wrote that  "I own that I did not trust much to it, but I gave it because it is necessary to give a medicine, and as what I call a placebo. If I had thought of any internal medicine, it would have been a dose of the Dover's powders (made up of Ipecac and opium)."\citep{Kerr2008}\citep{Finniss2018}
%Cullen studied the objective science of analytical chemistry. He also studied that the patient's therapeutic response could be influenced by his
%subjective, psychosomatic, or "sympathetic constitution"\citep{Kerr2008}.
%Cullen developed a theory of "sympathy" (or empathy) which informed his ideas about clinical medicine dealing with the psychosomatic theory of illness and mind-body therapeutics. % he do 炮製
%Cullen's theory of sympathy (or empathy) is a mind-body therapeutics, a kind of "vital force" that animated the human body, joint function, and transmitted sensation to target organs\citep{Forget2003}\citep{Kerr2008}.
%This finely calibrated, rational use of active placebo treatment, in which the physician-dispensed weak, physiologically active substance in order to please and calm the patient rather than to cure the patient's underlying disorder, was a product of the rational scientific culture of the 18th century Scottish Enlightenment and its embrace of a grounded, empirical approach to medicine. %啟蒙運動
%Cullen found the power of mind-body interaction that a physician's therapeutic intentions could be triggered by sympathy\citep{Sapin2014}.
%How does placebo effect work in patient's internal environment?  %改造患者的心，就能改造他的身體環境，有利消除癌症
%安慰劑效應的科學研究，驚人地發現現代西方醫學的不確定性。
%前額葉背側區 (dorsolateral prefrontal cortex)  是形成安慰劑療效的關鍵(PCA「治療關係」的生理學依據)
%But modern research, especially the results of neuroimaging studies, has shown unequivocally that placebo-induced analgesia is real in the sense that the neural correlates of the sensation of pain are reduced, in the same way that they would be reduced by a decrease in the intensity of a pain-evoking stimulus or by the administration of an analgesic drug\citep{Eippert2009}\citep{Watson2009}. 
%Pain researchers are united in the belief that placebo analgesia is real, in the sense that the neural processes that represent pain are reduced in neuroimaging studies\citep{Bennett2018}. 
%The anterior cingulate cortex and the dorsolateral prefrontal cortex of the brain have been found to be activated by placebo administration\citep{Benedetti2005}\citep{Benedetti2011}\citep{Carlino2011}\citep{Bennett2018}. 
%The prefrontal cortex runs conscious mind generating empathy and love, and will be influenced during mindfulness meditation\citep{Krummenacher2010}\citep{Bennett2018}.
%prefrontal lobe 理智腦 意識 正念 杏仁核 prefrontal cortex runs conscious mind「安慰劑」placebo effect，也已經有科學證明，與醫師、病患的意識（前額葉）功能，非常有關係。這也是負責同理心與愛的腦區。
%sympathy 富有同情心、慈悲心，靈性高度成熟的醫師，不是只用藥物或手術來治療病患，「他自己本身」就具有療癒效果。
% 安慰劑的研究，拓展了我們對醫病關係的理解，回歸到以病患為中心。
%醫療處理的不只是藥物和手術，人更是必須處理的對象，把病患看作一個整體，要掌握身與心的個人特質。
%#正念 減壓，療癒疼痛\citep{Krummenacher2010}\citep{Bennett2018}、免疫系統、內分泌系統或自律神經系統相關的疾病，以及憂鬱症、焦慮症 
%The idea of placebo as a diluted but active substance persisted into the 19th century but the term also came to mean a physiologically inert substance - bread or lactose, for example. Inert substances began to be used in comparative clinical trials during the first half of the 19th century but this latter meaning of the term was given particular impetus at the end of the 19th century and the beginning of the 20th century, when inert substances were used as controls in pharmacological experiments. This shift in meaning may have reflected the rise of a new emphasis (or even metaphysics) in medicine, in which the molecular make up of an active drug assumed central importance, and any other positive benefits were seen as non-specific, placebo effects.\citep{Kerr2008} % 漸漸被特意忽略
%物理作用、信號，雷射針灸
%non-verbal contact
%前額葉內側 (medial prefrontal cortex)
%額葉底部(orbitofrontal cortex)
%%%%%%%%
% mindfulness meditation
% 人的腦波一般分為β（12-30Hz）、α(8-12Hz）、θ(4-8Hz)、δ（1-4Hz)，四種波。當我們處於清醒、警覺狀態時，腦波是處於高頻率的β波。隨著身心狀態逐漸放鬆，卸下防衛時，腦波頻率也會逐漸下降至α、θ波，此時身心狀態是處於最寧靜、創造能力最好、對事情洞察能力最佳的狀態。如果能夠安祥進入睡眠階段，則腦波會降至δ波。靜坐時不僅可以使腦波由快速的β波降至α、θ波
%In human EEG studies, cortical theta wave (4–7 Hz) tends to appear during meditative, drowsy, hypnotic or sleeping states in older children and adults. Theta is a frequency lower than consciousness. Theta is actually a brain function also associated with imagination.

%cancer therapy by 患者自癒 Mindfulness Meditation help 復原力（resilience）
%Self-Compassion  慈悲
% 除我執 global self-esteem:
%自戀（narcissism）1. Neff, K. D. & Vonk, R. Self-compassion versus global self-esteem: Two different ways of relating to oneself. J. Pers. 77, 23–50 (2009).\citep{Neff2009}

%%%% healing and cure by Changing the Environment 環境
%genetic determinism 所以我們的科學家依舊在研究，基因對癌症的影響（單向）
%The genetics of the cell gives all kinds of potentials. % 自性空
%Environmental signals from the cell membrane can go into the nucleus via signal transduction. The signal selectively changes the reading of our gene, e.g., epigenetics regulation, controlling cell behaviors.% including genetics.
%The central dogma of molecular biology should be modified as environmental signals to regulatory protein to DNA to RNA and then to protein.
%Epigenetics control says that genetic expression is directly due to the environment of our cells and our perception of the environment\citep{Gustafson2017} (e.x. stressful environment, see Figure \ref{fig:figure5}).
% so do the cancer
%Individual treatment and avoiding assumptions. Although it is useful for therapists to have a knowledge of the trauma literature on the general reactions of people who experience trauma, the therapist should not assume that this knowledge necessarily applies to each and every client\citep{Joseph2004}.
%The membrane proteins and the chromosomal proteins that control the DNA, called regulatory proteins. But even those proteins are controlled by environmental signals. %It is not DNA to RNA to protein.
%療癒 
%Study showed that after several hours of meditation, the expression of some inflammatory genes changed\citep{Gustafson2017}.
%You can change your genetic activity by how you change your response to the environment. 
%to recognize your perceptions, via signal transduction, are translated into biological behavior and your genetic activity.
% 愛自己就是渡蜜月；其實是 正念 mindfulness (因為自他不二）
%You are creating a world of joy and love, and that’s called the honeymoon. When we fall in love, we stop focusing our conscious mind in thought and start keeping it present. 
%It is called being mindful. 
%You’ve been looking for yourself your whole life.
%愛、性、正念止觀
%Science has recognized that immediately after falling in love, we enter a period of mindfulness where we keep our conscious mind present.
%You start running programs that are based on your conscious wishes and desires. All of a sudden, without the programs—without the subconscious programs—we begin to experience a heavenly life.
%%%%%% carcinogenesis
%95\% of cognitive behavior runs on subconscious program:
%70\% %對於健康有不良影響 => 卻是惡劣競爭下的生存技能
%Of the downloaded behaviors acquired before age 7, the vast majority—70\% or more—are programs of limitation, disempowerment, and self-sabotage. These programs were acquired from other people, not from ourselves, with a survival advantage.
%Being subconscious, those programs acquired from other people before age 7, are occurring without conscious recognition and awareness. 
%The survival advantage programs, such as limitation, disempowerment, and self-sabotage, will affect epigenetic regulation.
%Genes are not self-actualizing. %The control of genes is not due to any inherent activity in the DNA itself. 
%The regulation of genetic activity is due to the interaction of the cell with environmental signals.
%Less than 1\% of disease is associated with genetics. 
%Over 90\% of disease is a total reflection of the environment.% and especially our programming.
%the disempowering, self-sabotaging behaviors that we acquired in the first 7 years.
%the cell membrane through receptors picking up the signals and translating them into biology, which then sends signals into the nucleus, which then controls the genetic activity.
%The mind-brain-body axis shows that subconscious fear causes releasing of inflammatory and stress hormones from the brain in terms of cell alteration in DNA methylation, mutation, mRNA expression. 
%Carcinogenesis will occur if the reprogramming is persistent for a longer time.
%Moreover, long-term stress could be correlated with cancer metastasis\citep{Lutgendorf2010}\citep{Moreno-Smith2010}\citep{Du2020}\citep{Xu2021},
%and cancer initiation\citep{Lutgendorf2010}\citep{Powell2013}\citep{Iftikhar2021}.(see Figure \ref{fig:figure5}) %HNSCC
%A%dverse psychosocial experiences in early life, such as child maltreatment, caregiver stress, and community violence, have been associated in epidemiological studies with an increased risk of diabetes, heart disease, cancers, and psychiatric illnesses. 
%The possible molecular mechanisms could be also related with the mind-brain-body axis\citep{Berens2017}.
%Additional work has shed light on the potential molecular mechanisms by which early adversity becomes "biologically embedded" in altered physiology across body systems.\citep{Berens2017}
%%%%%


\subsection{Risk Prediction Models}
The key clinical challenge is how to properly care patients with OVH for oral cancer prevention and early identification. 
One important strategy for dealing with this clinical situation should be the establishment of risk prediction models for malignant transformation. The development of risk prediction models are based on the identification of risk factors.

%%%

\subsubsection{Molecular Oncology}
%%% from Chi2022 Ph.D. dissertation ***************
%Prognostic significance of p53 mutation has been reported.
%% **** TP53 gene => p53 protein; mutation of TP53 ***
Genetic variations and/or gene-environment interactions, in addition to environmental and molecular variables, may influence the likelihood of malignant transformation in \acrshort{ovh} patients.
Accumulative knowledge shows that some drivers could facilitate carcinogenesis in \acrshort{oscc}.
For example, p53 is a tumour suppressor protein encoded by the TP53 gene.
The transcription factor, p53, that is activated by many genotoxic insults and induces cellular apoptosis. 
The gene TP53 is frequently mutated or functionally inactivated in cancer cells.
TP53 mutations are comprised of eight classes: missense, nonsense, frame-shift deletion, frame-shift insertion, in-frame deletion, in-frame insertion, silent and splice-site.
Immunohistochemical (IHC) staining for p53 is used as a surrogate for TP53 missense mutation in cancer cells. 
%Usually, wild-type TP53 could be found in 1--10\% of IHC; 0 or 50--100\% in missense mutation of TP53. [\url{https://www.nature.com/articles/modpathol201185/tables/1}]
% *** p53 is a tumour suppressor protein encoded by the TP53 gene. This protein is expressed and regulates the transition from G1 to S phase in cell cycle division. P53 is the key mediator of cell cycle arrest for induction of DNA repair. If failed in the repair, p53 might trigger apoptosis leading programmed cell death, as a response to cellular stress (ref). p53 is often referred to as the "guardian of the genome". Its prime function is to put cells, which has aberrant DNA, into repairment or apoptosis, and thereby prevent malignant transformation. P53 is mutated(?) presenting in more than 50\% of all cancers and found by IHC.
%The node-negative \acrshort{hnscc} patients with overexpression of mutated-TP53 were found to have poor prognosis~\citep{DeVicente2004}.
The mutations of TP53 are one of the most frequent abnormalities in \acrshort{oscc} found within up to 70\% of surgical specimen~\citep{Alsner2001, Singh2016, Wang2017, WARD2018}. It is also found in severe dysplasia of \acrshort{ovh}.
Ki-67, encoded by MKI67 gene, is a measurement of the proliferative rate of \acrshort{oscc}~\citep{Silva2004}.
%Diagnostic biomarkers don't predict prognosis or aggressiveness in HNSCC~\citep{Chatzkel2017}.
Overexpression of Ki-67 impacts on poor overall survival of \acrshort{oscc}~\citep{Perisanidis2012,Szentkuti2015}.
%*** biomarkers for radio-resistance or proton therapy? Concurrent systemic therapy:~\citep{Brockstein2020} (UpToDate database)
%It was found that radiation resistance of HNSCC is correlated with overexpression of hypoxia-inducible factor (HIF)-1 alpha~\citep{Aebersold2001} or Ki-67~\citep{Couture2002}. 
The epidermal growth factor receptor (EGFR) is found in 70\% of \acrshort{oscc}~\citep{O-Charoenrat2000, Bentzen2005}.
It is involved in cancer cells evading immunosurveillance~\citep{Stefanidakis2006}.
% **** ACTA2 and myofibroblastic CAF (myCAF)
% ACTA2: alpha-SMA
% The local fibroblasts within tissues are considered the predominant source of myofibroblasts. The myofibroblasts have the expression of alpha smooth muscle actin (α-SMA) and fibronectin during during wound healing and organ fibrosis (or oral submucous fibrosis).
% 1. Joshi RS, Kanugula SS, Sudhir S, Pereira MP, Jain S, Aghi MK. The role of cancer-associated fibroblasts in tumor progression. Cancers (Basel). 2021;13(6):1-27. doi:10.3390/cancers13061399
% some common CAF markers that have been identified include alpha smooth mus- cle actin (α-SMA), vimentin, fibroblast activation protein (FAP), fibroblast-specific protein 1 (FSP1), and platelet-derived growth factor receptor alpha/beta (PDGFR-α/β) => in HNSCC: α-SMA, PDPN, FAP, PDGFR-α, PDGFR-β, FSP1, NG2
% α-SMA—alpha smooth muscle actin, FAP—fibroblast activation protein, PDGFRα—platelet derived growth factor receptor alpha, PDGFRβ—platelet derived growth factor receptor beta, NG2—neural/glial antigen 2, FSP1— fibroblast specific protein 1, PDPN—podoplanin, MFAP5—microfibrillar associated protein 5, POSTN—periostin, FGF-1—fibroblast growth factor 1.

\subsubsection{Gene-environment Interaction in Oncology}
Chronic mechanical and chemical irritation from betel nut chewing could increase myofibroblasts in terms of developing oral submucous fibrosis~\citep{Angadi2011} and \acrshort{oscc}.
Myofibroblasts are contractile cells expressing \acrfull{acta2} (encoding alpha-smooth muscle actin, $\alpha$-SMA) when is stimulated by betel nut extract---arecoline---\textit{\fontfamily{lmss}\selectfont in vitro}~\citep{Chang2014}.
A elevated \acrshort{acta2} expression is also associated with poor overall survival in \acrshort{oscc}~\citep{Marsh2011}.
Moreover, overexpression of \acrshort{acta2} in myofibroblastic cancer-associated fibroblast (myCAF) could promote stemness of \acrshort{oscc}~\citep{Patel2018, Joshi2021}.

The possibility of malignant transformation is influenced by the severity of epithelial dysplasia on \acrshort{ovh} samples.
%Clinically helpful biomarkers for dysplasia grading include KRT13 (type I cytoskeletal 13, encoding Cytokeratin13---CK13), KRT17 (type I cytoskeletal 17, encoding Cytokeratin 17---CK17), TP53 (encoding p53), MKI67 (Marker Of Proliferation Ki-67, encoding Ki-67).
%CK13, CK17, p53, and Ki-67.
%  KRT13 (type I cytoskeletal 13, aka. CK13), KRT17 (type I cytoskeletal 17, aka. CK17), TP53 (p53), MKI67 Marker Of Proliferation Ki-67 (Ki-67)
Diagnostic markers are those that distinguish between different categories of lesions. They are obtained by comparing cross-sectional samples taken from various patients.
Diagnostic biomarkers, such as KRT13 (type I cytoskeletal 13, encoding Cytokeratin13---CK13), KRT17 (type I cytoskeletal 17, encoding Cytokeratin 17---CK17), TP53 (encoding p53), and MKI67 (Marker Of Proliferation Ki-67, encoding Ki-67), are applied in routine pathological examination for dysplasia grading (semi quantitative) , and a confirmation of \acrshort{oscc}.
The expressions for CK17 and CK13 are reciprocal in oral dysplasia lesions and that the CK17/p53 emergence is helpful to confirm a diagnosis of \acrshort{oscc}~\citep{Mikami2011}.

 



\section{Preliminary Results}
% P-value => \textit{p}~value
% Results: the main findings
%\textbf{Preliminary Results:}
Using pvalueTex~\citep{Chi2021} on the \acrshort{tcga} \acrshort{hnscc} cohort, we scanned human protein-coding genes (20,500) programmatically. After adjustment with other confounders, the clinical tumor stage and the surgical margin involvement are independent risk factors in patient survival. 
% *** usually safety margin: OVH 5 mm, SCC 10 mm safety margin

%According to the resulting tables with Bonferroni adjusted \textit{P} value under optimal cutoff as well as hazard ratio $(>=1.5)$, 
%There were ten candidate biomarkers, named as DKK1, CAMK2N1, STC2, PGK1, SURF4, USP10, NDFIP1, FOXA2, STIP1, and DKC1, which are 
And overexpression of CAMK2N1 is significantly associated with the poor prognosis of \acrfull{os} (Table~\ref{table:table2}). 


\comm{
%%%%%%%% xxxx
In our previous transtriptomics study~\citep{Chi2020}, %\citep{Tai2007},
% from pvalueTex
the 9416 Kaplan-Meier plots with associated Cox's univariate and multivariate tables were generated by Cox modeling %(see Figure \ref{fig:figure1}) 
and justified by the ranking of hazard ratios.
The 967 genes were kept by criteria of uncorrected \textit{P} value (below 0.05) and \acrfull{hr} (greater than 1.5 or less than 0.5). %(see Figure \ref{fig:figure2}(a) univariate, and Figure \ref{fig:figure2}(b) multivariate plots). 
At the final step, a stringent criteria (Bonferroni \textit{P} value correction) was used to yield a candidates: CAMK2N1. %(see Figure \ref{fig:figure2}(c), (d)). 
} % end of \comm{}

% before validation
%Among these candidates, ten overexpressed genes (symbol as DKK1, CAMK2N1, STC2, PGK1, SURF4, USP10, NDFIP1, FOXA2, STIP1, and DKC1) are associated with a poor prognosis of  \acrshort{os}.% (see Table \ref{table:table1}). 
%The other ten overexpressed genes (symbol as ZNF557, ZNF266, IL19, MYO1H, FCGBP, LOC148709, EVPLL, PNMA5, IQCN - former symbol as KIAA1683, and NPB) are correlated with better survival.% (see Table \ref{table:table3}, with their gene name).
%We made a volcano plot for 9416 genes by Kaplan-Meier \textit{P} value (less than 0.05, obtained during cutoff finding procedure) against the Cox hazard ratio (see Figure \ref{fig:figure3}). The plot reveals that the most significant (Bonferroni-adjusted \textit{P} $< 0.05$) candidate genes are located above the dotted line. At the same time, Cox's HR separated them on the two-side with prognostic impact.
% => after validation: The result showed six overexpressed genes (symbol as CAMK2N1, PGK1, SURF4, USP10, NDFIP1, FOXA2) are significantly associated with a poor prognosis of overall survival. Furthermore, the four overexpressed genes (symbol as IL19, FCGBP, IQCN - former symbol as KIAA1683, and NPB) are correlated with better survival.

%注意 強調GSE2837: 
%The prognostic significance of the candidate genes has been validated by an independent patient cohort.
% validation by GSE2837 cohort (with web-tools)
%\subsubsection*{GSE2837} % non-TCGA
% from Anwser2-2
%by GEO GSE2837 HNSCC dataset (PrognoScan)
%We validated the those 20 genes in the other independent \acrshort{hnscc} cohort (GSE2837).
%The survival significance (Kaplan-Meier \textit{p} value) is in the following probes among 10 genes:\\
%1) poor prognosis: CAMK2N1 (0.048214), PGK1 (0.009978), SURF4 (0.023127), USP10 (0.017768), NDFIP1 (0.022758), FOXA2 (0.001587);\\ % FOXA2 (0.038125)
%2) better prognosis: IL19 (0.049731), FCGBP (0.005658), KIAA1683 (IQCN, 0.005886), NPB (0.014177);\\
%17 out of 20 candidates
%?DKK1 (0.253635), 
%Nevertheless, PrognoScan has group separation cut by a skewed manner, and the GSE2837 has far fewer participants than the TCGA cohort.
%DKK1, CAMK2N1, STC2, PGK1, SURF4, USP10, NDFIP1, FOXA2, STIP1, DKC1;
%ZNF557, ZNF266, IL19, MYO1H, FCGBP, LOC148709, EVPLL, PNMA5, KIAA1683 (IQCN), NPB
%Those 11 genes achieve similar positive and negative prognostic effect comparable with our proposed candidate genes. 
%Thus, these 10 candidates have been confirmed by a comparative study using the GSE2837 dataset other than the TCGA cohort.
%Please see Kaplan-Meier plots in Supplementary Figure S1.%\ref{fig:fig_GSE2837}.


\comm{
%\subsubsection{Figure 2:h}
\begin{figure}[hp]
\centering
%\includegraphics[width=14cm]{Figure2.pdf}
%\bcaption{\acrshort{hnscc} Cox's hazard ratio and \textit{P} value plot.}
%{(a) Univariate HR versus uncorrected \textit{P} value; (b) Multivariate HR versus uncorrected \textit{P} value; (c) Univariate HR versus Bonferroni corrected \textit{P} value; and (d) Multivariate HR versus Bonferroni corrected \textit{P} value.}
\label{fig:figure2}
\end{figure}

% tables 1  3
%%% tables
% Table1
% wide table \usepackage{tabularx}
% makecell{}


%\subsubsection{Figure 3:h}
\begin{figure}[hp]
\centering
\includegraphics[width=14cm]{TCGA_HNSC_Optimal_Overall_allPlot_unKM_P_multiHR-Figure3.pdf} % .png from PvalueplotKM_20genes_Bonf.pdf
\bcaption{Volcano plot of genes under survival analyses.}
{X axis: unadjusted \textit{P} value of Kaplan-Meier survival (-log10 transformed).
Y axis: multivariate hazard ratio from Cox proportional regression model.
Dotted line: significant Bonferroni corrected \textit{P} value. 
\textcolor{red}{Red dots} mark 10 genes (unvalidated), which impact on poor prognosis ($HR>=1.5$). \textcolor{green}{Green dots} mark 10 genes (unvalidated), which affect on better survival ($HR<=0.5$).}
\label{fig:figure3}
\end{figure}
% new figure of volcano plot: TCGA_HNSC_Optimal_Overall_allPlot_unKM_P_multiHR.pdf, -log10(raw KM P-value) vs Cox multivariate HR
% X axis: "frequency" (number of uncorrected P-values less than 0.05 during Kaplan-Meier cutoff finding procedure). X axis: adjusted Kaplan-Meier P-value.

\clearpage
} % end of \comm

%After validated by GSE2837, our top 1 candidate is \acrfull{CAMK2N1}. The Kaplan-Meier curve reveals 152 patients bearing the higher expression of \acrshort{CAMK2N1} were suffered from only 35\% of 5-year \acrshort{os} rate. In comparison, the other 262 patients with lower expression (the cutoff at 0.027(RSEM)) had a better prognosis (adjusted \textit{P} = $0.002$) (see Figure \ref{fig:figure4}(a)).
%Figure \ref{fig:figure4}(b)'s cumulative \textit{P} value plot shows that the uncorrected 147 \textit{P} values ($< 0.05$) have been estimated by a serial cut from 144 to 290 persons for grouping the cohort in our cutoff finding procedure (cutofFinder\_func.R, see Figure \ref{fig:figure1}, cutoff engine). The smallest \textit{P} value (\num{2.97e-7}), when cut on n=262 (63.3\% of 414), has been defined as an optimal \textit{P} value.
%Conversely, the most associated gene with better survival is \acrfull{IL19}. In Figure \ref{fig:figure4}(c), a Kaplan-Meier curve reveals 215 patients bearing the higher expression of \acrshort{IL19} had 60\% of 5-year OS survival rate (adjusted \textit{P} = $0.002$). The cutoff finding procedure (cutofFinder\_func.R) generated cumulative \textit{P} value plot in Figure \ref{fig:figure4}(d). The 161 uncorrected \textit{P} values were estimated by a serial cut from 125 to 286 for grouping the cohort. The smallest \textit{P} value (\num{3.73e-7}), when cut on n=199 (48.1\% of total cohort 414), has been defined as an optimal \textit{P} value with a cutoff value -0.147(RSEM) of \acrshort{rnaseq}.
%The second candidate, which improving patient survival, is ZNF266.
%\subsubsection{Tables} in main article

\comm{
%\subsubsection{Figure 4:h}
\begin{figure}[hp]
\centering
\includegraphics[width=15cm]{Figure_4_CAMK2N1_IL19.pdf}
\bcaption{Kaplan-Meier survival analyses, by cutoff finding.}
{(a) Kaplan-Meier plot of CAMK2N1 under optimal \textit{P} value, and (b) the cutoff is derived from cumulative \textit{P} value plot of CAMK2N1. (c) Kaplan-Meier plot of IL19 under optimal \textit{P} value, and (d) the cutoff is derived from cumulative \textit{P} value plot of IL19.}
\label{fig:figure4}
\end{figure}

\clearpage
} % end of \comm{}


%Table \ref{table:table1} shows ten overexpressed genes are associated with poor prognosis in \acrshort{hnscc}, ranked by adjusted Kaplan-Meier \textit{P} value.
%Six out of ten preliminary candidates has been validated by using GSE2837.
%We found those six candidates have the Cox's univariate and multivariate HR above 1.914.
%There were few published articles of \acrfull{SURF4} and \acrfull{NDFIP1} (\acrshort{NEDD4}: \acrlong{NEDD4}), which were related to cancer research.
%In Table \ref{table:table2}, % table 3
%after adjustment of confounders, \acrshort{CAMK2N1} overexpression is the independent prognostic factor (multivariate HR 2.007 [95\% CI: 1.490-2.704, \textit{p} $<$ 0.001]), as well as clinical T stage (HR 1.982 [95\% CI: 1.048-3.745, \textit{p} = 0.035]) and surgical margins status (HR 1.631 [95\% CI: 1.182-2.250, \textit{p} = 0.003]). 
%Older age (more than 65) also worse the survival (HR 1.391 [95\% CI: 1.025-1.888, \textit{p} = 0.034]). 
%The M stage could be ignored in this cohort due to only 3 out of 414 patients with distant metastasis.





% CAMK2N1
%\subsubsection{Table3/legend} Table 3
%Table 2. Univariate/Multivariate Cox's proportional hazards regression analyses on OS time of DKK1 gene expression in HNSCC.
%(P-value Significant codes is denoted as  0.01 mark as *; 0.001 mark as **; if $<$0.001  mark as ***)
\begin{table}[hp]
\centering
\caption{Clinical (T)umor size, positive surgical margin, and CAMK2N1 expression are independent prognostic factors in HNSCC (TCGA) cohort.}
%Univariate/multivariate Cox's proportional hazards regression analyses on OS time of CAMK2N1 gene expression in HNSCC}

%\begin{table}[H] 
%\centering
%\captionsetup{labelformat=empty}
%\captionabove{%Univariate/multivariate Cox proportional hazard regression analyses on overall survival time of \hl{CAMK2N1} gene expression in HNSCC.}


%%%\label{table:table2}
\arrayrulecolor[rgb]{0.255,0.255,0.255}


\resizebox{0.8\linewidth}{!}{%
\begin{tabular}{|l|l|c|c|c|c|c|c|} 
\noalign{\hrule height 1.0pt}
%\begin{tabularx}{\textwidth}{|p{2.5cm}|l|l|l|l|l|l|l|} 
%\arrayrulecolor{black}\cline{1-2}
\arrayrulecolor[rgb]{0.255,0.255,0.255}
\cline{3-8}
\multicolumn{2}{|l!{\color{black}\vrule}}{\multirow{2}{*}{\textbf{Features}}}                                                          & \multicolumn{3}{c|}{\textbf{Univariate}}                                                                                                                                                                                                                & \multicolumn{3}{c|}{\textbf{Multivariate}}                                                                                                                                                                                                               \\ 
\cline{3-8}
\multicolumn{2}{|l!{\color{black}\vrule}}{}                                                                                   & \multicolumn{1}{l!{\color{black}\vrule}}{\textbf{HR}}                                   & \multicolumn{1}{c!{\color{black}\vrule}}{\textbf{CI95\%}}                              & \multicolumn{1}{l!{\color{black}\vrule}}{\textbf{\protect\textit{p}~Value}}                    & \multicolumn{1}{l!{\color{black}\vrule}}{\textbf{HR}}                                   & \multicolumn{1}{c!{\color{black}\vrule}}{\textbf{CI95\%}}                              & \multicolumn{1}{l!{\color{black}\vrule}}{\textbf{\protect\textit{p}~Value}}                     \\ 
\arrayrulecolor{black}\hline
\multirow{2}{*}{Gender}                 & \multicolumn{1}{l!{\color{black}\vrule}}{{\cellcolor[rgb]{0.62,0.812,0.878}}Female} & \multicolumn{1}{l!{\color{black}\vrule}}{{\cellcolor[rgb]{0.62,0.812,0.878}}1} & \multicolumn{1}{l!{\color{black}\vrule}}{{\cellcolor[rgb]{0.62,0.812,0.878}}} & \multicolumn{1}{l!{\color{black}\vrule}}{{\cellcolor[rgb]{0.62,0.812,0.878}}} & \multicolumn{1}{l!{\color{black}\vrule}}{{\cellcolor[rgb]{0.62,0.812,0.878}}1} & \multicolumn{1}{l!{\color{black}\vrule}}{{\cellcolor[rgb]{0.62,0.812,0.878}}} & \multicolumn{1}{l!{\color{black}\vrule}}{{\cellcolor[rgb]{0.62,0.812,0.878}}}  \\ 
\cline{2-8}
                                        & Male                                                                                & 1.157                                                                          & 0.843--1.587                                                                   & 0.367                                                                         & 1.076                                                                          & 0.767--1.510                                                                   & 0.671                                                                          \\ 
\arrayrulecolor[rgb]{0.255,0.255,0.255}\hline
\multirow{2}{*}{Age at diagnosis}       & {\cellcolor[rgb]{0.62,0.812,0.878}}$<=65y$                                             & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                            \\ 
\cline{2-8}
                                        & $>65y$                                                                                 & 1.329                                                                          & 0.990--1.784                                                                   & 0.058                                                                         & 1.391                                                                          & 1.025--1.888                                                                   & \textcolor{red}{0.034}                                                         \\ 
\hline
\multirow{2}{*}{\textcolor{red}{Clinical T Status}}      & {\cellcolor[rgb]{0.62,0.812,0.878}}T1+T2                                            & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                            \\ 
\cline{2-8}
                                        & T3+T4                                                                               & 1.409                                                                          & 1.028--1.931                                                                   & \textcolor{red}{0.033}                                                        & 1.982                                                                          & 1.048--3.745                                                                   & \textcolor{red}{0.035}                                                         \\ 
\hline
\multirow{2}{*}{Clinical N Status}      & {\cellcolor[rgb]{0.62,0.812,0.878}}N0                                               & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                            \\ 
\cline{2-8}
                                        & N1-3                                                                                & 1.185                                                                          & 0.890--1.577                                                                   & 0.246                                                                         & 1.145                                                                          & 0.801--1.636                                                                   & 0.457                                                                          \\ 
\hline
\multirow{2}{*}{Clinical M Status}      & {\cellcolor[rgb]{0.62,0.812,0.878}}M0                                               & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                            \\ 
\cline{2-8}
                                        & M1                                                                                  & 4.097                                                                          & 1.009--16.644                                                                  & \textcolor{red}{0.049}                                                        & 7.314                                                                          & 1.590--33.631                                                                  & \textcolor{red}{0.011}                                                         \\ 
\hline
\multirow{2}{*}{Clinical Stage}         & {\cellcolor[rgb]{0.62,0.812,0.878}}Stage I+II                                       & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                            \\ 
\cline{2-8}
                                        & Stage III+IV                                                                        & 1.245                                                                          & 0.882--1.759                                                                   & 0.213                                                                         & 0.621                                                                          & 0.287--1.343                                                                   & 0.226                                                                          \\ 
\hline
\multirow{2}{*}{\textcolor{red}{Surgical Margin status}} & {\cellcolor[rgb]{0.62,0.812,0.878}}Negative                                         & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                            \\ 
\cline{2-8}
                                        & Positive                                                                            & 1.591                                                                          & 1.155--2.191                                                                   & \textcolor{red}{0.004}                                                        & 1.631                                                                          & 1.182--2.250                                                                   & \textcolor{red}{0.003}                                                         \\ 
\hline
\multirow{2}{*}{Tobacco Exposure}       & {\cellcolor[rgb]{0.62,0.812,0.878}}Low                                              & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                            \\ 
\cline{2-8}
                                        & High                                                                                & 1.364                                                                          & 1.008--1.844                                                                   & \textcolor{red}{0.044}                                                        & 1.363                                                                          & 0.990--1.875                                                                   & 0.058                                                                          \\ 
\hline
\multirow{2}{*}{\textcolor{red}{CAMK2N1 Expression}}                & {\cellcolor[rgb]{0.62,0.812,0.878}}Low                                              & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                            \\ 
\cline{2-8}
                                        & High                                                                                & \textcolor{red}{2.101}                                                                                        & 1.572--2.809                                                                   & \multicolumn{1}{c|}{\textcolor{red}{\num{5.324E-07}}}       % 5.323587E-07
                                        & 2.007                                                                          & 1.490--2.704                                                                   & \multicolumn{1}{c|}{\textcolor{red}{\num{4.565E-06}}}       % 4.565120E-06
                                        \\ 
%\hline
%\multicolumn{8}{|l|}{}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           \\ 
%\hline
% \\
\noalign{\hrule height 1.0pt}
\end{tabular}
} % end of \resizebox

% \usepackage{pbox} 

\pbox{0.8\columnwidth}{\footnotesize {
(OS: overall survival;
HR: hazard ratio;
CI95\%: 95\% confidence interval;
\protect\textit{p}~value significant code is denoted: \textcolor{red}{red \textless{} 0.05}).} %; *** \textless{} 0.001).}                              
}
%\arrayrulecolor{black}
%\end{table}


\label{table:table2}
\end{table}



% IL19
%In Table \ref{table:table3}, the other ten overexpressed genes have been found in better prognosis of \acrshort{hnscc} patients. Cox's univariate and multivariate HR is under 0.5.
%Four out of ten has been validated by using GSE2837.
%In Table \ref{table:table4}, after adjustment of confounders, prognosis is influenced by advance clinical T Status (HR 1.961 [95\% CI: 1.035-3.714, \textit{P} = 0.039] ), positive surgical margin involvement (HR 1.631 [95\% CI: 1.18-2.254, \textit{P} = 0.003]) , and higher tobacco exposure (HR 1.453 [95\% CI: 1.055-2.000, \textit{P} = 0.022]).
%Overexpressed \acrshort{IL19} gene could has a protective influencce on prognosis (HR 0.499 [95\% CI: 0.372-0.669, \textit{P} $<$ 0.001]).


%In summary, those ten candidate biomarkers, clinical T stage, and surgical margin are independent prognosis factors in \acrshort{hnscc}.
%Thus, the prognosis model with coefficients is established from \acrshort{tcga} \acrshort{hnscc} cohort. 
%The important input $X_1...X_n$ should be patients' features: age, specific ten gene expressions, clinical T stage, and surgical margin.
%\\[0.5cm]
% total 6+4 =10
% After validation with an independent \acrshort{hnscc} cohort (GSE2837), the result showed six overexpressed genes (symbol as CAMK2N1, PGK1, SURF4, USP10, NDFIP1, FOXA2) are significantly associated with a poor prognosis of overall survival. 
%Furthermore, the four overexpressed genes (symbol as IL19, FCGBP, IQCN - former symbol as KIAA1683, and NPB) are correlated with better survival.



\comm{
%\subsubsection{Table4/legend} Table 4
\begin{table}[!hp]
\centering
\caption{Univariate/multivariate Cox's proportional hazards regression analyses on OS time of IL19 gene expression in HNSCC}
\arrayrulecolor[rgb]{0.255,0.255,0.255}
\resizebox{\linewidth}{!}{%
\begin{tabular}{|l|l|l|l|l|l!{\color{black}\vrule}l!{\color{black}\vrule}l!{\color{black}\vrule}} 
\arrayrulecolor{black}\cline{1-2}\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{3-8}
\multicolumn{2}{|l!{\color{black}\vrule}}{\multirow{2}{*}{Features}}                                                          & \multicolumn{3}{c|}{Univariate}                                                                                                                                                                                                                & \multicolumn{3}{c|}{Multivariate}                                                                                                    \\ 
\cline{3-8}
\multicolumn{2}{|l!{\color{black}\vrule}}{}                                                                                   & \multicolumn{1}{l!{\color{black}\vrule}}{HR}                                   & \multicolumn{1}{c!{\color{black}\vrule}}{CI95\%}                              & \multicolumn{1}{l!{\color{black}\vrule}}{\textit{P}~value}                    & HR                                   & \multicolumn{1}{c!{\color{black}\vrule}}{CI95\%} & \textit{P}~value                           \\ 
\arrayrulecolor{black}\hline
\multirow{2}{*}{Gender}                 & \multicolumn{1}{l!{\color{black}\vrule}}{{\cellcolor[rgb]{0.62,0.812,0.878}}Female} & \multicolumn{1}{l!{\color{black}\vrule}}{{\cellcolor[rgb]{0.62,0.812,0.878}}1} & \multicolumn{1}{l!{\color{black}\vrule}}{{\cellcolor[rgb]{0.62,0.812,0.878}}} & \multicolumn{1}{l!{\color{black}\vrule}}{{\cellcolor[rgb]{0.62,0.812,0.878}}} & {\cellcolor[rgb]{0.62,0.812,0.878}}1 & {\cellcolor[rgb]{0.62,0.812,0.878}}              & {\cellcolor[rgb]{0.62,0.812,0.878}}        \\ 
\cline{2-8}
                                        & Male                                                                                & 1.157                                                                          & 0.843-1.587                                                                   & 0.367                                                                         & 1.183                                & 0.847-1.652                                      & 0.325                                      \\ 
\hhline{>{\arrayrulecolor[rgb]{0.255,0.255,0.255}}|----->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}-|}
\multirow{2}{*}{Age at diagnosis}       & {\cellcolor[rgb]{0.62,0.812,0.878}}=65y                                             & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1 & {\cellcolor[rgb]{0.62,0.812,0.878}}              & {\cellcolor[rgb]{0.62,0.812,0.878}}        \\ 
\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{2-5}\arrayrulecolor{black}\cline{6-6}\arrayrulecolor{black}\cline{7-7}\arrayrulecolor{black}\cline{8-8}
                                        & 65y                                                                                 & 1.329                                                                          & 0.990-1.784                                                                   & 0.058                                                                         & 1.481                                & 1.087-2.018                                      & \textcolor{red}{0.013}                     \\ 
\hhline{>{\arrayrulecolor[rgb]{0.255,0.255,0.255}}|----->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}-|}
\multirow{2}{*}{Clinical T Status}      & {\cellcolor[rgb]{0.62,0.812,0.878}}T1+T2                                            & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1 & {\cellcolor[rgb]{0.62,0.812,0.878}}              & {\cellcolor[rgb]{0.62,0.812,0.878}}        \\ 
\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{2-5}\arrayrulecolor{black}\cline{6-6}\arrayrulecolor{black}\cline{7-7}\arrayrulecolor{black}\cline{8-8}
                                        & T3+T4                                                                               & 1.409                                                                          & 1.028-1.931                                                                   & \textcolor{red}{0.033}                                                        & 2.175                                & 1.152-4.106                                      & \textcolor{red}{0.017}                     \\ 
\hhline{>{\arrayrulecolor[rgb]{0.255,0.255,0.255}}|----->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}-|}
\multirow{2}{*}{Clinical N Status}      & {\cellcolor[rgb]{0.62,0.812,0.878}}N0                                               & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1 & {\cellcolor[rgb]{0.62,0.812,0.878}}              & {\cellcolor[rgb]{0.62,0.812,0.878}}        \\ 
\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{2-5}\arrayrulecolor{black}\cline{6-6}\arrayrulecolor{black}\cline{7-7}\arrayrulecolor{black}\cline{8-8}
                                        & N1-3                                                                                & 1.185                                                                          & 0.890-1.577                                                                   & 0.246                                                                         & 1.237                                & 0.864-1.770                                      & 0.245                                      \\ 
\hhline{>{\arrayrulecolor[rgb]{0.255,0.255,0.255}}|----->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}-|}
\multirow{2}{*}{Clinical M Status}      & {\cellcolor[rgb]{0.62,0.812,0.878}}M0                                               & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1 & {\cellcolor[rgb]{0.62,0.812,0.878}}              & {\cellcolor[rgb]{0.62,0.812,0.878}}        \\ 
\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{2-5}\arrayrulecolor{black}\cline{6-6}\arrayrulecolor{black}\cline{7-7}\arrayrulecolor{black}\cline{8-8}
                                        & M1                                                                                  & 4.097                                                                          & 1.009-16.644                                                                  & \textcolor{red}{0.049}                                                        & 8.629                                & 1.879-39.63                                      & \textcolor{red}{0.006}                     \\ 
\hhline{>{\arrayrulecolor[rgb]{0.255,0.255,0.255}}|----->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}-|}
\multirow{2}{*}{Clinical Stage}         & {\cellcolor[rgb]{0.62,0.812,0.878}}Stage I+II                                       & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1 & {\cellcolor[rgb]{0.62,0.812,0.878}}              & {\cellcolor[rgb]{0.62,0.812,0.878}}        \\ 
\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{2-5}\arrayrulecolor{black}\cline{6-6}\arrayrulecolor{black}\cline{7-7}\arrayrulecolor{black}\cline{8-8}
                                        & Stage III+IV                                                                        & 1.245                                                                          & 0.882-1.759                                                                   & 0.213                                                                         & 0.476                                & 0.222-1.024                                      & 0.057                                      \\ 
\hhline{>{\arrayrulecolor[rgb]{0.255,0.255,0.255}}|----->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}-|}
\multirow{2}{*}{Surgical Margin status} & {\cellcolor[rgb]{0.62,0.812,0.878}}Negative                                         & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1 & {\cellcolor[rgb]{0.62,0.812,0.878}}              & {\cellcolor[rgb]{0.62,0.812,0.878}}        \\ 
\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{2-5}\arrayrulecolor{black}\cline{6-6}\arrayrulecolor{black}\cline{7-7}\arrayrulecolor{black}\cline{8-8}
                                        & Positive                                                                            & 1.591                                                                          & 1.155–2.191                                                                   & \textcolor{red}{0.004}                                                        & 1.63                                 & 1.180-2.252                                      & \textcolor{red}{0.003}                     \\ 
\hhline{>{\arrayrulecolor[rgb]{0.255,0.255,0.255}}|----->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}-|}
\multirow{2}{*}{Tobacco Exposure}       & {\cellcolor[rgb]{0.62,0.812,0.878}}Low                                              & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1 & {\cellcolor[rgb]{0.62,0.812,0.878}}              & {\cellcolor[rgb]{0.62,0.812,0.878}}        \\ 
\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{2-5}\arrayrulecolor{black}\cline{6-6}\arrayrulecolor{black}\cline{7-7}\arrayrulecolor{black}\cline{8-8}
                                        & High                                                                                & 1.364                                                                          & 1.008-1.844                                                                   & \textcolor{red}{0.044}                                                        & 1.62                                 & 1.173-2.238                                      & \textcolor{red}{0.003}                     \\ 
\hhline{>{\arrayrulecolor[rgb]{0.255,0.255,0.255}}|----->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}->{\arrayrulecolor{black}}-|}
\multirow{2}{*}{RNA-Seq}                & {\cellcolor[rgb]{0.62,0.812,0.878}}Low                                              & {\cellcolor[rgb]{0.62,0.812,0.878}}1                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}                                           & {\cellcolor[rgb]{0.62,0.812,0.878}}1 & {\cellcolor[rgb]{0.62,0.812,0.878}}              & {\cellcolor[rgb]{0.62,0.812,0.878}}        \\ 
\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{2-5}\arrayrulecolor{black}\cline{6-6}\arrayrulecolor{black}\cline{7-7}\arrayrulecolor{black}\cline{8-8}
                                        & High                                                                                & 0.472                                                                          & 0.351-0.635                                                                   & \multicolumn{1}{c|}{\textcolor{red}{***}}                                     & \multicolumn{1}{l|}{0.459}           & 0.340-0.619                                      & \multicolumn{1}{c|}{\textcolor{red}{***}}  \\ 
\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{1-6}\arrayrulecolor{black}\cline{7-7}\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{8-8}
\multicolumn{8}{|l|}{}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                \\ 
\hline
\multicolumn{8}{|l|}{(\textit{P}~value significant codes is denoted: red \textless{} 0.05; *** \textless{} 0.001)}                                                                                                                                                                                                                                                                                                                                                                                                    \\
\hline
\end{tabular}
}% end of \resizebox
\arrayrulecolor{black}
\label{table:table4}
\end{table}

} % end of \comm{}

% end of result


% from sp1 研究構想： Dr Wu 吳明恆老師 FAT1 versus TP53 **********
%   (Head Neck 2018, Head Neck 2020 and Medicine 2021). (Wu et al., 2020, 2021) Ming-Heng Wu
Wu and his colleagues~\cite{Wu2018a} have studiy for \acrshort{ovh} through whole-exome sequencing to identify a biomarker. 
They found a candidate biomarker for malignant transformation: \acrfull{fat1}.
The \acrshort{fat1} missense mutation rate is higher in progressive \acrshort{ovh} (83.3\%) than in their non-progressive counter partner (22.2\%), while 6.25\% of TP53 is mutated in \acrshort{ovh}. 
Missense mutation of FAT1 is also significantly associated with poor recurrence-free survival (RFS) of \acrshort{oscc}.
%Based on a whole exome-sequencing analysis, we observed that FAT1 somatic mutation rate was strikingly higher in progressive OVH (83.3\%) than in non-progressive ones (22.2\%) and 
%FAT1-regulated YAP1/Hippo pathways has been well addressed. 
%As a result, this  project aims to verify FAT1 gene mutation in a large cohort of OVH patients and to investigate the FAT1 and tp53 relationships in OVH transformation. 
%At the same time, the other ten genes were over-expressed in the better survival patients (with hazard ratio $<=0.5$), named as ZNF557, ZNF266, IL19, MYO1H, FCGBP, LOC148709, EVPLL, PNMA5, IQCN (previous name as KIAA1683), and NPB.

As a result, we believe that FAT1-regulated YAP1/Hippo pathway drives malignant transformation in \acrshort{ovh}. FAT1 study will be further investigated in our subproject 1.
We assume those biomarkers, CK13, CK17, p53, Ki-67, EGFR, $\alpha$-SMA, CAMK2N1, and FAT1, should be found within surgical margin (dysplasia zone) of \acrshort{oscc} specimen and \acrshort{ovh}.
%*** validated by \acrfull{hpa}.
Further investigation will be conducted by using deep learning of whole-slide pathology images.
% are warranted 

%%%%%%%%%%%


%%%%%

%\section*{Figure}

\clearpage

\section{Genome-wide Association Testing}
There are not too many similar studies so far, and they are candidate gene studies~\cite{Shridhar2016, Huang2019a, Chung2019}.
%Huang et al., 2019
Using a microarray-based chip to acquire genotypic data for a large number of genetic variants across the human genome has been cost-effective. Genome-wide association testing is a simple and effective way to find disease-associated genetic variants that is not even constrained by current knowledge of the disorder~\citep{Hosking2011}.


\section{Deep Learning in Bioinformatics}

%why deep learning?
According to the recommendation of \acrfull{nci} (accessed March 2022, at \url{https://www.cancer.gov/\\research/areas/diagnosis/artificial-intelligence}), 
\begin{outline}
\1 Artificial Intelligence (AI) is helpful to create opportunities in cancer research
\1 Image-based phenotype of cancer could be correlated with their underlying genomics data by deep learning methods
%Deep learning could predict commonly mutated genes and mRNA expression from the images
%\1 Aiding the genomics characterization of tumors by AI is crucial
\end{outline}

%== NCI deep learning could predict commonly mutated genes from the images:
Deep learning (advanced machine learning algorithm in AI) can be used to identify specific gene mutations and mRNA expression from hyperplasia/tumor pathology images instead of using traditional genomic sequencing. 
For instance, NCI-funded researchers~\citep{Coudray2018} at New York University used deep learning to analyze pathology images of lung tumors obtained from \acrfull{tcia}.
% Michael Snyder, Ph.D., chair of genetics at Stanford University, sees AI as the future of diagnosis. “I think we need to shift to using machine learning rather than rely on pathologists alone to do all the work,” he said. “Algorithms won’t replace pathologists, but they will assist them in making classifications. They will reduce the errors that pathologists would otherwise make.”
%The code developed for the project is available for other researchers to use for other diagnostic applications. The NYU team has already started applying the code to learn to diagnose kidney, breast, and other cancers.
The \acrshort{tcia} archives pathology images of tumor specimens available as a quality control measure for researchers studying the genetic sequence data collected in the \acrshort{tcga} project.
Coudray and his colleagues found that the deep learning method could accurately distinguish between lung adenocarcinoma and lung squamous cell carcinoma from the images~\citep{Coudray2018}.
%how machine learning could help us?
% OFA:   The Cancer Imaging Archive (TCIA) whole-slide by Deep learning (ResNet)
%https://www.deeplearningbook.org/
%https://www.biorxiv.org/content/10.1101/2020.03.10.985887v1.full
%https://github.com/Serian1992/ImgBio.
%https://www.cancer.gov/research/areas/diagnosis/artificial-intelligence
Since a deep learning algorithm could potentially extract molecular phenotype from pathology images, 
mRNA expression could also be predicted from the images.
A HE2RNA model, developed by Schmauch and his colleagues~\citep{Schmauch2020}, could predict RNA-Seq profiles from whole-slide images of \acrshort{tcia}. 
It has been validated by CD3- and CD20-\acrshort{ihc} staining. 
The transcriptomic representation learned by HE2RNA can also be transferred on other datasets (i.e., transfer learning).
%MKI67 is a well-known marker of cell proliferation, expressed both by tumor and adjacent normal cells, clinically determined by MIB1 or Ki67 IHC staining of whole-slide images. 
%  its surrounding cancerous area 
%Its overexpression around "surgical margin?" has been found to correlate with tumor growth, histological grade, and tumor recurrence.
% A high expression level of MKI67 is correlated with the most advanced stages of liver cancer.






\section{Conclusion}

Predictive markers are those that can predict cancer risk even with proper intervention. We will find them by comparing pre-cancerous samples to outcomes that suggest malignant transformation. 

Through a genome-wide approach, we aim to uncover genetic variations linked with \acrshort{ovh} malignant transformation and examine potential connections between genetic and environmental risk factors in this subproject 3.

High-throughput RNA-sequencing (RNA-Seq) is a powerful tool to characterize and quantify transcriptomes. Even the cost of RNA-Seq is continuously dropping, 
the budget of whole-genome transcriptomics study for survival analysis is still very high (USD350 to 500 per sample). %; 46 participants of HNSCC in TMU Joint Biobank, accessed on \today at http://ohr.tmu.edu.tw/front/jbb1/CaseNumber/pages.php?\\ID=dG11X29ociZDYXNlTnVtYmVy).
% NCI HE2RNA
By using the transcriptomic representation of RNA-Seq learned by NCI's HE2RNA project~\citep{Schmauch2020}, ST-Net (He et al., 2020) and HistoGene (Pang et al., 2021),
%TCIA for whole-slide deep learning
we could find and validate predictive biomarkers from pathology images by deep learning platform under R/Pytorch/Keras framework.
Finally, using the risk factors acquired from all subprojects, this subproject will construct risk prediction models for malignant transformation in \acrshort{ovh} patients.\\[2.5cm]
%To further improve the quality of spatial transcriptomics data, methods have been developed to leverage other data types in addition to scRNA-seq data. For example, xFuse [49] is a deep generative model that integrates in situ RNA capturing data with histology image data to infer transcriptome-wide expression maps. 
%The quantification of gene expression both within and between the original capture locations enhances the resolution of spatial transcriptomics data. 
%HistoGene [62] is another deep learning model to leverage information learned from spatial transcriptomics data to predict gene expression for tissue sections where only histology images are available. HistoGene [62] outperforms other approaches that were designed to predict gene expression profiles from whole-slide images, including % **a multilayer perceptron-based method HE2RNA [79] and 
% **a supervised convolutional neural network-based approach ST-Net [80].

\clearpage



%%%%%%%%%%%%%%%%%

\section*{\cntext{(二)研究方法、進行步驟及執行進度}}
%1.本計畫採用之研究方法與原因。2.預計可能遭遇之困難及解決途徑 fallback approaches
% ***執行的策略
% 科技部鼓勵研究主題的系統性，也鼓勵具研究能力者從事較大型但 可能是較困難的長期研究。在建立研究架構的同時，要審慎考慮如何適當分配 每一年的執行進度。例如，容易有具體成果的部分可能要先放在前面作為基 礎，較困難的部分便可以逐年按部就班地進行調查分析。


% since 2022/05/10
\begin{outline}

Specific Aims %特定研究目標 (specific aims), 執行策略不能互相綁在一起, go paralleling, and equally balanced
\1 Aim 1: \aimone
% 祁力行醫師與吳家佑醫師協助建立北醫OPMD cohort(白斑和OVH為主)、IRB申請及切片染色影像AI開發，收案檢體包含:切片、血漿、口水、冷凍組織、buffy coat，建議收案時間為手術後確認非OSCC再與病患溝通收案，收案人力需再協調
%TCGA/TCIA and TMU dataset\\
%Survival Analysis\\

\1 Aim 2: \aimtwo
%To identify genetic variants associated with malignant transformation of \acrshort{ovh} by implementing genome-wide association testing (GWAS)


%王紋璋老師需要buffy coat檢體抽DNA進行GWAS分析，一片chip可做96檢體，希望盡量湊足200個檢體的量來做分析。

\1 Aim 3: \aimthree
%To construct a deep learning platform of genomics (mutation and expression) from pathology images

\1 Aim 4: \aimfour
%To develop risk prediction model for malignant transformation of \acrshort{ovh} by integrating risk factors identified from all sub-projects: sp1, sp2, and sp3

\end{outline}

%\clearpage
%\subsection*{\cntext{執行進度}} % Flow Chart => Gantt chart
% A flowchart of a TeX workflow
% Author: Stefan Kottwitz
% https://www.packtpub.com/hardware-and-creative/latex-cookbook
%https://www.overleaf.com/project/60a23af9c83411b0f9067470
% https://texdoc.org/serve/smartdiagram/0
%\begin{document}



% A simpler example from the package documentation:
%
%\begin{ganttchart}{1}{12}
%  \gantttitle{2021}{12} \\
%  \gantttitlelist{1,...,12}{1} \\
%  \ganttgroup{Group 1}{1}{7} \\
%  \ganttbar{Task 1}{1}{2} \\
%  \ganttlinkedbar{Task 2}{3}{7} \ganttnewline
%  \ganttmilestone{Milestone}{7} \ganttnewline
%  \ganttbar{Final Task}{8}{12}
%  \ganttlink{elem2}{elem3}
%  \ganttlink{elem3}{elem4}
%\end{ganttchart}

\begin{figure}
    \centering
%    \includegraphics{}

\begin{ganttchart}[
    bar label node/.style={text width=5cm,align=right,font=\scriptsize\RaggedLeft,anchor=east},
    expand chart=\textwidth,
    x unit=0.8cm,
    y unit title=0.4cm,
    y unit chart=0.6cm, %0.5
    vgrid,
    time slot format=isodate-yearmonth, %compress calendar,
    time slot unit=month,
    title/.append style={draw=none, fill=barblue},
    title label font=\sffamily\bfseries\color{white},
    title label node/.append style={below=-1.6ex},
    title left shift=.05,
    title right shift=-.05,
    title height=1,
    bar/.append style={draw=none, fill=groupblue},
    bar height=.6,
    bar label font=\normalsize\color{black!50},
    group right shift=0,
    group top shift=.6,
    group height=.3,
    group peaks height=.2,
    bar incomplete/.append style={fill=green}
   ]{2022-01}{2023-12} % {2026-12}
   
   \gantttitlecalendar{year, month}\\ %=name
   \ganttbar[
    progress=10,
    bar progress label font=\small\color{barblue},
    bar progress label node/.append style={left=4pt},
    bar label font=\normalsize\color{barblue},
    name=pp
   ]{IRB application}{2022-06}{2022-08} \\ % name=pp
   
\ganttset{progress label text={}, link/.style={black, -to}}

% aim1
% \textcircled{\raisebox{-0.9pt}{1}}
\ganttgroup{Aim 1}{2022-09}{2023-12} \\
%\ganttbar[progress=0, name=T1A]{Construct \acrshort{ovh} and leukoplakia cohorts}{2023-01}{2023-01} \\
% ganttlinkedbar
\ganttbar[progress=30, name=T1A]{Retrospective cohorts \textcircled{\raisebox{-0.9pt}{A}} and \textcircled{\raisebox{-0.9pt}{B}}
}{2022-09}{2023-07} \\
\ganttbar[progress=0, name=T1B]{Prospective cohort \textcircled{\raisebox{-0.8pt}{C}}}{2022-10}{2023-12} \\

%\ganttnewline

% aim2
\ganttgroup{Aim 2}{2022-9}{2023-12} \\
\ganttbar[progress=2, name=T2A]{Implementing genome-wide association testing (GWAS)}{2022-9}{2023-12} \\ % 因Covid-19，臨床研究資料庫獨立作業區2021.05.19暫停服務

\ganttnewline
%\ganttlinkedbar[progress=0, name=T2B]{Development of source code (RNN)}{2023-02}{2023-12} \\


%\ganttnewline
% aim3  of genomics $\&$ (mutation and expression)
\ganttgroup{Aim 3}{2022-09}{2023-12} \\
  \ganttbar[progress=0, name=T3A]{Construct a deep learning platform for pathology images}{2022-09}{2023-12}
  \ganttnewline
%  \ganttnewline
  
% aim4
\ganttgroup{Aim 4}{2023-7}{2024-2} \\ %[group top shift =2] to avoid vertically overlap of bars
  \ganttbar[progress=0, name=T4A]{Integration data $\&$ development of risk prediction models}{2023-07}{2024-2}
  % , bar top shift =1.8
  

% linkers
% ***[link bulge=1, link mid=0.3] H and V

% IRB
  \ganttset{link/.style={green}}
  \ganttlink[link mid=.3]{pp}{T1A}
  \ganttlink[link mid=.3]{pp}{T1B}
  
\ganttset{link/.style={black}}
  \ganttlink[link mid=0.54]{T1A}{T2A}
  \ganttlink[link mid=0.3]{T1B}{T2A}
\ganttset{link/.style={brown}}
  \ganttlink[link mid=0.71]{T1A}{T3A}
  \ganttlink[link mid=0.65]{T1B}{T3A}
\ganttset{link/.style={red}}
  \ganttlink[link mid=0.72]{T2A}{T4A} % .159
  \ganttlink[link mid=0.3]{T3A}{T4A}
%  \ganttlink[link mid=.159]{T2A}{T4A}
\end{ganttchart}

    \caption{\cntext{執行進度} (OVH and leukoplakia cohorts from \textcircled{\raisebox{-0.9pt}{A}} Chi-Mei Hospital, \textcircled{\raisebox{-0.9pt}{B}} and \textcircled{\raisebox{-0.8pt}{C}} Taipei Medical University Hospital)
%    \cntext{(*因Covid-19 pandemic in Taiwan，臨床研究資料庫獨立作業區2021.05.19起暫停服務)}
}
    \label{fig:gantt}
\end{figure}

%\clearpage

\subsection*{\cntext{計畫主體內容}} 
%[proposal main body] %(計畫的主體內容)” 解釋每個 aim 要怎麼做，著重策略之邏輯性鋪陳，盡量使論述具有說服力。
%新創一個方法或技術平台，須要詳細交代步驟

% *****specific Aims (SP3 = old SP3 + old sp4)	
\begin{minipage}[t]{5.5in}
\vspace{0pt}
\begin{itemize}
        
% These are the instructions for the class
\item \textbf{Aim 1}: \aimone, 
with collection of clinical data and bio-specimen (i.g., pathological specimen, saliva, and serum).
% buffy coat; frozen specimen and paraffin block


\item \textbf{Aim 2}: \aimtwo. 
% (GWAS) Dr Wang
And also investigate the potential interactions between genetic and environmental risk factors.

\item \textbf{Aim 3}: \aimthree.
From \acrshort{ovh} and \acrshort{oscc}, a deep-learning-based algorithm could extract clinical features and interpret pathology whole-slide images.
%A deep-learning-based algorithm might extract clinical features, and read pathology whole-slide images from \acrshort{ovh} and \acrshort{oscc}.

%Deep Learning-based biomarker identifying in depression analysis with holistic healthcare study. The cortisol is one of the most common objective outcomes studied in psychoneuroimmunological (PNI) factors~\citep{Hulett2016}. Interleukin, cytokines, T-lymphocytes ($CD4^+$, $CD8^+$), $CD56^+$ (neural cell adhesion molecule), and heart rate variability (HRV) are included in other PNI studies.
%The subjective psychosocial measurements should include depression, stress, quality of life, anxiety, fatigue, mindfulness, mood, and specific spiritual growth---the meaning of life~\citep{Hsiao2012}. Our hypothesis is that \acrshort{ovh} could be transformed into oral cancer because of the stress, via mind-brain-body network~\citep{Rogers1959}. Thus, besides using those serum biomarkers, we encourage physicians to write their \acrfull{ehr} in a "holistic way".	And the \acrshort{ovh} carriers should use wearable device (e.g., smart watch) to monitor their HRV. Once the \acrshort{ovh} becoming oral cancer, we could early treat the disease as well as analyze the contributing factors of carcinogenesis. That could be a predictive biomarker.


\item \textbf{Aim 4}: 
\aimfour

%\item \textbf{Simplify all answers as much as possible.} This means that you need to need to combine like terms, reduce fractions, etc. (You do not need to rationalize denominators.)





\end{itemize}



We expect to improve risk prediction models that will aid clinical care for patients with \acrshort{ovh} in terms of cancer prevention and early diagnosis.

\end{minipage}

\vspace{3em}
%%%%%%%%%

%\begin{outline}
% Dr Rich Wu and Dr Chi

\subsubsection*{Aim 1:} Construct \acrshort{ovh} and leukoplakia datasets\\[0.5cm]


%\end{outline}
%central issue 1 with preliminary data (key data); fallback approaches %先以概要式的背景介紹破題，點出研究目標的重要性，然後用邏輯之陳述將如何達成目標的策略和方法講清楚。請務必注意：每個研究目標基本上需具備其獨立性 (Independence)，但其重要性卻要在目標之間有所連結，且要能回應總體目標之研究顯著性


%\clearpage

% Dr Chi
\paragraph*{Cohort collection} %Construct %Chi-Mei and TMUH
by two study designs -\\[0.5cm]

% \subparagraph{Study design for cohort (A) and (B)}

\pagebreak
% pathology request form (or a protocol?)
\begin{outline}
%\0 Hargeisa Group Hospital \\
\0 Pathology Test Request Form

(Please complete and submit)

\1 PATIENT DETAILS
    \2 Name:
    \2 Gender: \rule{1cm}{0.15mm} Male/ \rule{1cm}{0.15mm} Female/ \rule{1cm}{0.15mm} Prefer not to say
    \2 Date of birth:
    \2 Mobile:
    \2 Email:
    \2 Residential address:
    \2 Travel history:
    
\1 CLINICAL INFORMATION
   \2 Clinical (Provisional) Diagnosis
   \2 Procedure (for specimen collection)
        \3 \rule{1cm}{0.15mm} aspiration cytology (fine/core needle)
        \3 \rule{1cm}{0.15mm} incisional biopsy
        \3 \rule{1cm}{0.15mm} excisional biopsy
   \2 Collection Date/Time
   \2 Organ or Tissue (of specimen)
        \3 (ie, breast, lung, soft tissue, oral cavity)
        \3 \rule{6cm}{0.15mm}
%   \2 Findings during operation
   \2 Brief History of Present Illness

%\1 REFERRER DETAILS; GP/REFERRER
\0 Submitting Physician
    \1 Name:
    \1 Contact number:
    \1 Email:
\0 $=>$ Once pathology report is available, we will send it via your email, WhatsApp, and short message (SMS).\\

SIGNATURE \rule{4cm}{0.15mm}
\vspace{5mm}

DATE \rule{5cm}{0.15mm}

%\1 PAYMENT DETAILS (IF KNOWN)

\end{outline}



\pagebreak


%Ethics Statement\\
%***TMU Joint IRB approval\\
First retrospective cohort will applied from biobank of Chi-Mei Hospital (CMH) as cohort (A). Data should include clinical features and bio-specimen (i.g., pathological specimen, saliva, and serum).
The \acrshort{ovh} and leukoplakia samples will be obtained from 150 consecutive participants at Chi-Mei Hospital. 
Written informed consent was obtained from all participants and was also reviewed by the Institutional Review Board Committee of CMH. 
Tissues, saliva and, serum were preserved in liquid nitrogen for DNA extraction.
Formalin-fixed paraffin embedded tissues (FFPE) will be used for pathological imaging.


Second retrospective cohort (B) will retrieve the clinical data and FFPE from \acrshort{tmuh}. Data analysis will be waived review and consent by the TMU Joint Institutional Review Board (IRB), as all data will being analyzed after de-identification.
%Clinical samples were carried out with the approval of the Institutional Review Boards. 
% Written informed consent was obtained from all participants and was also reviewed by TMU joint-IRB Committee.
%\textcircled{\raisebox{-0.9pt}{A}} and \textcircled{\raisebox{-0.9pt}{B}}
The cohort (B) contains total 200 patients including xx men and xx women with their mean age at 53 year-old, range 30-88 years. The xx patients were diagnosed in non- or mild, and xx in moderate/sever oral epithelial dysplasia.
Pathological imaging will be performed on FFPE samples.
%%

\subparagraph*{Study design for cohort (C)}
%  A STROBE-compliant retrospective/prospective cohort study.
% a good methodology: 1. Crawford M, Johnson EH, Liu KYP, Poh C, Tsai RYL. On the Cutting Edge of Oral Cancer Prevention: Finding Risk-Predictive Markers in Precancerous Lesions by Longitudinal Studies. Cells. 2022;11(6). doi:10.3390/cells11061033
According the STROKE statement~\citep{VonElm2008} for observational studies, one prospective cohort (C) study will be conducted.


% 祁力行醫師與 ***吳家佑醫師協助建立北醫OPMD cohort(白斑和OVH為主)、IRB申請及切片染色影像AI開發，
%收案檢體包含:切片、血漿、口水、冷凍組織、buffy coat，建議收案時間為手術後確認非OSCC再與病患溝通收案

% 1. Liu H, Liu X wei, Dong G, et al. P16 Methylation as an Early Predictor for Cancer Development From Oral Epithelial Dysplasia: A Double-blind Multicentre Prospective Study. EBioMedicine. 2015;2(5):432-437. doi:10.1016/j.ebiom.2015.03.015
% oral epithelial dysplasia (OED) 
Between 2023 and 2025, 200 patients will be chosen from cases of oral leukoplakia, verrucous hyperplasia, or lichen planus at \acrshort{tmuh}. Biopsy specimens are fixed in neutral buffered formalin, paraffin embedded, and H\&E stained. At least two senior pathologists should use criteria from the 2017 WHO Classification System to classify oral epithelial dysplasia of baseline lesions as mild, moderate, or severe~\cite{Gale2005, Ranganathan2019}. The sample size was calculated according to a suggestion from the prospective study~\cite{Cao2009}.
%Peking University School of Stomatology (Central-A, n = 82), Capital Medical University School of Stomatology (Central-B, n = 68), and Fourth Military Medical University Hospital of Stomatology (Central-C, n = 31) 
All of the participants have initial lesions that have never been treated with surgery, LASER, radiation therapy, or chemotherapy. Clinical features, lesion site, cigarette smoking, alcohol consumption, betel-nut chewing, and past medical history are all gathered. For at least three years, follow-up examinations will be conducted every three months in a double-blind fashion. If the clinician suspects a malignant event, a further examination and re-biopsy will be performed. Patients who do not require a re-biopsy during the follow-up examination period because of baseline lesion has clearly disappeared/regressed. TMU Joint Institutional Review Boards will approve the trial (TMU-IRB 111xxx), and all patients will provide written informed consent. This study will be also registered in the U.S. National Institutes of Health Clinical Trials Protocol Registration System in accordance with the criteria outlined by the International Committee of Medical Journal Editors (trial number NCTxxxxxxxx, available at \url{http://ClinicalTrials.gov}).

%% %收案檢體包含:切片、血漿、口水、冷凍組織、buffy coat
 %\textcircled{\raisebox{-0.8pt}{C}}
Data should include clinical features and bio-specimen (i.g., pathological specimen in fresh state, saliva, and serum).
All experimental protocols will be applied for approval under the IRB protocol
with TMU Joint Institutional Review Board, and all experiments will be carried out in accordance with approved guidelines.
%In three cohorts, the patient ages ranged from 25 to 81 (average age 58).
%Patient specific information pertaining to race, tumor grade, and outcome were not explicitly recorded for this study.




%%%

%\subparagraph*{Patient and tissue samples} % tissue microarray
%% KVGH-IRB-XXXXX and TMU-IRB 99049; Written informed consent was obtained from all participants and was also reviewed by IRB Committee.
% (demographic data in Supplementary Table S1)


%\subparagraph*{Serum sample preparation}
%for GWAS


\subparagraph*{Tissue sample preparation} % for HE and IHC

The fresh frozen tissue samples (CMH cohort (A) and TMUH cohort (C)) are maintained with the desired orientation on the cutting block using Optimum Cutting Temperature (OCT) polymer. A thin frozen section is cut at -$15\,^{\circ}\mathrm{C}$ by using a cryostat (Thermo Electron) and deposited on cold glass slides. The slide and tissue section are then quickly warmed, which let the tissue thaw-mounted onto the slide. Tissue sections are fixed by immersion in 70\% then 95\% ethanol bath each for 30 sec and allowed to dry for 40 min in a desiccator.
%indium tin oxide (ITO)-coated conductive  (Bruker Daltonics)
% tissue microarray

Formalin-fixed paraffin embedded tissues (FFPE), obtained from cohort (A)(B)(C), are organized into block for H\&E staining as usual manner.
%a set of serial tissue microarray (TMA) blocks while each samples contained one normal part and three tumor part derived from different tumor regions to represent its heterogeneity.  
%All participants underwent a series of clinical evaluations then to be treatment according to NCCN guidelines~\citep{Pfishter2019}. The clinicopathological data was carefully recorded. The clinical staging was registered based on the American Joint Committee on Cancer (AJCC) TNM classification of \acrshort{hnscc}.
%The survival analysis was evaluated by the Kaplan--Meier estimates and compared by the log-rank test.
%%

\subparagraph{Immunohistochemistry (IHC) staining}

% FFPE tissue microarray (from TMUH cohort)
The blocks of FFPE are sectioned at 3-\si{\um} slice thickness. Slides are heated for 1 hour at $60\,^{\circ}\mathrm{C}$ to deparaffinize in xylene, rehydrated in graded alcohol, and rinsed in distilled water. Antigen retrieval and immunohistochemistry staining are performed by using a Benchmark XT automated immunohistochemistry system (Ventana Medical Systems). 
Slides are stained with primary antibodies at 1:200 or 1:400 dilution. Primary antibodies are detected by using an indirect biotin streptavidin system (iVIEW DAB Detection Kit, Ventana Medical Systems, Inc.) and tissue is counterstained in haematoxylin and eosin. Images of stained slides are retrieved by digital scanner (ScanScope CS2 Systems Aperio, CA, USA) and diagnosed and graded manually by one pathologist, Dr. Ling-Cheng Mong.

% primary TMSB4X antibodies (Proteintech)
% % 抗體 (10= 9 + FAT1)
We assume those biomarkers, CK13, CK17, p53, Ki-67, EGFR, $\alpha$-SMA, CAMK2N1, FAT1, p16INK4a, and p14ARF, should be used in \acrshort{ihc} staining.
%found within surgical margin (dysplasia zone) of \acrshort{oscc} specimen and \acrshort{ovh}.
% p16INK4a and p14ARF for (PVL) proliferative verrucous leukoplakia

%%%
\paragraph*{Whole-slide imaging} Construct a deep learning platform of Taiwan datasets %\\[0.5cm]

% **** 病理切片 去背
% TCGA/TCIA and
A quantitative tissue pathology (QTP) approach based on digital whole-slide images has enabled the profiling of a wide range of microscopic features at tissue, single cell and subcellular levels that are not visible to the naked eye. These phenotypes could be a reflection of its genetic carcinogenesis~\cite{Crawford2022}.

% pickup severe one as a representative 挑最嚴重的為代表
There is 3300 slides from 300 patients with \acrshort{ovh} or leukoplakia in datasets (A)(B)(C).
%Slide Digitization and Pathologists Ground Truth for TMU dataset:
The 300 H\&E stained pathology slides will be digitized via a whole-slide scanner at 40x magnification for this study, with 0.25 $\mu$m/pixel resolution (Aperio Scanscope CS2).
The other 3000 slides with \acrshort{ich} stained (300 per antibody) will be scanned as the same manner.
The SVS file will be obtained.
%a digital slide image file created by an Aperio ScanScope slide scanner.
%The Ventana iCoreo scanner was used for scanning the UHCMC/CWRU and NC data cohorts. 
%under 40x magnification corresponds to Aperio's slides at 0.25 $\mu$m/pixel resolution.
%and to Ventana’s slides at 0.23 μm/pixel.

%https://wiki.cancerimagingarchive.net/display/Public/TCGA-HNSC
%https://cancer.digitalslidearchive.org/#!/CDSA/hnsc/TCGA-BA-4076/TCGA-BA-4076-01A-01-BS1?slide=TCGA-BA-4076-01A-01-BS1.4aef481c-f634-4ffc-9fb3-609f59a8caf9.svs

NCI's \acrfull{gdc} stores SVS image of TCGA's HNSCC pathology slide (we transferred and saved them at \url{https://github.com/texchi2/HE2RNA_code/blob/master/gdc_manifests/gdc_manifest.2018-06-26_diagnostic_TCGA-HNSC.txt}).


%https://portal.gdc.cancer.gov/image-viewer?caseId=b843df7e-db06-4d29-b00d-6a93effd9191&filters=%7B%22op%22%3A%22and%22%2C%22content%22%3A%5B%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22cases.project.project_id%22%2C%22value%22%3A%5B%22TCGA-HNSC%22%5D%7D%7D%2C%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22files.experimental_strategy%22%2C%22value%22%3A%5B%22Tissue%20Slide%22%5D%7D%7D%5D%7D&selectedId=45fffd2f-e282-4198-8c49-0247507ebb8a
% HE2RNA
%*** github: https://github.com/owkin/HE2RNA_code

%the ability of HE2RNA to focus specifically on cancer-related molecular information\citep{Schmauch2020}.
% detailed introduction
%*** A deep learning model to predict RNA-Seq expression of tumours from whole-slide images\cite{Schmauch2020}.
%A HE2RNA model should be trained to systematically predict RNA-Seq profiles from whole-slide images\cite{Schmauch2020}.
% . The application of deep-learning algorithms to histological
The whole-slide image of pathology slide has high dimensionality (up to 100,000 × 100,000 pixels for a single image). It could be analyzed by deep learning algorithm.



\clearpage

%%%%
%%%%%%%%%%%%%%%%
% Dr Wang's GWAS
\subsubsection*{Aim 2:}
To identify genetic variants associated with malignant transformation
of OVH by GWAS\\[0.5cm]
%Use genome-wide association study (GWAS) to test genotype alternations\\[0.5cm]
%To identify genetic variants associated with malignant transformation of \acrshort{ovh} by implementing genome-wide association testing (GWAS)

%王紋璋老師需要buffy coat檢體抽DNA進行GWAS分析，一片chip可做96檢體，希望盡量湊足200個檢體的量來做分析。

\paragraph{Genotype Data}
The GWAS will be conducted by using Axiom Genome-Wide TWB 2.0 Array Plate (Thermo Fisher Scientific, Inc., Santa Clara, CA,USA), which contains 686,463 markers~\citep{Wei2021}. The genotyping will be implemented by the Taiwan National Center for Genomic Medicine (\url{http://ncgm.sinica.edu.tw/affymetrix_prep_requirements_c_01.html}). The genotypic data quality control will be carried out from two aspects. Firstly, markers with high proportion of missing genotype ($> 0.05$), significant deviation from Hardy-Weinberg equilibrium (\textit{p} $< \num{10E-4}$), or low minor allele frequency ($< 0.01$) will be excluded. Then subjects with high proportion of missing genotypes ($> 0.01$), sex error, or contamination will be excluded. Furthermore, based on the cleaned genotyped data, genotype information at untyped marked will be deduced by genotype imputation through the Michigan imputation server ~\citep{Das2016}. For imputed data, markers with low imputation score ($< 0.3$), high proportion of missing genotype ($> 0.05$), significant deviation from Hardy-Weinberg equilibrium (\textit{p} $< \num{10E-4}$), or low minor allele frequency ($< 0.01$) will be excluded. 

\paragraph{Association testing of individual markers} % confounders: dysplasia grading
The phenotypes considered in our GWAS include "whether malignant transformation occur" and "the time from detection of OVH to malignant transformation" and the logistic regression and Cox proportional hazard model will be used to test for associations of genetic markers with these two phenotypes, respectively. For both phenotypes, the test of genetic association will be adjusted for gender, age of \acrshort{ovh} detection, betel-nut chewing, smoking, alcohol drinking, advanced oral submucous fibrosis, non-buccal and non-tongue lesions, dysplasia grading, as well as the top ten principal components. Computation of the principal components is based on cleaned genotypic data through implementing principal component analysis. 

\paragraph{Gene-environment interaction effects}
In addition to test for association of individual genetic markers, we would like to investigate gene-environment interaction effects on the phenotypes. Here, the environmental factors examined include betel-nut chewing, smoking, and alcohol drinking. Given a specific environmental factor, the gene-environment interaction effect will be examined by two approaches as follows. 
In the first approach, an interaction variable of the specific environmental factor and genetic marker will be used as an additional covariate in above logistic regression and Cox proportional hazard model. Then we look for marker with significant interaction variable. 
In the second approach, we will perform a stratified analysis. Specifically, the whole study sample is separated into two groups: one group is exposed to a specific environmental factor; the other group is not exposed to this specific environmental factor. The logistic regression and Cox proportional hazard model, which test for association of individual markers, will be implemented in each group. Then we look for marker whose results of association testing are significantly different in these two groups. 

\paragraph{Validation}
Significant associations observed at genotyped or imputed markers will be validated by re-genotyping through different genotyping platform such as TaqMan or sequenom. 

%%%%%%%%%%%%%%%%%



\clearpage


%%%%%%%%
%\begin{outline}
% Dr Chi Transfer learning
\subsubsection*{Aim 3:} Construct a deep learning platform to predict genotype alternations \\[0.5cm]

\paragraph{Transfer learning in Artificial Intelligent (AI)}
% 觸類旁通
% deep learning model building

%The small size of available datasets. => small case number
A NCI's HE2RNA workflow could divide the whole-slide images into squares of 112 × 112 $\mu$m (224 × 224 pixels) called "tiles", and use the Otsu algorithm~\citep{Otsu1979} (as implemented in python package scikit-image, available at \url{https://scikit-image.org/docs/dev/api/skimage.html}) to select only those containing tissue, excluding the white background. 
A maximum of 8000 such tiles from each slide are sampled then extracted 2048 features with a 50-layer ResNet model~\citep{He2016a} pretrained on the ImageNet dataset (available at \url{https://www.image-net.org/challenges/LSVRC/index.php}).
The ResNet and HE2RNA model are developed by using the TensorFlow/Keras implementation with a architecture of \acrfull{cnn}.
%Graph convolutional neural network (GCNN).
This AI-driven computer program analyzes tissue by creating a map of thousands of tiles.
A pathology slide could be represented as a 8000 × 2048 matrix.
%For the first phase of this work (transcriptome prediction), we accelerated the training of our models through a simple preprocessing step inspired by simple linear iterative clustering (SLIC)65: 
THe k-means algorithm (as implemented in python package libkmcuda) could be used to create 100 clusters (supertiles) of tiles based on their location on the slide, and HE2RNA averages the features of the tiles within each cluster. 
Use of supertiles could reduce the dimension of a slide to 100 x 2048 pixels.
% training supertiles
Transcriptome prediction model will be first trained on this reduced dataset, with all the TCGA data. Then, fine-tuning of "hyperparameters" will be achieved with full-scale data from \acrshort{hnscc} dataset by Bayesian optimization technique (i.g., Sequential Model-Based Optimization algorithms, SMBO)~\citep{Hutter2011}.
The similar procedure of image process will be applied on datasets (A)(B)(C).\\[0.5cm]

%Graph convolutional neural network (GCNN) will be used for feature extraction from whole-slide images.

%%%%%

% x construction of gene expression network.
%GNN graph neural network for CNN convolutional neural network
%The Cancer Imaging Archive (TCIA) whole-slide by Deep learning (ResNet)
% *** AI: deep learning, machine learning \citep{Huang2020}
%plot http://iltabiai.github.io/tips/latex/2015/09/15/latex-tikzdevice-r.html
%***Deep learning-based cancer survival prognosis from RNA-Seq data: approaches and evaluations
%*transfer learning from TCGA model
%torchvision, NanoMets Models
%save checkpoint, load then deepcopy()
% trying in https://colab.research.google.com/drive/1byj2IxxHayVT62GSlkpofSKsd4dr7niS#scrollTo=j-NgvaUbVdBh

% *** HE2RNA has a file containing transcriptomes (RNA-Seq) matched to whole-slide images (available at
%https://github.com/texchi2/HE2RNA\_code/blob/master/transcriptome\_data.py).


TCGA/TCIA modeling (with RNA-Seq data):\\[0.2cm]
%Y (RNA-seq) ← X (slide represents gene expression)
$Y_{mrna} = \beta_0 + \beta_1 X_1 + \beta_2 X_2 + \beta_3 X_3 + ... + \beta_n X_n + \epsilon$\\[0.3cm]
While 
$Y_{mrna}$ is known RNA-Seq data, and
$X_1, X_2, ...X_n$ are the features extracted by HE2RNA model from TCIA whole-slide images of pathology.\\[0.5cm]
%Gene expression is not an independent $X_{mrna}$. It could be correlated with other (for example)\\ $X_{body}, X_{stress}, X_{fear}, X_{meaningOfLife}, X_{inflammation}, X_{empathy}$ factors.\\[1cm]

%*transfer learning from TCGA model
%torchvision, NanoMets Models
%save checkpoint, load then deepcopy()
We transfer HE2RNA model on dataset (A)(B)(C) to predict their RNA-Seq representatives:\\[0.2cm]
%Y (survival) ← X (slide represents gene expression)
$\hat{Y}_{mrna} = \beta_0 + \beta_1 X_1 + \beta_2 X_2 + \beta_3 X_3 + ... + \beta_n X_n + \epsilon$\\[0.3cm]
While $\hat{Y}_{mrna}$ is predicted RNA-Seq data, and
$X_1, X_2, ...X_n$ are the features extracted from cohorts (A)(B)(C) whole-slide images of pathology.\\
%CAMK2N1 is one of our suggested candidate marker (through TCGA survival analysis~\citep{Chi2021}).
The deep learning model of HE2RNA, which was trained under TCGA/TCIA \acrshort{hnscc} dataset, could be applied for RNA-Seq predictive of TMUH/CMH datasets. 
% HE2RNA
The representatives will be used for subsequent gene expression analysis (Figure~\ref{fig:HE2RNA}).
% HE2Displasia
A deep learning model, HE2Displasia, will also be trained for digital grading system for oral epithelial dysplasia.
% HE2DNA
A deep learning model, HE2DNA, will also be trained for a representative of DNA mutation.
Those DNA/RNA representatives will be used for subsequent gene mutation/expression analysis.
% ***We want to validate that IL19 still has survival impact on TMU \acrshort{hnscc} dataset (please see Figure \ref{fig:HE2RNA}).

%\subsubsection{Figure HE2RNA}
%\smartdiagramset{uniform sequence color=true}
% sequence diagram; flow diagram:horizontal
% 2021

% Figure 5 HE2RNA workflow
\begin{figure}[hp]
\centering
%\setcounter{figure}{0}
%\setcounter{diagram}{4}
%\newcounter{diag} % as zero
\setcounter{diag}{\value{figure}} % for floating environment

%\begin{center}
%\smartdiagram[sequence diagram]{2021/July, Aug, Sep, Oct, Nov, Dec}
\vspace*{40mm}
\smartdiagramset{
    module x sep=5cm,
%    module minimum width=3cm,
    text width=3cm,
    back arrow disabled=true,
    border color=none,
    arrow tip=to,
    additions={
additional item bottom color=orange!60!yellow,
   additional item border color=gray,
   additional item shadow=drop shadow,
   additional item offset=1cm,
   additional arrow line width=2pt,
   additional arrow tip=to,
   additional arrow color=orange!60!yellow,
    }%,
   %set color list={blue!50!cyan,green!60!lime,orange!50!red,red!80!black},
}

%\smartdiagramset{module x sep=5}
%\smartdiagram[flow diagram:horizontal]{TMU cohort (WSI/EHR), HE2RNA, RNA-Seq}
\smartdiagramadd[flow diagram:horizontal]{
 TMUH/CMH datasets (WSI), HE2RNA, RNA-Seq (Representatives)
}{
    above of module2/Model (transfer from TCIA),
    below of module3/%Survival Analysis 
} % addition x2
\smartdiagramconnect{to-}{module2/additional-module1}
\smartdiagramconnect{-to}{module3/additional-module2}

% \acrshort{rnn}, Survival Analysis
\vspace*{10mm}
\smartdiagramadd[flow diagram:horizontal]{
 TMUH/CMH datasets (WSI), IHC, Validation
}{}%{
%    above of module2/Model (transfer from TCIA),
%    below of module3/Survival Analysis 
%} % addition x2
%\smartdiagramconnect{to-}{module2/additional-module1}
%\smartdiagramconnect{-to}{module3/additional-module2}
\vspace*{30mm}
%\smartdiagramset{border color=none, set color list={blue!50!cyan,green!60!lime,orange!50!red,red!80!black}, back arrow disabled=true}

%\smartdiagramset{module x sep=5}
%\smartdiagram[flow diagram:horizontal]{Aim1, Aim2, Aim3}


%\includegraphics[width=14cm]{Figure_HE2RNA.pdf}
%\bcaption{}{} %\captionof{diag} for floating "diag"
\bcaption{Workflow of RNA-Seq representatives of TMUH/CMH datasets.}
%\acrshort{hnscc} Cox's hazard ratio and \textit{P} value plot.
{The deep learning model of HE2RNA, which was trained under TCGA/TCIA \acrshort{hnscc} dataset, could be applied for RNA-Seq predictive of TMUH/CMH datasets. The representatives will be used for subsequent gene mutation/expression analysis.\\
%The EHR of TMU dataset could also be used for subsequent survival analysis.\\
(
TCGA: \acrlong{tcga};
TCIA: \acrlong{tcia};
WSI: whole-slide images of pathology; 
%EHR: \acrlong{ehr}; 
RNA-Seq: \acrlong{rnaseq}; 
\acrshort{rnn}: \acrlong{rnn}
)
} %uncorrected \textit{P} value.}
\label{fig:HE2RNA}
\end{figure}
%\end{center}

\clearpage



%%%
\paragraph{Mutation/expression profile of FAT1} %, CAMK2N1}

% ****deep learning can HELP SP1
%HE2DNA, HE2RNA

%\end{outline}
%central issue 2 with preliminary data (Cox model); fallback approaches % 需有充分的文獻分析和縝密的論述來補足，使得 “convincing power” 依然強而有力

%To construct a deep learning platform of genomics (mutation and expression) from pathology images

%Our preliminary data (biomarker candidates) could be served as the bridge between the prognosis of patients and their pathology images of specimen.
We will develop a deep-learning-based model (HE2DNA) as well as a validation platform. 
It might incorporate whole-slide images of pathology of TMUH/CMH cohorts with the mutation status of FAT1, TP53.
%\acrshort{hnscc} from the Cancer Imaging Archive (TCIA) and from Joint Biobank of Taipei Medical University (TMU dataset).
%***From sp1 Dr Wu\\
%Aim 1. 
It will also be validated in progressive and non-progressive \acrshort{ovh} samples using DNA-Seq on targeted gene.
%deep learning might help mutation representatives by "HE2DNA" model.\\

%Aim 2. 
The HE2RNA model will also be used to compare the transcriptome of FAT1 in wild-type versus mutant \acrshort{ovh} tissues.
%Aim 3. To dissect the cooperative effect of FAT1 and TP53 gene mutation on OVH transformation. \\
%Aim 4. To analyze the effect of mutated FAT1 on the reciprocal relationship of YAP1 and tp53  pathways in progressive OVH cells by the analysis of YAP1 and p53-regulated transcriptome and genome-binding regions.








\clearpage
%%%%%%%%%%%%%%%%%%%%%%%
% by Dr Wang's and Dr Chi's Integration and modeling
\subsubsection*{Aim 4:}\\[0.5cm]
%To develop risk prediction model for malignant transformation of \acrshort{ovh} by integrating risk factors identified from all sub-projects: sp1, sp2, and sp3

The candidate biomarkers, dysplasia grading, surgical margin status and saliva extract will be suggested as risk factors in malignant transformation of \acrshort{ovh} through analysis of Chi-Mei and TMUH datasets.


\paragraph{Feature engineering by explainable deep learning}

%HE2DNA, HE2RNA, HE2Displasia

% ***explainable by
% 1) silency map
% 2) *** auto encoder 2048 to 512 dimensions （visual subjectively 人眼就會發現 moth size, wing span, dark color ….) 
%傳統分類的名詞定義 by human、features from nature

% unsupervised learning 

HE2Displasia and \acrfull{gcnn} models will be conducted for survival analysis to predict malignant transformation of \acrshort{ovh} and even leukoplakia.
A survival time is defined as a progressive \acrshort{ovh} becoming cancer during the follow-up period.
%non-progressive OVH


% survival modeling by Dr Chi2022
%% TCGA new holistic features %%%%%
%Our previous \acrshort{hnscc} study\cite{Chi2020} 
Forty-two features will be collected and analyzed as potential independent risk factors for malignant transformation. 
The features will be selected using the 'least absolute shrinkage and selection operator' (LASSO) method for dimensional reduction.

%Our preliminary research data summarizes that 20 candidate biomarkers, clinical T stage, and surgical margin are independent prognosis factors.
The predictive model with coefficients will be established from cohorts (A)(B). The important input $X_1...X_n$ should be patients' features: age, candidate gene mutations/expressions, and surgical margin so far.
When we consider the survival prediction model,
ground truth Y (i.e., malignant transformation):\\[0.5cm]
%Y$:\\[0.3cm]
$Y = \beta_0 + \beta_1 X_1 + \beta_2 X_2 + \beta_3 X_3 + ... + \beta_n X_n + \epsilon$\\[0.3cm]
Actually, FAT1 gene expression is not an independent $X_{mrna}$. It could be influenced with other factors (for example):\\[0.3cm]
%$X_{body}, X_{stress}, X_{fear}, X_{meaningOfLife}, X_{inflammation}, X_{empathy}$\\[0.3cm]
$X_{CK13}, X_{CK17}, X_{p53}, X_{Ki-67}, X_{p53}, X_{CAMK2N1}, ... $\\[0.3cm]
That is because there is much interaction in gene regulatory network. 
%to be done in investigating the functional mind-body network\citep{Rogers1959}.
%We wish the TCGA does collect more holistic features from their participants in the future.
%依舊是以管窺天
% https://link.springer.com/referenceworkentry/10.1007%2F978-1-4419-1428-6_575
%癌症與身心靈的重要關係：中醫、citation? emotional centered approach
%We suggest TCGA should cover more holistic features:
%The cortisol is one of the most common objective outcomes studied in psychoneuroimmunological (PNI) factor. Interleukin, cytokines, lymphocytes (CD4+, CD8+), CD56+, vitals signs, telomere length, telomere activity, and heart rate variability (HRV) are others.
%The subjective psychosocial measures should include depression, stress, quality of life, anxiety, fatigue, mindfulness, mood, and specific spiritual growth (i.e., the meaning of life)\citep{Hsiao2012}.
%冤親債主的意念 unknown X factors 回到我們的文章
%Since psychological benefit (reduction of stress, fatigue, sleep disturbance, and depressive symptoms) also optimizes immune function (higher natural killer cell activity, higher \acrshort{ifng}, lower TNF-alpha, lower IL-6) in terms of better survival of patients from cancer.

% holistic EHR
%Moreover, we encourage physicians to write their \acrshort{ehr} in a "holistic way".
% Deep Patient:  
%Thus, deep learning can convert EHR to the Fast Healthcare Interoperability Resources (FHIR) format~\citep{Rajkomar2018}~\citep{HealthLevelSeven2019} for handling free-text notes by physicians.
Thus, deep learning can derive patient representations and gene network that offer improved clinical predictions~\citep{Miotto2016}.
%=> Level 4 Record-keeping and Data Exchange for the healthcare process (clinical data) %Under HL7, RESTful
%Allergy, Problem,
%Procedure,
%CarePlan/Goal,
%ServiceRequest,
%Family History,
%Risk Assessment (APGAR, Glasgow coma scale)
%condition: physical, mental
%{observations: 
%vital signs, 
%body temperature, body weight and body height, 
%x-ray, 
%laboratory data, 
%pulse oximetry, 
%ECG, EEG, EMG, 
%clinical findings (symptom and sign), 
%tobacco exposure, family support, cognitive status and pregnancy status.
%death assertion
%HIV, HBV, HCV status
%}
%https://mos2718.github.io/FHIRspec/Index.html
%https://www.hl7.org/fhir/patient.html
%https://mos2718.github.io/FHIRspec/Spec/Patient/EHR_Patient.docx
Once clinical features and models for pathology images are available, a graph convolutional neural network (GCNN) is mandatory for survival analysis~\citep{Ching2018a} in our study.

%In Taipei Medical University Hospital, holistic-enabled EHR has been designed in the daily-work for charting our outpatients with psycho-social-spiritual evaluation~\citep{Ling-ChengMong2021}:\\

\comm{
\begin{outline}

\1 Emotional: stable/depression/ anxiety/ agitated/ perplexed\\
\1 Interaction: expression disorder/ confusion/ hard to make decision/ uncooperative to dental treatment\\
\1 Social: (disturbing living/ disturbing working/ disturbing other life aspect)\\
\1 Function loss(subjectively): mastication/ pronunciation/ life quality/ state of mind\\
\1 Self-expectation: lack of confidence/ low self-esteem/ social-withdrawal\\
\1 Impact to life: financial problem/ family problem\\
\1 Medical relationship: bad experience of dental treatment/ loss confidence to dentist/ anxious to treatment/ side-effects/ litigation experience\\[0.1cm]

\end{outline}
} % end of \comm

%In personal history section of admission note for inpatient:\\
%<Habits>\\
%-Tobacco: \underline{ }\underline{ } pack/day for \underline{ }\underline{ } years\\
%-Alcohol drinking: Occasionally\\
%-Betel nuts: \underline{ }\underline{ } nuts/day for \underline{ }\underline{ } years\\
%<Social/Economic Evaluation>\\
%-Living condition: Living with family\\
%-Family support: Well\\
%-Main caregiver: Spouse\\
%-Financial support: Sufficient\\
%<Psychological Evaluation>\\
%Stable but worry about this oral lesion\\
%Suspected to be abuse or neglected victim: negative\\
%<Spiritual Assessment>\\
%Religion: Nil\\
%Diagnosed with life-threatening diseases or terminal stage of chronic diseases or cancers? No


%子計畫 升格為主計畫 EHR of TMU Joint Biobank
%\1 Aim 3: Biomarkers validation cohort for 10 candidates from pvalueTex project


%Deep learning can convert \acrfull{ehr} to the Fast Healthcare Interoperability Resources (FHIR) format\citep{Rajkomar2018}\citep{HealthLevelSeven2019} for handling free-text notes by physicians.
% RNN recurrent neural network \acrfull{rnn} will deal with "holistic features" of \acrshort{ehr} obtained from Taipei Medical University Clinical Research Database ***(TMUCRD).
% https://ods.tmu.edu.tw/portal_c3_cnt.php?owner_num=c3_69585&button_num=c3&folder_id=3714


%These patient's features should include physical, pathological, psychological data, and even more spiritual information investigated by physicians.
%The "SUB\_DESC", coded by the coding book (data schema) of TMUCRD, should store psychological and spiritual information in EHR.

% 攜入(linked my data) with 特定對象識別代碼(如身分證字號、病歷號碼)
% => biobank 代為 linking (收費?)
%申請資料欄位 *
%臨床資料: 常用欄位清單請見
% Coding book: https://ods.tmu.edu.tw/portal_a1.php?owner_num=a1_69914&button_num=a1；例如申請癌症登記、化療、腫瘤型態、病理報告、新生兒、母親、心臟超音波報告或非常用清單等欄位，請選自選欄位；若無，則選常用欄位。
% 自選欄位包含下列欄位，基於去識別化考量，僅限攜出(report to go)或攜入(linked my data)之服務模式(如下圖)。
%OPD_BASIC / SUB_DESC, SUB_DESC2, OBJ_DESC, OBJ_DESC2, PLAN_DESC, PLAN_DESC2, PATT_REPORT, CR_TCASE / NOTE







%\subsubsection*{Aim 3':} Biomarkers Discovery from RNA-Seq, Whole-slide Images, and EHR from TMU dataset\\[0.5cm]

%\1 Aim 3: RNA-Seq Incorporated with Whole-slide Images models from TCIA and TMU\\
%central issue 3 without preliminary data; fallback approaches

%\begin{outline}
Whole-slide images are the surrogate of expression data of RNA-Seq.
From the result of Aim 3, the transcriptomic representatives of TMUH/CMH datasets
%(n=46, obtained at TMU Joint Biobank) 
could be applied to validate those candidates discovered by our previous study: FAT1, CAMK2N1.
% Biomarkers validation cohort for 10 candidates from pvalueTex project
Moreover, the result of Aim 4 (features extracted from pathology images) will be used for survival analysis 
%(Figure ~\ref{fig:HE2RNA}) by conventional Cox proportional hazard regression and 
by \acrshort{gcnn} (Figure \ref{fig:GCNN}). % try Ipe; or Inkscape
% modifying % Figure 5 HE2RNA workflow: EHR -> survival analysis




%%%%%%%%%%%%%%
% ***explainable by
% 1) silency map
% 2) *** auto encoder 2048 to 512 dimensions （visual subjectively 人眼就會發現 moth size, wing span, dark color ….) 
%傳統分類的名詞定義 by human、features from nature
% unsupervised learning 

HE2Displasia model could be applied on an autoencoder for explainable machine learning.
An autoencoder is a type of unsupervised learning for unlabeled data.
Autoencoder with cox regression (AutoCox)~\citep{Torkey2021} combines an explainable deep learning and Cox Proportional Hazards Modeling. Code is available at GitHub: \url{https://github.com/Mostafa-Samy-Atlam/Autoencoder-with-survival-analysis-methods}, accessed on May 2022.
%%%
As a result, AutoCox will inform us on how to estimate the malignant transformation of an \acrlong{ovh} based on "visual subjectively features" learned from pathology images.

%%%%%%%%%%%%%%



% figure by Ipe app
% make Figure 6
\begin{figure}[hp]
\centering
\includegraphics[width=14cm]{GCNN_survival_AutoCox.png}
\bcaption{Overview of the proposed workflow for explainable survival analysis}
{\\
%We suggest that the TMU dataset should collect "holistic features" in electric healthcare records (EHR), which include physical, pathological, psychological data, and spiritual information. Combined with RNA-Seq representation, 
With whole-slide-images pathology, the \acrfull{gcnn} should extract high-dimensional information for \acrshort{ovh} survival prediction. Predictors should be "seen" by AutoCox algorithm.}
\label{fig:GCNN}
\end{figure}


%需有充分的文獻分析和縝密的論述來補足，使得 “convincing power” 依然強而有力
%*transfer learning from TCGA model
%torchvision, NanoMets Models
%save checkpoint, load then deepcopy()


%\subsubsection{TCIA validation by deep learning}
%https://www.cancerimagingarchive.net/
%from CPTAC and TCGA images
%poster 
%https://wiki.cancerimagingarchive.net/download/attachments/1966514/TCGA_TCIA_Poster_111511_HHFa_WIDE.pptx?version=1&modificationDate=1323207330686&api=v2

%The Cox proportional hazards regression is the widely accepted approach for modeling survival while accounting for confounding factors\cite{Magen2019}. Univariate and multivariate Cox proportional regression model\cite{Andersen1982}, using the "coxph" function in R package "survival", will be applied to calculate hazard ratio, \acrfull{ci95} and its significance, and to estimate the independent contributions of each clinicopathological features to the overall survival.
%Results are considered statistically significant when a two-sided \textit{P} value $<$ 0.05, or a lower threshold if indicated.
%The \acrfull{fdr} ($< 0.05$) could be used to pick up the optimal \textit{P} value to ensure the control for type I error of multiple survival test during the survival analysis.
%There were also multiple correlated tests of null hypotheses in the Kaplan-Meier family during our global scanning of protein-coding genes. The stringent Bonferroni correction could result in an adjusted \textit{P} value to ensure the control for type I error. 

%Cox regression tables, were exported to ".xlsx" and ".Rda" files (by R package "r2excel") for subsequent biomarker selection.




%The transcriptomic representation of \acrfull{tcia} and TCGA could be applied to validate those candidates by independent TMU cohort.

%We will find that \acrshort{rnaseq} (or \acrshort{rnaseq} representation with associated pathology images) and holistic features will help to facilitate the biomarker discovery in \acrshort{hnscc}.
%\end{outline}




%\clearpage
\paragraph{Nomograms for malignant transformation of OVH}
%Modeling by Dr Wang}


%Nomograms for malignant transformation of OVH
Based on the factors associated with malignant transformation of \acrshort{ovh} identified from all subprojects SP1, SP2, and SP3, we would like to construct nomograms for predicting risk of malignant transformation in the \acrshort{ovh} cohort. The nomogram approach provides a simple graphical representation of a predictive model and estimate a probability of a specific clinical event according to the profile of an individual patient, which is a predictive tool widely used in clinical practice. 
The typical workflow of conducting a nomogram prediction model consists of the following steps~\citep{Iasonos2008}:

\begin{outline}

\1 To identify the patient population
\1 To define the outcome
\1 To identify potential covariates
\1 To construct the nomogram
\1 To validate the constructed model
\end{outline}

In this study, the target population consists of patients who had OVH but who had not yet developed oral cancer. We will consider two outcomes. The first one is the occurrence of malignant transformation; the second one is the time to malignant transformation. For the first outcome, we implement multiple logistic regression model to construct the nomogram. For the second outcome, we perform Cox proportional hazard model to construct the nomogram. 
Typically, the accuracy of a nomogram is assessed by a concordance index (c-index), which quantifies the level of concordance between predicted and actual probabilities of having the specified event. Furthermore, a 10-ford cross-validation will be implemented to address and prevent model overfit~\citep{Iasonos2008}. 

\paragraph{Summary of Statistics considerations}
\begin{outline}


\1 Statistical tools for genetic association testing (GWAS) 
\1 Kaplan--Meier analyses \1 Statistical tools for risk prediction models 
\1 Cox regression analyses
with backward stepwise elimination approach
\1 Explainable deep learning (machine learining) algorithm
\1 Nomogram prediction model
    \2 Cox proportional hazard regression model
    \2 Multiple logistic regression model
\1 The receiver operating characteristic (ROC) curve analysis
\1 Concordance index (c-index) for model evaluation
%, which quantifies the level of concordance between predicted and actual probabilities of having the specified event
\1 K-fold cross-validation for prevention of model overfit

\end{outline}

\clearpage



%%%%%%
%\include{chapter0}
%\include{chapter1} % introduction
%\include{chapter2} % Review of Literature
%\include{chapter3} % Methodology (Research Design & Methods)
%\include{chapter4} % Results (Presentation of Research) C004 or C005...
%%%%%

\section*{\cntext{(三)預期完成之工作項目、成果及績效}}
%1.預期完成之工作項目
\begin{outline}

\1 We expect to obtain some genetic markers associated with malignant transformation of \acrshort{ovh}.

\1 We expect to find some gene-environment interaction effects on malignant transformation of \acrshort{ovh}.

\1 We will construct nomograms for malignant transformation of \acrshort{ovh}.

%\1 Validation of candidate biomarkers of \acrshort{ovh}.
%2.對於學術研究、國家發展及其他應用方面預期之貢獻
%It will be submitted for further diagnostic method for personalized medicine of \acrshort{ovh}.
% 分析\acrshort{tcia}和TMU的癌細胞基因表現量、數位病理影像和全人照護特徵，將有助於驗證候選基因對預後的影響，以發展未來頭頸部鱗狀細胞癌的精準治療計劃。

%) or \cntext{精準醫療}
%3.對於參與之工作人員，預期可獲之訓練
\1 We will improve our programming skill of R, PyTorch (Python) and C ++ for deep learning.
%4.預期完成之研究成果及績效(如期刊論文、研討會論文、專書、技術報告、專利或技術移轉等質與量之預期績效)。請詳述其必要性以及預期成果等。
\1 We will finish manuscripts for journal publication.

% 致謝詞 acknowledgments in US
% acknowledgements in British
\1 Acknowledgments: We thank the staff of the Clinical Data Center, Office of Data Science, Taipei Medical University, Taiwan, for statistical consultation, analysis/interpretation of data, and technical support. 
We also thank the National Center for Genome Medicine for the technical support in the genotyping.
The results shown here are in whole based upon data generated by the \acrshort{tcga} Research Network: https://www.cancer.gov/tcga.
The tables were generated by latex code with the help from https://www.tablesgenerator.com/latex\_tables\#.
%https://github.com/JDMCreator/LaTeXTableEditor (accessed at https://www.latex-tables.com/v1/\\index.html).
We sincerely want to express our thanks to the \acrshort{ovh} patients who donated their data, and help from TMU Joint Institutional review board.
% *** URL Accessed 20 May 2013.



\end{outline}
\clearpage

% C004
%\thispagestyle{fancy004}

\section*{\cntext{四、申請補助經費:}}
*\cntext{業務費(臨時工資及耗材、物品及雜支): 新台幣五十萬元}\\
\cntext{*經費細項請於表C005~表C006填寫。}

%%%
% https://docs.google.com/file/d/15QxU4ok7jpN9jVl0h3w10b2mSDUmxt8o/edit?usp=docslist_api&filetype=msexcel

% ************ from Dr Wang and Dr Chi
%TWB2.0 DNAchip 192000, 抽DNA 34560, TWB2.0 DNAchip檢體送樣前QC 9600, 資料儲存硬碟 9890, 國網運算資源 86630; 生物資料庫使用費(切片費) 9900, 生物資料庫使用費(染色費) 42000, IHC 免疫染色抗體 78000, Aperio ScanScope數位病理切片掃描 31500, 文具雜支 5920.

% antibody x 6: p53, CAMK2N1, FAT1, CK13, CK17, Ki-67

%業務費
%(臨時工資及耗材、物品及雜支)
%合 計 50萬元
%%%

%(四)計畫經費之編列}}
% 核定的經費不會高於所 要求的預算，如經費不足對日後計畫執行也是障礙，因此每項經費的使用最好 想清楚，說明白，比如設備費，逐年需購買哪些圖書?何以需購買電腦、筆 電，尤其國立大學設備資源較充裕，平均四年即更新汰舊電腦，編列時還經常 可見便宜行事的情況，複製經費編列，以致於每年都購買電腦或印表機，顯見 並不適合。參與國際會議及移地研究，停留天數及其必要性都需填寫，說明越 清楚，通過的機會就越高。此外，編列項目越詳細越好，免得日後有需要時， 因計畫書未列，校方不准核銷，而影響計畫的進行。
 
%主要研究人力，主持人每周平均投入工作時數比例，請「合理」。(100% 就是不合理喔!!請注意。一般而言，20-40%或 50%是比較合理)

%%%% C004 table
% by https://www.tablesgenerator.com
% Please add the following required packages to your document preamble:
% \usepackage[table,xcdraw]{xcolor}
% If you use beamer only pass "xcolor=table" option, i.e. \documentclass[xcolor=table]{beamer}
\begin{CJK*}{UTF8}{bkai}

% Please add the following required packages to your document preamble:
% \usepackage[table,xcdraw]{xcolor}
% If you use beamer only pass "xcolor=table" option, i.e. \documentclass[xcolor=table]{beamer}
\begin{table}[hp]
\centering
\begin{tabular}{|
>{\columncolor[HTML]{FFFFFF}}l |
>{\columncolor[HTML]{FFFFFF}}l |
>{\columncolor[HTML]{FFFFFF}}l |}
\hline
補助項目                 & 預計申請金額                       & 說明                     \\ \hline
人事費 (兼任助理、臨時工資等)     & 0                            &                        \\ \hline
耗材、物品及雜支                & 500,000                      &                        \\ \hline
%論文發表費及論文編修費 (含英譯/編輯) & 60,000                       &                        \\ \hline
合計                   & \multicolumn{2}{l|}{\cellcolor[HTML]{FFFFFF} 500,000元} \\ \hline
\end{tabular}
\end{table}

\clearpage
%%% C005
五、研究人力費:\\
(1) 兼任助理、臨時工。\\
(2) 兼任助理、臨時工之每月工作酬金標準，不得超過本院補助專題研究計畫臨時工工作酬金參考表及本院補助專題研究計畫兼任助理、臨時工工作酬金支給標準表之規定。\\[1cm]
%
\begin{flushright}
金額單位:新台幣元
\end{flushright}

\begin{table}[hp]
\centering
\begin{tabular}{|
>{\columncolor[HTML]{FFFFFF}}l |
>{\columncolor[HTML]{FFFFFF}}l |
>{\columncolor[HTML]{FFFFFF}}l |
>{\columncolor[HTML]{FFFFFF}}l |
>{\columncolor[HTML]{FFFFFF}}l |
>{\columncolor[HTML]{FFFFFF}}l |}
\hline
類別/級別 & 人數 & 工作 時數(人/月) & 月支酬金 & 小計 & 說明: 最高學歷、工作性質、項目 
\\ \hline
      &    &            &      &    &                                   \\ \hline
      &    &            &      &    &                                   \\ \hline
      &    &            &      &    &                                   \\ \hline
\multicolumn{6}{|l|}{\cellcolor[HTML]{FFFFFF}合計}                        \\ \hline
\end{tabular}
\end{table}

\clearpage
%%% C006
% 業務費(臨時工資及耗材、物品及雜支)
六、耗材、物品及雜支費用:\\
(1) 凡執行研究計畫所需之耗材、物品(非屬研究設備者)及雜支費用，均可填入本表內。\\
(2) 說明欄請就該項目之規格、用途等相關資料詳細填寫，以利審查。\\[1cm]
\begin{flushright}
金額單位:新台幣元
\end{flushright}


% 抗體？(10= 9 + FAT1)
%We assume those biomarkers, CK13, CK17, p53, Ki-67, EGFR, $\alpha$-SMA, CAMK2N1, and FAT1, should be found within surgical margin (dysplasia zone) of \acrshort{oscc} specimen and \acrshort{ovh}.
% p16INK4a and p14ARF for (PVL) proliferative verrucous leukoplakia

\begin{table}[hp]
\centering
\resizebox{\textwidth}{!}{%
\begin{tabular}{|c|l|c|r|r|r|r|}
\hline
項目名稱   & 說明                        & 單位 & \multicolumn{1}{l|}{單價} & \multicolumn{1}{l|}{數量} & \multicolumn{1}{l|}{金額} & \multicolumn{1}{l|}{備註}                                              \\ \hline
耗材費    
& 研究計畫人體試驗委員會審查費            & 次  & 12000                       & 1                  & 12000                  & \begin{tabular}[c]{@{}r@{}}審查10000\\ 修正1000\\ 結案1000\end{tabular} \\ \hline
%耗材費    & 生物資料庫使用費(切片費)             & 5片 & 19                      & 330                     & 6270                    &                                                                      \\ \hline
耗材費
& TWB2.0 DNAchip & 盤 & 96000 & 2 & 192000 & 一盤96樣本，施作兩盤\\
\hline
耗材費
& 抽DNA & 例 & 180 & 192 & 34560 & 以上述兩盤樣本數計\\
\hline
耗材費
& TWB2.0 DNAchip檢體送樣前QC & 例 & 50 & 192 & 9600 & 以上述兩盤樣本數計\\
\hline
耗材費
& 資料儲存硬碟 & 顆 & 9890 & 1 & 9890 & \\
\hline
耗材費
& 國網運算資源 & & & & 3950 & \\
\hline

% Dr Chi EGFR CAMK2N1
耗材費    & 生物資料庫使用費(切片費)           & 5片  & 330                      & 30                      & 9900                    &                                                                      \\ \hline

耗材費    & 生物資料庫使用費(HE染色費)           & 片  & 40                      & 150                      & 6000                    &                                                                      \\ \hline


耗材費    & 生物資料庫使用費(FAT1染色費)           & 片  & 40                      & 150                      & 6000                    &                                                                      \\ \hline

耗材費    & 生物資料庫使用費(p53染色費)           & 片  & 40                      & 150                      & 6000                    &                                                                      \\ \hline

耗材費    & 生物資料庫使用費(Ki-67染色費)           & 片  & 40                      & 150                      & 6000                    &                                                                      \\ \hline

耗材費    & 生物資料庫使用費(CK13染色費)           & 片  & 40                      & 150                      & 6000                    &                                                                      \\ \hline

耗材費    & 生物資料庫使用費(CK17染色費)           & 片  & 40                      & 150                      & 6000                    &                                                                      \\ \hline

耗材費    & 生物資料庫使用費(EGFR染色費)           & 片  & 40                      & 150                      & 6000                    &                                                                      \\ \hline

耗材費    & 生物資料庫使用費($\alpha$-SMA染色費)           & 片  & 40                      & 150                      & 6000                    &                                                                      \\ \hline

耗材費    & 生物資料庫使用費(CAMK2N1染色費)           & 片  & 40                      & 150                      & 6000                    &                                                                      \\ \hline

耗材費    & 生物資料庫使用費(p16INK4a染色費)           & 片  & 40                      & 150                      & 6000                    &                                                                      \\ \hline
耗材費    & 生物資料庫使用費(p14ARF染色費)           & 片  & 40                      & 150                      & 6000                    &                                                                      \\ \hline
耗材費    & IHC 免疫染色抗體           & 支  & 13000                     & 9                      & 117000                    &                                                                      \\ \hline

%耗材費    & 生物資料庫使用費(臨床病歷)            & 每名 & 46                      & 1400                    & 64400                   &                                                                      \\ \hline
%耗材費    & 臺北醫學大學數據處研究資料庫檢索費         & 小時 & 4                       & 0                       & 0                       & 目前免費                                                                 \\ \hline
儀器使用費  & Aperio ScanScope 數位病理切片掃描 & 小時 & 300                      & 105                    & 31500                   &                                                                      \\ \hline
雲端運算   & 國網中心高速電腦GPU人工智慧訓練         & 小時 & 689                      & 19.7                 & 13600                 & cm.4xsuper                                                           \\ \hline
%資訊硬體設備 & Mac Mini M1               & 部  & 1                       & 37,100                  & 37,100                  &                                                                      \\ \hline
合 計    &                           &    &                         &                         & 500,000                 &                                                                      \\ \hline
\end{tabular}%
}
\end{table}


\clearpage
%%%%%%%%%%%%%%%%
附件二\\
\begin{center}
    %    \centering
%\begin{tabular}{@{}c@{}}
%\centering
副主持人個人資料表\\
%\end{tabular}
\end{center}

%一、基本資料:      簽名:      \\
一、基本資料:      簽名:\\  
\begin{flushright}
身份證字號: A120475979\hspace{8cm} 填表日期: \today\\
\end{flushright}

%身份證字號: A120475979\\%[0.2cm]

\includegraphics[width=18cm]%, trim=0cm 0 0 3cm]
{MOST_PI_resume.png}
%\begin{figure}[hp] %主持人資料表.pdf
%    \centering
%    \includegraphics{MOST_PI_resume}
%    \caption{Caption}
%    \label{fig:my_label}
%\end{figure}

%%

%\begin{center}
    二、主要學歷 (由最高學歷依次填寫，若仍在學者，請在學位欄填「肄業」)\\
%\end{center}

\begin{table}[hp]
\centering
\resizebox{\textwidth}{!}{%
\begin{tabular}{|l|l|l|l|l|}
\hline
學校名稱                                          & 國別     & 主修學門系所                                  & 學位 & 起訖年月(西元年/月)       \\ \hline
Taipei Medical University and Academia Sinica & Taiwan & PhD in Translational Medicine & 博士 & 2013/02 - 2022/01 \\ \hline
Taipei Medical University                     & Taiwan & MS in Biomedical Informatics            & 結業 & 2011/09 - 2013/01 \\ \hline
Taipei Medical University                     & Taiwan & BS in Dentistry                         & 學士 & 1988/09 - 1994/06 \\ \hline
\end{tabular}%
}
\end{table}


%\begin{center}
    三、現職及與專長相關之經歷 (指與研究相關之專任職務，請依任職之時間先後順序由最 近者往前追溯)\\
%\end{center}
\begin{table}[hp]
\centering
\resizebox{\textwidth}{!}{%
\begin{tabular}{|l|l|l|l|}
\hline
服務機構                               & 服務部門/系所                          & 職稱                   & 起訖年月(西元年/月)       \\ \hline
Taipei Municipal Wanfang Hospital  & Oral and Maxillofacial Surgery   & Director             & 2021/04 - 2022/04 \\ \hline
Taipei Medical University Hospital & Oral and Maxillofacial Surgery   & Attending physician & 2002/01 - Present \\ \hline
Taipei Medical University Hospital & Taiwan Medical Mission in Africa & Attending physician & 2009/05 - 2012/01 \\ \hline
\end{tabular}%
}
\end{table}


%\begin{center}
    四、專長 (請填寫與研究方向有關之學術專長名稱)\\
%\end{center}


\begin{table}[hp]
\centering
\resizebox{\textwidth}{!}{%
\begin{tabular}{|l|l|l|l|}
\hline
\begin{tabular}[c]{@{}l@{}}Programming in:\\ R, Python, Matlab, SAS, deep learning.\\
Translational Medicine
\end{tabular} & \begin{tabular}[c]{@{}l@{}}Languages\\ Chinese, English \end{tabular} & \begin{tabular}[c]{@{}l@{}}Sports:\\ Swimming, water rescue, \\ SCUBA diving, bicycle cycling,\\ moutaineering\end{tabular} & \begin{tabular}[c]{@{}l@{}}Mindfulness\\ meditation\end{tabular} \\ \hline
\end{tabular}%
}
\end{table}
\end{CJK*}
%%

\clearpage

\begin{CJK*}{UTF8}{bkai}
%\section{\cntext{對社會的貢獻}}
表C301
\section*{五、著作目錄:}
%\cntext{過去研究之表現}}

% 申請人過去五年著作及五年以上且深具影響之著作，最好詳細? 精簡
% * 系統的關聯性 for one researcher
%而有條理地說明自己近五年來研究的系統性、延續性以及對學界的貢獻，如果 能列出每篇論文被引用的次數，或哪些重要期刊論文曾引用，可以更凸顯自己 的優勢，爭取更有利的評分。個人學術表現既如此重要，因此每一次的申請都 應記得更新個人資料 C302 表，確認與完成登錄著作目錄之作業，產生近 5 年著 作目錄表

(1) 請詳列個人申請截止日前五年內發 表之學術性著作，包括:期刊論文、專書及專書論文、研討會論文、技術報告及其 他等，並請依各類著作之重要性自行排列先後順序。\\
(2) 各類著作請按發表時間先後順序填寫。各項著作請務必依作者姓名(按原出版之 次序，通訊作者請加註*)、出版年、月份、題目、期刊名稱(專書出版社)、起迄頁數之順序填寫，被接受刊登尚未正式出版者請附被接受函。\\
(3) 若期刊是屬國內或國際期刊資料庫(如SCI、EI、SSCI、A\&HCI、Scopus、TSSCI 、THCI Core...\\等)所收錄者，請於該著作書目後註明資料庫名稱;若著作係經由國科會補助之研究計畫所產生，請於最後填入相關之國科會計畫編號。\\

\textbf{Publication:}\\
\begin{outline}
\1 Chi L-H, Chang W-M, Chang Y-C, et al. Global Proteomics-based Identification and Validation of Thymosin Beta-4 X-Linked as a Prognostic Marker for Head and Neck Squamous Cell Carcinoma. Sci Rep. 2017;7(1):9031. doi:10.1038/s41598-017-09539-w.[SCI]

\1 Chi L-H, Wu ATH, Hsiao M, Li Y-CCJ. A Transcriptomic Analysis of Head and Neck Squamous Cell Carcinomas for Prognostic Indications. J Pers Med. 2021;11(8):782. doi:10.3390/jpm11080782. [SCI]

\1 Chi L-H. Global Transcriptomics and Proteomics Analyses for Biomarker Identification and Validation in Head and Neck Squamous Cell Carcinoma. PhD Progr Transl Med. Published online 2022. [Ph.D. Dissertation]

\end{outline}

Chi and his colleagues~\citep{Chi2017} applied the matrix-assisted laser desorption/ionization (MALDI) imaging mass spectrometry (IMS) and liquid chromatography with tandem mass spectrometry (LC-MS/MS) to identify tumor-associated biomarkers of \acrshort{hnscc}. The gene thymosin beta-4 X-linked (TMSB4X) is a candidate biomarker in HNSCC whose functions resulted in enhanced proliferation and metastasis in vitro and in vivo.
That publication has been cited by cancer research of Roška and his colleague, published on Oncotarget~\citep{Roska2020, Chu2019a, Makowiecka2019a}, and by research of epidermal morphogenesis~\citep{Padmanabhan2020}.

In two articles~\citep{Chi2021, Chi2022}, Chi has emphasized that a breakthrough improvement of \acrshort{oscc} therapies should depend on "1) adequate surgical margin and radiotherapy guided by pathology, 2) predictive biomarkers, and 3) holistic cancer care". 
%Four validated biomarkers (TMSB4X, CAMK2N1, CALML5, and FCGBP), clinical tumor size, and surgical margin status all are independent prognosis factors in HNSCC. Overexpression of TMSB4X is associated with a poor prognosis of overall survival (OS) and recurrence-free survival (RFS) of those patients. TMSB4X knockdown in the HNSCC cell line reduced proliferation, invasion, and cervical lymph node metastasis in vitro and in vivo. More- over, CAMK2N1, CALML5, and FCGBP, are also substantially linked with patients’ OS. Holistic cancer care considers the importance of the spirit, emotion, body, and social relationships. Ultimately, holistic cancer therapy will be a comprehensive tailored personalized medication.
The betel-nut chewing, alcohol \acrfull{aldh2} mutation should be the major concern as a variety of \acrshort{hnscc} in Taiwan.
Pathology images should be analyzed with deep learning platform for a transcriptome representatives of mRNA learned by NCI's HE2RNA project~\citep{Schmauch2020}. Chi has a hypothesis that top-down research by deep learning algorithm~\citep{Huang2020} could find biomarkers surrounding the surgical margin.

\end{CJK*}


%% C302 2 pages MOST_proposal_DeepLearning_HNSCC-C302_2
% \usepackage{pdfpages}
%\includegraphics[width=18cm, pages={1-}]%, trim=1.5cm 0 0 3cm]{MOST_proposal_DeepLearning_HNSCC-C302}
\includegraphics[width=18cm]%, trim=1.5cm 0 0 3cm]
{MOST_proposal_DeepLearning_HNSCC-C302}
\includegraphics[width=18cm]%, trim=1.5cm 0 0 3cm]
{MOST_proposal_DeepLearning_HNSCC-C302_2}
%%
 % ------------------------------------------------------
%
% References
%
% ------------------------------------------------------
 \clearpage
\renewcommand\bibsection{\section*{REFERENCES}}
\setlength{\bibsep}{2pt}
%\bibliography{IEEEabrv, pvalueTex, TMSB4X, TCGA_margin_cutoff, SLC2A4_metabolism.bib, Holistic, Deep_Learning-Radiomics, Deep_Learning}

% all merged at most_merged, repeated entry removed
\bibliography{dissertation2022_Tex.bib, Deep_IHC_2023.bib, Tex_publications_since2002.bib}
%most_merged}
% by
% $ bibtool -f '%s($key)' -Ac -s -v /Users/texchi/Documents/Documents\ -\ Mac\ mini/Holistic.bib /Users/texchi/Documents/Documents\ -\ Mac\ mini/Deep_Learning.bib /Users/texchi/Documents/Documents\ -\ Mac\ mini/pvalueTex.bib /Users/texchi/Documents/Documents\ -\ Mac\ mini/SLC2A4_metabolism.bib /Users/texchi/Documents/Documents\ -\ Mac\ mini/TMSB4X.bib /Users/texchi/Documents/Documents\ -\ Mac\ mini/TCGA_margin_cutoff.bib -o most_merged.bib$
\end{document}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\smartdiagramset{uniform sequence color=true}
% sequence diagram; flow diagram:horizontal
% 2021
\smartdiagram[sequence diagram]{2021/July, Aug, Sep, Oct, Nov, Dec}

\smartdiagramset{border color=none,
   set color list={blue!50!cyan,green!60!lime,orange!50!red,red!80!black},
   back arrow disabled=true}

\smartdiagramset{module x sep=5}
\smartdiagram[flow diagram:horizontal]{Aim1, Aim1}


\smartdiagramset{border color=none,
   set color list={blue!50!cyan,green!60!lime,orange!50!red,red!80!black},
   back arrow disabled=true}
\smartdiagramset{module x sep=5}
\smartdiagram[flow diagram:horizontal]{Aim1, Aim2, Aim3}

% 2022
\smartdiagram[sequence diagram]{2022/Jan, Feb, March, April, May, June}

\smartdiagramset{module x sep=15}
\smartdiagram[flow diagram:horizontal]{Aim3, Aim4}
%\end{center}


%%%%%%%%%%

\paragraph{Immune System and Cancer}
% balanced Yan and Yin 陰陽要調合
Two-faced roles of tumor-associated neutrophils, eosinophils, and basophils

*** Tex's opinion: there is no "central console" of immune control system, which disseminated remote regular % 地方自治 

chronic inflammation by mechanical force or chemicals

** or even in autoimmune disease:
Risks from disease and treatment
Autoimmune disorders generally attack a single organ or part of the body, often causing inflammation in the affected area. In some cases, that inflammation may increase cancer risk. 
At the same time, tumor necrosis factor (TNF) inhibitors or immunosuppressant (e.x. cyclosporin), help reduce inflammation and also are suspected of increasing the risk of multiple cancers. It may impair the ability to kill cancer cells.

two of ten hallmarks: avoiding immune destruction, tumor-promoting inflammation \citep{Ohman2015}



The basic principles that guide cancer immunology are immune surveillance \citep{BURNET1957, Shankaran2001}, immune editing, and immune tolerance.
%immunosurveillance
Peritumoral immune responses might predict patients’ prognosis in a wide range of cancers. Beyond local accumulation, systemic immune response in serum is seen in patients with \acrshort{hnscc} \citep{Lee2010a}.
% immunoediting 敵消我長的過程
Moreover, the immune response is not only achieving protection of the host but also editing the immunogenicity of tumours \citep{Dunn2002}.
The cancer immunoediting includes elimination, equilibrium then escape.
%elimination by immunosurveillance
The cells, which have aberrant major histocompatibility complex (MHC) class I expression, will be eliminated by cytotoxic mechanism or induction of apoptosis by immune cells, involving granulocytes (neutrophils, eosinophils, and basophils), macrophages, natural
killer cells, in innate immunity \citep{Tallerico2013}.
Tumour-associated macrophages include  M1, which has antitumoral properties, and M2, which has tumour-promoting phenotype in \acrshort{hnscc} \citep{Kumar2019}.

Through dendritic cells presenting the tumor antigens to T helper cells in context with MHC class II molecules, naive or memory T cells will conduct activation of  T cells and adaptive immunity \citep{Banchereau2000}. 
The plasmacytoid cells are a subtype of dendritic cells that are one of the main sources of secreting \acrshort{ifng}. 
T cells consist of at least five subgroups: Th1 (CD4+), Th2 (CD4+), Th17 (CD4+), regulatory T cells (CD4+ CD25+ Tregs), and cytotoxic T cells (CD8+). 
Cytotoxic T cells is triggered by helper T cells to attack directly and execute those tumor cells, whose antigen is presented by dendritic cells, via effector molecules such as perforin and granzyme. Cytotoxic T cells also use TRAIL receptor to release signal on tumor's FasL to induce cell apoptosis.
% By inducing intracellular signaling through lck, the CD8 coreceptor is an important factor affecting TCR‐mediated T cell activation and modulation of immunosuppression and cytotoxicity.
All subsets play important roles in mucosal immune response, including antitumoral responses. T cells represent approximately 10\% of the total cells in a tumour mass \citep{Balkwill2012}.

% equilibrium under immunoselection:  Cancer cells avoid immunosurveillance by the selection of non-immunogenic tumour-cell variants % 鄭爺爺
***When immunosurveillance systems are not able to eradicate the tumor cells, the result may be tumor dormancy, where an equilibrium with defending cells occurs.

equilibrium represents a time of tumour cell persistence without progression.
equilibrium is maintained solely by adaptive immunity
The interplay of the potentially malignant cells and immune cells from immunosurveillance, equilibrium to tumor escape.
Lymphocytes of adaptive immunity have the capacity to take over and exert enough antitumoral effects to kill and limit tumour growth, and the tumour is thus kept in their dormitory \citep{Koebel2007}.
The tumour cells in equilibrium are highly immunogenic (unedited), whereas those spontaneously exiting equilibrium (escape) that become growing tumours have attenuated immunogenicity (edited)—results that place this process temporally between elimination and escape \citep{Koebel2007}.
maintaining cancer in an equilibrium state may represent a relevant goal of cancer immunotherapy in which augmentation of adaptive tumour immunity could result in improved tumour control. *** A TREATMENT GOALS
%the downregulation or loss of expression of HLA class I molecules
% decoy receptors for TRAIL: DcR3, Decoy receptor 3 (DcR3), to prevent FasR-FasL interactions by competitively binding to membrane-bound Fas ligand; FasL binds to DcR3 (TNFRSF6B) without TRAIL function 假的誘餌
% Death receptors:  tumour-necrosis-factor (TNF) receptor 1, CD95 (also known as FAS) and two receptors for TNF-related apoptosis-inducing ligand (TRAILR1 and TRAILR2).


% escape under immunesubversion: by the active suppression of the immune response
Escape is the process wherein the immunologically sculpted tumor expands in an uncontrolled manner in the immunocompetent host \citep{Dunn2002, Dunn2004}.
It was suggested by increasing the presence of Tregs in terms of suppression of Th1 and cytotoxic T cells response by \acrfull{ctla4} \citep{Halvorsen2014}.
Moreover, myelo-derived suppressor cells (MDSCs, CD34+) could also down-regulation of granzyme from T cells in \acrshort{hnscc} \citep{Si2019}.
The antitumoral effect of adaptive immunity also help to the tumour evolution. It lets tumour cells to modify immunogenicity to avoid immune recognition, and to resist the cytotoxicity of T cells.
The activated oncogenes can trigger 1) DNA-damage sensors, [such as ATM (ataxia-telangiectasia mutated) and ATR (ATM and Rad3 related)]; 2) checkpoint kinases, [such as CHK1 (checkpoint kinase 1 homologue) and CHK2]; 3) and the tumour-suppressor protein p53. 
The transcription factor, p53, that is activated by many genotoxic insults and induces cellular apoptosis. The gene TP53, encoding p53, is frequently mutated or functionally inactivated in cancer cells.
Through suppression of the DNA-damage response by above mechanisms, tumor cells might downregulation of NKG2D ligands (\acrshort{mica}, \acrshort{micb}) to avoid recognition by NKG2D-expressing NK cells, natural killer T (NKT) cells, $\gamma\delta$ T cells and some cytolytic CD8+ $\alpha\beta$ T cells \citep{Ljunggren2007}. They also express PD‐L1 and Fas ligand to suppress cytotoxic T cells \citep{Zitvogel2006}. Thus, the cell-cycle arrest, DNA repair or apoptosis (aka. DNA-damage response) of tumor cells will be inhibited.
The activation of $\acrshort{tgf}\beta$ in terms of promoting epithelial-to-mesenchymal transition (EMT). 
%IL-10
% Activation of the DNA-damage response by cell-intrinsic stimuli (oncogene activation or reactive oxygen species, ROS) or therapy activates the ATM (ataxia-telangiectasia mutated)–CHK1 (checkpoint kinase 1 homologue) pathway. This pathway leads either to p53- or NKG2D- (ligand: MICA or MICB) dependent apoptosis of tumour cells.
% NKG2D (Natural-killer group 2, member D). A lectin-type activating receptor that is encoded by the NK complex and is expressed at the surface of NK cells, NKT cells, γδ T cells and some cytolytic CD8+ αβ T cells. 
% The ligands for NKG2D are MHC-class-I-polypeptide-related sequence A (MICA) and MICB in humans; and retinoic acid early transcript 1 (RAE1) and H60 in mice

% 小巷弄防巷戰 (金門防線)
Finally, tumor cells have downregulated homing receptors, and malformation of peripheral vacularity. It causes hypoxia and increased interstitial pressure which lead to a hostile microenvironment for against the infiltrating defense cells \citep{Ljunggren2007}.


%{ references
ICI, hallmarks of cancer
%** Sharma, P. et al. Cancer Immunotherapy. in Holland‐Frei Cancer Medicine 1–23 (John Wiley & Sons, Inc., 2017). doi:doi:10.1002/9781119000822.hfcm068.

%**1. Öhman, J. Potentially Malignant Disorders and Oral cancer -A study on Immunosurveillance. GUPEA (University of Gothenburg. Sahlgrenska Academy, 2015). doi:10.16880/sec.2015.58.01.107.
https://gupea.ub.gu.se/handle/2077/37523 a dissertation of HNSCC
%gupea_2077_37523_4.pdf

There is 5 papers for this dissertation. :-)
The concept of immunosurveillance originally proposed by Dunn et al. in 2004 is well in line with the findings in this thesis of PMOD and oral cancer.

%** Cillo, A. R., Kürten, C. H. L., Tabib, T., Qi, Z., Onkar, S., Wang, T., … Vignali, D. A. A. (2020). Immune Landscape of Viral- and Carcinogen-Driven Head and Neck Cancer. Immunity, 52(1), 183-199.e9. https://doi.org/10.1016/j.immuni.2019.11.014
HPV or chemicals (with Graphical Abstract)

%%
%% 
Evasion of apoptosis contributes to the acquisition of initial oncogenesis, resistance to chemotherapy and to immune effectors
The apoptotic program is carried out through two main pathways: the mitochondrial pathway, which involves mitochondrial outer-membrane permeabilization (MOMP)61; and the death-receptor pathway, which involves ligation of a plasma-membrane receptor, leading to the formation of a death-inducing signalling complex (DISC).

% a long way to go becoming cancer
Chronic inflammation is a risk factor in promoting carcinogenesis. The persistent inflammation in oral lichen planus (with reticular striations) was found that the influx of T helper, cytotoxic T cells, and B cells were increased in oral mucosa with moderate to severe dysplasia and oral squamous cell carcinoma compared to normal or mild dysplasia \citep{Gannot2002}. The infiltration of T cells and dendritic cells were also found in oral lichen planus \citep{Lorenzini2013}.
The incidence of malignant transformation of oral lichen planus is approximately 0.5–1.5\% \citep{Bermejo-Fenoll2009}, while oral leukoplakia is about 10\%.

%%%
\begin{table}
%\begin{sidewaystable}[hp]
\centering
\caption{ The top 10 genes overexpressed with poor prognosis in \acrshort{hnscc} (ranked by Bonferroni adjusted \textit{P} value) }
\arrayrulecolor[rgb]{0.255,0.255,0.255}
\resizebox{\linewidth}{!}{
%\begin{tabular}{|l|l|l|l|l|l|l|l|c|}
\begin{tabularx}{\textwidth}{|l|p{3.5cm}|l|l|l|l|l|l|c|} % *** capital X or p{2cm}
% p{2cm}
\hline
\multicolumn{1}{|c|}{\multirow{2}{*}{Gene ID}} & \multicolumn{1}{c|}{\multirow{2}{*}{Gene Description}}     & \multicolumn{2}{c|}{Kaplan-Meier survival}                                                                                   & \multicolumn{2}{c|}{Univariate~}                                                        & \multicolumn{2}{c|}{Multivariate}                                                       & \multirow{2}{*}{Remark}                     \\ 
\cline{3-8}
\multicolumn{1}{|c|}{}                         & \multicolumn{1}{c|}{}                                      & \multicolumn{1}{c|}{\textit{P} value}               & \multicolumn{1}{c|}{\begin{tabular}[c]{@{}c@{}}Adjusted \\\textit{P} value\end{tabular}} & \multicolumn{1}{c|}{HR*}                   & 95\% CI                                    & HR*                                        & 95\% CI                                    &                                             \\ 
\hline
DKK1                                           & dickkopf WNT signaling pathway inhibitor 1                 & \num{8.9e-8}                               & 0.001                                                                           & 2.266                                      & 1.666-3.082                                & 2.135                                      & 1.559-2.924                                &                                           \\ 
\hline
CAMK2N1                                        & calcium/calmodulin-dependent protein kinase II inhibitor 1 & \num{2.9e-7}                               & 0.002                                                                           & 2.101                                      & 1.572-2.809                                & 2.007                                      & 1.490-2.704                                & \checkmark                                          \\ 
\hline
STC2                                           & stanniocalcin 2                                            & \num{6.5e-7}                               & 0.004                                                                           & 2.147                                      & 1.578-2.921                                & 2.075                                      & 1.515-2.843                                &                                           \\ 
\hline
PGK1                                           & phosphoglycerate kinase 1                                  & \num{9.1e-7}                               & 0.006                                                                           & 2.127                                      & 1.563-2.895                                & 2.046                                      & 1.498-2.795                                & \checkmark                                          \\ 
\hline
SURF4                                          & surfeit 4                                                  & \num{9.6e-7}                               & 0.006                                                                           & 2.055                                      & 1.531-2.757                                & 2.089                                      & 1.543-2.829                                & \checkmark                                           \\ 
\hline
USP10                                          & ubiquitin specific peptidase 10                            & \num{1.7e-6}                               & 0.012                                                                           & 2.083                                      & 1.532-2.834                                & 2.119                                      & 1.551-2.895                                & \checkmark                                         \\ 
\hline
NDFIP1                                         & Nedd4 family interacting protein 1                         & \num{2.6e-6}                               & 0.017                                                                           & 2.031                                      & 1.502-2.746                                & 2.027                                      & 1.483-2.771                                & \checkmark                                           \\ 
\hline
FOXA2                                          & forkhead box A2                                            & \num{2.7e-6}                               & 0.018                                                                           & 1.976                                      & 1.479-2.640                                & 1.914                                      & 1.426-2.569                                & \checkmark                                          \\ 
\hline
STIP1                                          & stress-induced-phosphoprotein 1                            & \num{4.3e-6}                              & 0.029                                                                           & 1.958                                      & 1.463-2.621                                & 1.957                                      & 1.451-2.640                                &                                           \\ 
\hline
DKC1                                           & dyskeratosis congenita 1, dyskerin                         & \num{6.3e-6}                               & 0.042                                                                           & 2.046                                      & 1.490-2.808                                & 1.837                                      & 1.332-2.534                                &                                           \\ 
\hline
\multicolumn{1}{|l!{\color{black}\vrule}}{}    & \multicolumn{1}{l!{\color{black}\vrule}}{}                 & \multicolumn{1}{l!{\color{black}\vrule}}{} & \multicolumn{1}{l!{\color{black}\vrule}}{}                                      & \multicolumn{1}{l!{\color{black}\vrule}}{} & \multicolumn{1}{l!{\color{black}\vrule}}{} & \multicolumn{1}{l!{\color{black}\vrule}}{} & \multicolumn{1}{l!{\color{black}\vrule}}{} & \multicolumn{1}{l!{\color{black}\vrule}}{}  \\ 
\arrayrulecolor{black}\hline
\multicolumn{9}{|l!{\color{black}\vrule}}{\begin{tabular}[c]{@{}l@{}}Selection criteria:~~\\~Kaplan-Meier Bonferroni adjusted \textit{P} $< 0.05 $~\\~Cox's univariate and multivariate$ HR >= 1.5$ \end{tabular}}                                                                                                                                                                                                                                                              \\ 
\hline
\multicolumn{9}{|l!{\color{black}\vrule}}{* Cox's model: \textit{P} $< 0.001$ }                                                                                                                                                                                                                                                                                                                                                                                                  \\ 
\hline
\multicolumn{9}{|l!{\color{black}\vrule}}{Remark: \checkmark validated by GSE2837}                                                                                                                                                                                                                                                                                                                                                             \\
\hline
\end{tabularx}
}
\arrayrulecolor{black}
\label{table:table1}

%\end{sidewaystable}
\end{table}


%%%

%\subsubsection{Table3/legend} or Table 2
%Table3. The 10 candidate genes overexpressed with better prognosis in HNSCC (ranked by Bonferroni corrected Kaplan-Meier P-value).
\begin{table}[hp]
\centering
\caption{The other top 10 genes overexpressed with better prognosis in \acrshort{hnscc} (ranked by Bonferroni corrected \textit{P} value) }
\arrayrulecolor[rgb]{0.255,0.255,0.255}
\resizebox{\linewidth}{!}{%
\begin{tabular}{|l|l|l|l|l|l|l|l|c|} 
\hline
\multicolumn{1}{|c|}{\multirow{2}{*}{Gene ID}} & \multicolumn{1}{c|}{\multirow{2}{*}{Gene Description}} & \multicolumn{2}{l|}{Kaplan-Meier survival}                                                                                                                                                & \multicolumn{2}{c|}{Univariate~}                                                                        & \multicolumn{2}{c|}{Multivariate}                                                                       & \multicolumn{1}{l|}{\multirow{2}{*}{Remark}}  \\ 
\cline{3-8}
\multicolumn{1}{|c|}{}                         & \multicolumn{1}{c|}{}                                  & \multicolumn{1}{c!{\color{black}\vrule}}{\textit{P} value}                                  & \multicolumn{1}{c!{\color{black}\vrule}}{\begin{tabular}[c]{@{}c@{}}Adjusted\\~\textit{P} value\end{tabular}} & \multicolumn{1}{c!{\color{black}\vrule}}{HR*}   & \multicolumn{1}{c!{\color{black}\vrule}}{95\% CI}     & \multicolumn{1}{c!{\color{black}\vrule}}{HR*}   & \multicolumn{1}{c!{\color{black}\vrule}}{95\% CI}     & \multicolumn{1}{l|}{}                         \\ 
\cline{1-2}\arrayrulecolor{black}\cline{3-8}\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{9-9}
ZNF557                                         & zinc finger protein 557                                & \multicolumn{1}{l!{\color{black}\vrule}}{\textcolor[rgb]{0,0,0.471}{\num{8.6e-8}}} & \multicolumn{1}{l!{\color{black}\vrule}}{0.001}                                                      & \multicolumn{1}{l!{\color{black}\vrule}}{0.465} & \multicolumn{1}{l!{\color{black}\vrule}}{0.348-0.619} & \multicolumn{1}{l!{\color{black}\vrule}}{0.499} & \multicolumn{1}{l!{\color{black}\vrule}}{0.372-0.669} &                                              \\ 
\cline{1-2}\arrayrulecolor{black}\cline{3-8}\arrayrulecolor[rgb]{0.255,0.255,0.255}\cline{9-9}
ZNF266                                         & zinc finger protein 266                                & \textcolor[rgb]{0,0,0.471}{\num{2.2e-7}}                                           & 0.001                                                                                                & 0.474                                           & 0.355-0.632                                           & 0.453                                           & 0.338-0.607                                           &                                              \\ 
\hline
IL19                                           & interleukin 19                                         & \textcolor[rgb]{0,0,0.471}{}\num{3.7e-7}\textcolor[rgb]{0,0,0.471}{}               & 0.002                                                                                                & 0.472                                           & 0.351-0.635                                           & 0.459                                           & 0.340-0.619                                           & \checkmark                                            \\ 
\hline
MYO1H                                          & myosin 1H                                              & \textcolor[rgb]{0,0,0.471}{}\num{3.8e-7}\textcolor[rgb]{0,0,0.471}{}               & 0.003                                                                                                & 0.468                                           & 0.347-0.632                                           & 0.467                                           & 0.344-0.634                                           &                                              \\ 
\hline
FCGBP                                          & Fc fragment of IgG binding protein                     & \textcolor[rgb]{0,0,0.471}{}\num{1.2e-6}\textcolor[rgb]{0,0,0.471}{}               & 0.008                                                                                                & 0.484                                           & 0.359-0.653                                           & 0.496                                           & 0.366-0.674                                           &  \checkmark                                           \\ 
\hline
LOC148709                                      & LncRNA LOC148709                                       & \textcolor[rgb]{0,0,0.471}{\num{1.5e-6}}                                           & 0.010                                                                                                & 0.499                                           & 0.374-0.666                                           & 0.485                                           & 0.361-0.652                                           &                                              \\ 
\hline
EVPLL                                          & envoplakin-like protein                                & \textcolor[rgb]{0,0,0.471}{\num{2.0e-6}}                                           & 0.013                                                                                                & 0.490                                           & 0.363-0.661                                           & 0.494                                           & 0.364-0.672                                           &                                              \\ 
\hline
PNMA5                                          & paraneoplastic antigen like 5                          & \textcolor[rgb]{0,0,0.471}{\num{2.6e-6}}                                           & 0.017                                                                                                & 0.499                                           & 0.371-0.671                                           & 0.481                                           & 0.357-0.650                                           &                                              \\ 
\hline
KIAA1683                                       & new name as IQ Motif Containing N (IQCN)                           & \textcolor[rgb]{0,0,0.471}{\num{3.1e-6}}                                           & 0.020                                                                                                & 0.500                                           & 0.371-0.673                                           & 0.483                                           & 0.356-0.654                                           &   \checkmark                                           \\ 
\hline
NPB                                            & neuropeptide B                                         & \textcolor[rgb]{0,0,0.471}{\num{4.0e-6}}                                           & 0.027                                                                                                & 0.460                                           & 0.328-0.646                                           & 0.457                                           & 0.324-0.646                                           &      \checkmark                                        \\ 
\hline
                                               &                                                        &                                                                                    &                                                                                                      &                                                 &                                                       &                                                 &                                                       & \multicolumn{1}{l|}{}                         \\ 
\hline
\multicolumn{9}{|l|}{\begin{tabular}[c]{@{}l@{}}Selection criteria:~\\~Kaplan-Meier Bonferroni adjusted \textit{P} value \textless{} 0.05~\\~Cox's univariate and multivariate HR \textgreater{}= 1.5\\ * Cox's model: \textit{P} value \textless{} 0.001\\lncRNA: Long non-coding RNA\\Remark: \checkmark Validated by GSE2837 ~\end{tabular}}                                                                                                                                                                                                                 \\
\hline
\end{tabular}
}
\arrayrulecolor{black}
\label{table:table3}
\end{table}


%%%%%%%%%%%%%%%%%%%%%%%%

\begin{outline}


%計畫主持人得依計畫實際需要，申請下列各項補助款。
% https://research.nchu.edu.tw/upfile/editor/files/科技部補助專題研究計畫耗材物品圖書及雜項費用及研究設備費支出用途範例.pdf

\1 \cntext{業務費}
%\2 \cntext{研究人事費：34,000 per month}
%含研究助理費及臨時工資等，以科技部補助專題研究計畫助理人員工作酬金支給標準為核定依據: Ph.D. candidate 34,000 per month (2,000 x 17 units)

%\2 \cntext{研究主持費：}
% 近五年內研究績效優異者，得編列研究主持費，經計畫審查小組審核通過後，每人至多核給每月新台幣一萬元 。計畫主持人或共同主持人於研究計畫執行期間僅得由此補助 辦法支領一份研究主持費。於研究計畫執行期間，因離職、轉 任或無法執行該計畫時，即停止核發研究主持費。
\2 \cntext{耗材費：} %消耗性器材與藥品費、問卷 調查費 及其它事務性費用(如電腦耗材費、郵電費、印刷影印費、資料檢索費、國內差旅費、論文發表費) 在核定 經費限額內核實列支
\3 \cntext{研究計畫人體試驗委員會審查費: 10,000元，修正案: 1,000元，結案: 1,000元。小計: 12,000元。)}
\3 \cntext{生物資料庫使用費: 切片費 5片330元，H\&E stain 每片40元，臨床病歷每名 1,400元。46名案例小計: 69,540元。}
\3 \cntext{臺北醫學大學數據處研究資料庫檢索費: 目前免費}
% 攜入(linked my data) IRB applying 勾選門診病歷 EHR
%(使用傳輸網路所供應新穎數據或索取各交換系統資料庫中之資料所需費用)}
\3 \cntext{儀器使用費: Aperio ScanScope 數位病理切片掃描處理費: 300元/小時，約須10小時，小計3,000元} 
% AT2 Scanner WSI
% free at 中央研究院國家基因體醫學研究中心 GRC 5L09B, Aperio AT2 400片自動掃描
% QuPath; 20x15 mm for 3Gb storage of HDD
% TMUH pathology: aetherAI 雲象科技
% 張資昊 Tzu-Hao Chang 醫學資訊研究所 副教授 (兼任core manager) kevinchang@tmu.edu.tw
%\3 \cntext{資料庫使用費}
\3 \cntext{程式設計與雲端運算: 國網中心高速電腦GPU人工智慧訓練與儲存費用 cm.4xsuper for PyTorch NTD689/小時，小計13,780元} % 容器運算服務 (Container Compute Service, CCS) 記憶體優化容器：可負擔記憶體內部處理大型資料集之工作負載，加強效能，專為 TWCC 三種容器工作環境設計：
%PyTorch： GPU 加速的張量運算框架，並以 Python 為前端語言。使用常見的Python 函式庫(如 NumPy、SciPy 和 Cython )便能輕鬆地擴展多項功能。 https://man.twcc.ai/@twccdocs/SJWlN3YDr?type=view 有關網站建置、程式設計和拍攝影片、實作等相關費用 https://man.twcc.ai/@twccdocs/SJWlN3YDr?type=view
%https://www.twcc.ai 臺灣杉二號
%\3 \cntext{問卷調查費(實地調查訪問及搜集各種資料所需費用)} 
%\3 \cntext{受試者禮品、營養品及交通費}
%\3 \cntext{國內或國際性學會之年費或入會費}
\3 \cntext{論文發表費: 56,000元 (APC for SCI journal)}
%(本部補助研究計畫之研究成果發表於國內外著名之學術期刊所需之相關費用)}

% http://ohr.tmu.edu.tw/front/jbb1/Charges/news.php?ID=dG11X29ociZDaGFyZ2Vz&Sn=738; 家佑可以打折
% 校內預算或計畫經費轉帳：不需請購，直接請款，統編為03724606-B，並將繳費單連同初審通過之《會一單》送回聯合人體生物資料庫結案。
% Unstained paraffin section 切片費 5片/piece 330
% H\&E stain 片/piece 40
% Data process and consult fee 基本規格 1,200
% => https://ods.tmu.edu.tw/portal_c3_cnt.php?owner_num=c3_69585&button_num=c3&folder_id=4165 Taipei Medical University Clinical Research Database (TMUCRD)
% It collects various electronic medical records (\acrshort{ehr}) data of TMUH, including structured data (such as patient's basic information, medical information, laboratory reports, diagnosis results, treatment procedures, surgery and medication status) and unstructured data (such as physicians records, pathology reports, medical imaging reports).
%\3 \cntext{電腦使用費: }
% (使用電腦所需費用，如以執行機構出具之收據報銷，應檢附計算標 準、實際使用時數及耗材支用情形等支出 數據資料)
%\3 \cntext{翻譯及潤稿費}
%\3 \cntext{會議餐費(因研究計畫需要召開會議而逾用餐時間所提供之餐點)}
%\3 \cntext{田野調查之茶點費}
%\3 \cntext{計畫相關之教育訓練費}
%\3 \cntext{研究計畫所需的圖書(購置之圖書是否列入財產，由執行機構依「圖書館法」及「財物標準分類」有關財產與非財產之規定辦理)}

%\3 \cntext{物品(非屬研究設備者)} 
%\3 \cntext{印刷與影印費}
%\3 \cntext{文具}
%\3 \cntext{紙張}
%\3 \cntext{郵電費}


\1 \cntext{研究設備費:} 
\2 \cntext{資訊硬體設備 Mac Mini M1 37,100元}
% Apple 教育優惠 Mac Mini M1
% Apple M1 晶片配備 8 核心 CPU、8 核心 GPU
% 16 核心神經網路引擎
%16GB 統一記憶體
%1TB SSD 儲存裝置

%在核定 經費限額內核實列支
% 凡執行研究計畫所需單價在新台幣一萬元以上且使 用年限在二年以上之各項儀器、機械及資訊設備(含各項電腦設 施、網路系統、周邊設備、套裝軟體、程式設計費)等之購置、 裝置費用及圖書館典藏等，本項設備之採購，以與研究計畫直接 有關者為限，研究設備費補助金額不得超過總補助經費20%
%\1 \cntext{管理費:} % 5% 為執行機 構配合執行研究 計畫所需之費 用，由執行機構統 籌支用

\1 \cntext{總計: 191,420元}
\end{outline}
